{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import os\n",
    "import re\n",
    "import datetime\n",
    "\n",
    "import pandas as pd\n",
    "from tqdm.auto import tqdm\n",
    "\n",
    "from dotenv import load_dotenv\n",
    "from langchain_huggingface import HuggingFaceEndpoint\n",
    "from langchain.chains import LLMChain\n",
    "from langchain_core.prompts import PromptTemplate\n",
    "\n",
    "from extractor import Store\n",
    "\n",
    "tqdm.pandas()  # load tqdm's pandas support\n",
    "pd.set_option(\"display.max_colwidth\", None)\n",
    "\n",
    "load_dotenv()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "QUESTION_GEN_LLM = \"NousResearch/Nous-Hermes-2-Mixtral-8x7B-DPO\"\n",
    "QUESTION_GEN_LLM_CONFIG = {\n",
    "    \"max_new_tokens\": 256,\n",
    "    \"top_k\": 10,\n",
    "    \"top_p\": 0.95,\n",
    "    \"typical_p\": 0.95,\n",
    "    \"temperature\": 0.01,\n",
    "    \"repetition_penalty\": 1.03,\n",
    "}\n",
    "\n",
    "now = datetime.now()\n",
    "date_time_str = now.strftime(\"%d.%m.%Y_%H.%M\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "vec_store = Store(\"main\",presist_dir=\"./chroma_langchain_db\")\n",
    "vec_store.setup()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Dumbassery\n",
    "Currently we use a kinda dumbass method of manually picking docs to be contex wherein the model generates a question\n",
    "\n",
    "In the future maybe we could use something like [AutoRAG](https://docs.auto-rag.com/)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Page context tuple generation from doc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "def divide_chunks(l, n):\n",
    "    \n",
    "    # looping till length l\n",
    "    for i in range(0, len(l), n): \n",
    "        yield l[i:i + n]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "TUP_PAGE_COUNT = 2\n",
    "QUESTION_PER_CTX_COUNT = 5\n",
    "context_doc_tups = []\n",
    "\n",
    "all_docs_pages = vec_store.store.get()\n",
    "all_docs_pages_ids = [tuple(doc.split(\"_\")) for doc in all_docs_pages[\"ids\"]]\n",
    "doc_names = {doc.split(\"_\")[0] for doc in all_docs_pages[\"ids\"]}\n",
    "\n",
    "# NESTING HELL!\n",
    "all_docs = {}\n",
    "for doc_name in doc_names:\n",
    "    all_docs[doc_name] = []\n",
    "    for doc in all_docs_pages_ids:\n",
    "        if doc[0] == doc_name:\n",
    "            all_docs[doc_name].append(\"_\".join(doc))\n",
    "\n",
    "for k, v in all_docs.items():\n",
    "    all_docs[k] = list(divide_chunks(v, TUP_PAGE_COUNT))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "('<SENT 01_01>\\nAssociations of polymorphisms of eight muscle- or metabolism-related genes\\nwith performance in Mount Olympus marathon runners\\nGeorgios I. Tsianos,1 Evangelos Evangelou,1 Arnoud Boot,2 M. Carola Zillikens,2 Joyce B. J. van Meurs,2\\nAndre G. Uitterlinden,2,3 and John P. A. Ioannidis1,4\\n1Department of Hygiene and Epidemiology, University of Ioannina School of Medicine, Ioannina, Greece; Departments\\nof 2Internal Medicine and3Epidemiology, Erasmus Medical Center, Rotterdam, The Netherlands; and4Center for Genetic\\nEpidemiology and Modeling, Institute for Clinical Research and Health Policy Studies, Tufts Medical Center, Tufts University\\nSchool of Medicine, Boston, Massachusetts\\nTsianos GI, Evangelou E, Boot A, Zillikens MC, van Meurs JB,\\nUitterlinden AG, Ioannidis JP.Associations of polymorphisms of eight\\nmuscle- or metabolism-related genes with performance in Mount Olympus marathon runners.J Appl Physiol108: 567–574,2010.\\n</SENT 01_01>\\n<SENT 01_02>\\nFirst published\\nDecember 31, 2009; doi:10.1152/japplphysiol.00780.2009.—Athletic endurance performance is probably partly under genetic control, but\\ngenetic association studies have yielded inconclusive results.The\\nobjective of the present study was to evaluate the association of\\npolymorphisms in eight muscle- or metabolism-related genes with\\nendurance performance in participants of the Olympus Marathon\\nrunning race.We recruited 438 athletes who participated in the 2007\\nand 2008 annual running events of the Olympus Marathon: a 43.8-km\\nrace with an ascent from sea level to 2,690-m altitude and then a\\ndescent to 300 m. Phenotypes of interest were thecompetitive event\\ntime at the specific Olympus Marathon where the athlete was enrolled,\\nthe fastest reported timing ever achieved in an Olympus Marathon, and\\nhow many kilometers per week the athlete ran during the previous year.\\n</SENT 01_02>\\n<SENT 01_03>\\nEleven polymorphisms in/H92513-actinin ( ACTN3), AMP deaminase-1\\n(AMPD1), bradykinin B 2 receptor (BDKRB2), /H92522-adrenergic receptor\\n(ADRB2), peroxisome proliferator-activated receptor (PPAR)-/H9253coactivator-1/H9251(PPARGC1A), PPAR-/H9251(PPARA), PPAR-/H9254(PPARD), and apoliprotein E (APOE) were evaluated.Hardy-Weinberg equilibrium testing\\non the overall cohort of male athletes showed a significant deviation for\\nBDKRB2rs1799722 (P /H110050.018;P /H110050.006 when limited to 316 habitual\\nmale runners) with an excess of the TT genotype.\\n</SENT 01_03>\\n<SENT 01_04>\\nAcross all athletes, no\\nassociations showed nominal statistical significance for any of the\\nthree phenotypes, and the same was true when analyses were limited\\nto men (n /H11005417).When limited to 316 male athletes who identified\\nrunning as their preferred sport, ADRB2 rs1042713 had nominally\\nsignificant associations with faster times for the minor (A) allele for\\nthe fastest time ever (P /H110050.01).The direction of effect was identical\\nas previously postulated only for BDKRB2 rs1799722 and ADRB2\\nrs1042713, indicating consistency.BDKRB2 rs1799722 and ADRB2\\nrs1042713 have some support for being implicated in endurance\\nperformance among habitual runners and require further investigation.\\n</SENT 01_04>\\n<SENT 01_05>\\ngenetics; endurance\\nPHYSICAL FITNESS is a very complex phenotype influenced by\\nnumerous genetic and environmental factors contributing to the\\nobserved interindividual variation not only in the general\\npopulation but also in trained athletes (24).Numerous studies\\nhave been performed in the last decade to try to establish\\nwhether specific genes with postulated functional roles in\\nhuman physiology influence athletic performance and endurance, and a growing number of candidate gene associations\\nhave been proposed (5, 27).\\n</SENT 01_05>\\n<SENT 01_06>\\nStudies have varied in sample size\\nand type of population, and several proposed associations have\\nnot been consistently replicated in independent investigations\\nby different teams of investigators.Multiple genetic loci are likely to underlie the heritability of\\nthe complex phenotype of athletic performance.Some aspects\\nof this phenotype have high estimated heritability, for example,\\nthe heritability of maximal O2 uptake (V˙O2max)i s /H1101150% in\\nboth the sedentary state and after training (3, 4).Even if\\nheritability is less for other aspects of athletic performance,\\neach genetic variant is likely to explain only a small fraction of\\nthe genetic predisposition.\\n</SENT 01_06>\\n<SENT 01_07>\\nRecently, Williams and Folland (44)\\ncalculated the probability that exists for an individual to be in\\npossession of any of the most optimal of the 23 selected\\npolymorphisms related to athletic performance.Studies investigating the collective influence of a number of genetic polymorphisms could possibly help unravel and thus possibly better\\nexplain the inherent individual variations in athletic performance.In the present study, we studied participants in the\\nOlympus Marathon, an athletic event that requires considerable\\nendurance and for which there are large differences in performance among the participants.The optimal physiological phenotype of athletes competing in long-duration events probably\\nentails an inherent genetic makeup conferring cardiovascular,\\npulmonary, and skeletal competence to perform during such\\nevents and efficient metabolism of available substrates to\\nsustain the performance throughout the event’s duration.\\n</SENT 01_07>\\n<SENT 01_08>\\nWe evaluated single-nucleotide polymorphisms (SNPs) in\\neight genes that have known functional roles, and we based our\\nselection on previous gene association studies investigating\\naspects of athletic endurance.In particular, the selected genes\\nin our study have been previously associated with appropriate\\nsubstrate metabolism as a source of energy during exercise, in the\\ncardiovascular and pulmonary responses to exercise that would\\nallow for a much efficient O2 uptake and transfer to the muscles\\nfor utilization, and/or in muscle contractility suitable for the\\npurposes of endurance performance.\\n</SENT 01_08>\\n<SENT 01_09>\\nMoreover, the selected\\npolymorphisms had been evaluated in the past in one or more\\nstudies, where at least one of the studies had reported significant associations with endurance performance (5, 27).METHODS\\nCohort participants.We enrolled athletes participated in the annual\\nOlympus Marathon event set on Mount Olympus in Greece.We only\\nenrolled athletes of self-verified Greek ancestry to avoid confounding\\ndata from population stratification.Athletes were approached at the\\ncompetition site during the 2 days before the race by a member of\\nthe research team.\\n</SENT 01_09>\\n<SENT 01_10>\\nThey were informed about the aim and purpose of\\nthe study and were asked whether or not they would be interested in\\nparticipating in the study.Written informed consent was obtained\\nAddress for reprint requests and other correspondence: J. P. A. Ioannidis,\\nDept.of Hygiene and Epidemiology, Univ.of Ioannina School of Medicine,\\nIoannina 45 110, Greece (e-mail: jioannid@cc.uoi.gr).J Appl Physiol108: 567–574, 2010.First published December 31, 2009; doi:10.1152/japplphysiol.00780.2009.8750-7587/10 $8.00 Copyright © 2010 the American Physiological Societyhttp://www.jap.org 567\\nDownloaded from journals.physiology.org/journal/jappl (110.138.091.074) on January 26, 2023.\\n</SENT 01_10>\\n',\n",
       " '<SENT 02_01>\\nfrom all volunteers.Athletes filled out a short questionnaire on\\npersonal identifiers; age, sex, and current height and weight; whether\\nrunning was their preferred sport (and, if not, which sport was their\\npreferred one); how many kilometers they had run per week in the last\\nweek, month, and year before the competition; the highest altitude\\nthey had ever reached; and the longest distance they had ever ran\\nwithout stopping.A saliva sample was finally collected for DNA\\nanalysis.The data collection for the study occurred at the 2007 and\\n2008 annual events.\\n</SENT 02_01>\\n<SENT 02_02>\\nThe study protocol was reviewed by the University Hospital of Ioannina Scientific Committee, and ethical approval\\nwas granted.Race course description.The race course length is 43.8 km.It\\nentails a gradual ascent from the start line located a t 3 m above sea\\nlevel, reaches a maximum altitude of 2,690 m, and then has a downhill\\nroute to the finish line at an altitude of 300 m. The race provides an\\nexcellent event for assessing variability in endurance performance.\\n</SENT 02_02>\\n<SENT 02_03>\\nThe winner usually completes the race in/H110115 h, but times of people\\nthat have completed the race vary substantially and can exceed 14 h.\\nGenotyping.Saliva samples (2 ml) were collected from all volunteers using the Oragene DNA Self Collection Kits (DNA Genotek)\\naccording to the manufacturer’s instructions; the Oragene collection\\ntube also contained 2 ml of stabilizing agent.All samples were\\ncollected at the competition site in Greece and were then sent for\\nanalysis at Erasmus University Rotterdam (Rotterdam, The Netherlands).Samples were processed using a PUREGENE DNA Purification kit (DNA Genotek’s Oragene DNA Purification protocol, DNA\\nGenotec, Ottawa, Ontario, Canada).\\n</SENT 02_03>\\n<SENT 02_04>\\nFor DNA isolation, 3 ml of the\\nsaliva mixture were used, and 120 ml of Oragene purifier were added.After a 10-min incubation on ice, the vial was spun down, and the\\npellet was discarded.Subsequently, an equal volume of ethanol was\\nadded to precipitate the DNA.After a 10-min incubation at room\\ntemperature, the vial was spun down, the supernatant was discarded,\\nand the pellet was air dried.The DNA pellet was then dissolved in 300\\nml MilliQ water.Polymorphisms under study.The selected genes, their respective\\nSNP identification numbers, and the implicated function of these\\ngenes are shown in Table 1.For all genes under study, one variant has\\nbeen investigated for associations with endurance performance and\\nrelated phenotypes except for the peroxisome proliferator-activated\\nreceptor (PPAR)-/H9254gene ( PPARD), where three variants have been\\nproposed to affect endurance.\\n</SENT 02_04>\\n<SENT 02_05>\\nFor all SNPs, Taqman assays were\\ngenerated and applied according to the manufacturers’s specifications.Results were analyzed with the ABI Taqman 7900HT using sequence\\ndetection system 2.22 software (Applied Biosystems, Foster City,\\nCA).To confirm the accuracy of the genotyping results, 5% of the\\nrandomly selected samples were regenotyped with the same method.No inconsistencies were observed.All primers and probes used in the\\npresent study are available on request.Phenotypes.We defined a priori the following phenotypes: 1) the\\nevent completion time at the specific Olympus Marathon where the\\nathlete was enrolled in the study,2) the fastest reported event\\ncompletion time ever achieved in the Olympus Marathon race, and\\n3) how many kilometers per week the athlete ran during the 12 mo\\nbefore enrollment in the study.For athletes who ran both years and were included in our sample\\nsize, we used for the time to finish the time in the year during which\\nthey were first enrolled.\\n</SENT 02_05>\\n<SENT 02_06>\\nHowever, if they only completed one of the\\ntwo races, then we used the time when they completed the race.All\\nathletes that exceeded the time limit for finishing the race set by the\\norganizing committee (12 h in 2007 and 10 h in 2008) were personally\\ncontacted to provide a self-reported time finish, since the official time\\nwas not available.Statistical analyses.Genotypes for each tested polymorphism were\\ntested for compliance with the Hardy-Weinberg equilibrium (HWE)\\nlaw using an exact test.Deviations from HWE may be due to different\\nreasons (31), but, in this design, they may also be used as a crude test\\nof association (36), since the analyzed cohort is a selected population\\nTable 1.\\n</SENT 02_06>\\n<SENT 02_07>\\nPolymorphisms under study\\nGene Name Major Role rs Number(s) Genotypes, number (%)\\nACTN3 /H92513-Actinin Skeletal muscle component rs1815739 CT: 214 (48.9) CC: 132 (30.1) TT: 85 (19.4) Undetermined: 7 (1.6)\\nAMPD1 AMP deaminase-1 Skeletal muscle metabolism rs17602729 GG: 336 (76.7) AG: 93 (21.2) AA: 4 (0.9) Undetermined: 5 (1.1)\\nBDKRB2 Bradykinin B2 receptor Bradykinin receptor rs1799722 CC: 169 (38.6) CT: 183 (41.8) TT: 80 (18.3) Undetermined: 6 (1.4)\\nADRB2 /H92522-Adrenergic receptor Adrenergic receptor rs1042713 GG: 165 (37.7) GA: 207 (47.3) AA: 60 (13.7) Undetermined: 6 (1.4)\\nPPARGC1A PPAR coactivator-1/H9251Regulation of energy\\nmetabolism genes\\nrs8192678 AA: 297 (67.8) GA: 119 (27.2) GG: 17 (3.9) Undetermined: 5 (1.1)\\nPPARA PPAR-/H9251/H9252 -Oxidation of fatty acids rs4253778 GG: 284 (64.8) GC: 135 (30.8) CC: 15 (3.4) Undetermined: 4 (0.9)\\nPPARD PPAR-/H9254/H9252 -Oxidation of fatty acids rs6902123 TT: 351 (80.1) TC: 78 (17.8) CC: 2 (0.5) Undetermined: 7 (1.6)\\nrs1053049 TT: 221 (50.5) CT: 188 (42.9) CC: 23 (5.3) Undetermined: 6 (1.4)rs2267668 AA: 297 (67.8) GA: 119 (27.2) GG: 17 (3.9) Undetermined: 5 (1.4)\\nAPOE Apolipoprotein E Binding and catabolism of\\nlipoproteins\\nrs7412 and rs429358 other/other: 366 (83.5) ε4/other: 59 (13.5) ε4/ ε4: 4 (0.9) Undetermined: 9 (2.1)\\nPPAR, peroxisome proliferator-activated receptor.For APOE, the two polymorphisms are linked and define six possible genotypes ( ε2/ε2, ε2/ε3, ε3/ε3, ε3/ε4, ε4/ε4, and ε2/ε4).568 POLYMORPHISMS IN OLYMPUS MARATHON RUNNERS\\nJ Appl Physiol\\x7f VOL 108 \\x7f MARCH 2010 \\x7f www.jap.org\\nDownloaded from journals.physiology.org/journal/jappl (110.138.091.074) on January 26, 2023.\\n</SENT 02_07>\\n')"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "all_docs_texts = {}\n",
    "for k, chunks in all_docs.items():\n",
    "    chunks_docs = []\n",
    "    for chunk in chunks:\n",
    "        # NOTE: Maybe add zip with page_doc names?\n",
    "        chunk_doc = tuple(vec_store.store.get(chunk)['documents'])\n",
    "        chunks_docs.append(chunk_doc)\n",
    "    all_docs_texts[k] = chunks_docs\n",
    "\n",
    "for context_tups in all_docs_texts.values():\n",
    "    context_doc_tups.extend(context_tups)\n",
    "\n",
    "context_doc_tups[0]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Question Generation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "QUESTION_GEN_PROMPT = \"\"\"\n",
    "You are a question-generation assistant. Your sole task is to generate thoughtful, clarifying, and exploratory questions based on the provided context.  \n",
    "\n",
    "Context: {context}\n",
    "\n",
    "Provide the questions as follows:\n",
    "Questions:\n",
    "1.\n",
    "2.\n",
    "3.\n",
    "...\n",
    "\n",
    "Generate only questions that aim to explore or refine the given context further. Avoid any assumptions, interpretations, or answers—just ask questions. Seperate different question with a newline\n",
    "\n",
    "Questions:\n",
    "\"\"\"\n",
    "prompt = PromptTemplate.from_template(QUESTION_GEN_PROMPT)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "llm = HuggingFaceEndpoint(\n",
    "    repo_id=QUESTION_GEN_LLM,\n",
    "    huggingfacehub_api_token=os.environ[\"HUGGINGFACEHUB_API_TOKEN\"],\n",
    "    **QUESTION_GEN_LLM_CONFIG\n",
    ")\n",
    "llm_chain = prompt | llm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "4e5f9a6d19874703b3afb118cddb8532",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "  0%|          | 0/15 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "question_context_pair = []\n",
    "for context in tqdm(context_doc_tups):\n",
    "    ctx_text = \"\\n\\n\".join(context)\n",
    "    questions = llm_chain.invoke({\"context\": ctx_text}).split(\"\\n\")[:QUESTION_PER_CTX_COUNT]\n",
    "    question_context_pair.append((ctx_text, questions))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('<SENT 01_01>\\nAssociations of polymorphisms of eight muscle- or metabolism-related genes\\nwith performance in Mount Olympus marathon runners\\nGeorgios I. Tsianos,1 Evangelos Evangelou,1 Arnoud Boot,2 M. Carola Zillikens,2 Joyce B. J. van Meurs,2\\nAndre G. Uitterlinden,2,3 and John P. A. Ioannidis1,4\\n1Department of Hygiene and Epidemiology, University of Ioannina School of Medicine, Ioannina, Greece; Departments\\nof 2Internal Medicine and3Epidemiology, Erasmus Medical Center, Rotterdam, The Netherlands; and4Center for Genetic\\nEpidemiology and Modeling, Institute for Clinical Research and Health Policy Studies, Tufts Medical Center, Tufts University\\nSchool of Medicine, Boston, Massachusetts\\nTsianos GI, Evangelou E, Boot A, Zillikens MC, van Meurs JB,\\nUitterlinden AG, Ioannidis JP.Associations of polymorphisms of eight\\nmuscle- or metabolism-related genes with performance in Mount Olympus marathon runners.J Appl Physiol108: 567–574,2010.\\n</SENT 01_01>\\n<SENT 01_02>\\nFirst published\\nDecember 31, 2009; doi:10.1152/japplphysiol.00780.2009.—Athletic endurance performance is probably partly under genetic control, but\\ngenetic association studies have yielded inconclusive results.The\\nobjective of the present study was to evaluate the association of\\npolymorphisms in eight muscle- or metabolism-related genes with\\nendurance performance in participants of the Olympus Marathon\\nrunning race.We recruited 438 athletes who participated in the 2007\\nand 2008 annual running events of the Olympus Marathon: a 43.8-km\\nrace with an ascent from sea level to 2,690-m altitude and then a\\ndescent to 300 m. Phenotypes of interest were thecompetitive event\\ntime at the specific Olympus Marathon where the athlete was enrolled,\\nthe fastest reported timing ever achieved in an Olympus Marathon, and\\nhow many kilometers per week the athlete ran during the previous year.\\n</SENT 01_02>\\n<SENT 01_03>\\nEleven polymorphisms in/H92513-actinin ( ACTN3), AMP deaminase-1\\n(AMPD1), bradykinin B 2 receptor (BDKRB2), /H92522-adrenergic receptor\\n(ADRB2), peroxisome proliferator-activated receptor (PPAR)-/H9253coactivator-1/H9251(PPARGC1A), PPAR-/H9251(PPARA), PPAR-/H9254(PPARD), and apoliprotein E (APOE) were evaluated.Hardy-Weinberg equilibrium testing\\non the overall cohort of male athletes showed a significant deviation for\\nBDKRB2rs1799722 (P /H110050.018;P /H110050.006 when limited to 316 habitual\\nmale runners) with an excess of the TT genotype.\\n</SENT 01_03>\\n<SENT 01_04>\\nAcross all athletes, no\\nassociations showed nominal statistical significance for any of the\\nthree phenotypes, and the same was true when analyses were limited\\nto men (n /H11005417).When limited to 316 male athletes who identified\\nrunning as their preferred sport, ADRB2 rs1042713 had nominally\\nsignificant associations with faster times for the minor (A) allele for\\nthe fastest time ever (P /H110050.01).The direction of effect was identical\\nas previously postulated only for BDKRB2 rs1799722 and ADRB2\\nrs1042713, indicating consistency.BDKRB2 rs1799722 and ADRB2\\nrs1042713 have some support for being implicated in endurance\\nperformance among habitual runners and require further investigation.\\n</SENT 01_04>\\n<SENT 01_05>\\ngenetics; endurance\\nPHYSICAL FITNESS is a very complex phenotype influenced by\\nnumerous genetic and environmental factors contributing to the\\nobserved interindividual variation not only in the general\\npopulation but also in trained athletes (24).Numerous studies\\nhave been performed in the last decade to try to establish\\nwhether specific genes with postulated functional roles in\\nhuman physiology influence athletic performance and endurance, and a growing number of candidate gene associations\\nhave been proposed (5, 27).\\n</SENT 01_05>\\n<SENT 01_06>\\nStudies have varied in sample size\\nand type of population, and several proposed associations have\\nnot been consistently replicated in independent investigations\\nby different teams of investigators.Multiple genetic loci are likely to underlie the heritability of\\nthe complex phenotype of athletic performance.Some aspects\\nof this phenotype have high estimated heritability, for example,\\nthe heritability of maximal O2 uptake (V˙O2max)i s /H1101150% in\\nboth the sedentary state and after training (3, 4).Even if\\nheritability is less for other aspects of athletic performance,\\neach genetic variant is likely to explain only a small fraction of\\nthe genetic predisposition.\\n</SENT 01_06>\\n<SENT 01_07>\\nRecently, Williams and Folland (44)\\ncalculated the probability that exists for an individual to be in\\npossession of any of the most optimal of the 23 selected\\npolymorphisms related to athletic performance.Studies investigating the collective influence of a number of genetic polymorphisms could possibly help unravel and thus possibly better\\nexplain the inherent individual variations in athletic performance.In the present study, we studied participants in the\\nOlympus Marathon, an athletic event that requires considerable\\nendurance and for which there are large differences in performance among the participants.The optimal physiological phenotype of athletes competing in long-duration events probably\\nentails an inherent genetic makeup conferring cardiovascular,\\npulmonary, and skeletal competence to perform during such\\nevents and efficient metabolism of available substrates to\\nsustain the performance throughout the event’s duration.\\n</SENT 01_07>\\n<SENT 01_08>\\nWe evaluated single-nucleotide polymorphisms (SNPs) in\\neight genes that have known functional roles, and we based our\\nselection on previous gene association studies investigating\\naspects of athletic endurance.In particular, the selected genes\\nin our study have been previously associated with appropriate\\nsubstrate metabolism as a source of energy during exercise, in the\\ncardiovascular and pulmonary responses to exercise that would\\nallow for a much efficient O2 uptake and transfer to the muscles\\nfor utilization, and/or in muscle contractility suitable for the\\npurposes of endurance performance.\\n</SENT 01_08>\\n<SENT 01_09>\\nMoreover, the selected\\npolymorphisms had been evaluated in the past in one or more\\nstudies, where at least one of the studies had reported significant associations with endurance performance (5, 27).METHODS\\nCohort participants.We enrolled athletes participated in the annual\\nOlympus Marathon event set on Mount Olympus in Greece.We only\\nenrolled athletes of self-verified Greek ancestry to avoid confounding\\ndata from population stratification.Athletes were approached at the\\ncompetition site during the 2 days before the race by a member of\\nthe research team.\\n</SENT 01_09>\\n<SENT 01_10>\\nThey were informed about the aim and purpose of\\nthe study and were asked whether or not they would be interested in\\nparticipating in the study.Written informed consent was obtained\\nAddress for reprint requests and other correspondence: J. P. A. Ioannidis,\\nDept.of Hygiene and Epidemiology, Univ.of Ioannina School of Medicine,\\nIoannina 45 110, Greece (e-mail: jioannid@cc.uoi.gr).J Appl Physiol108: 567–574, 2010.First published December 31, 2009; doi:10.1152/japplphysiol.00780.2009.8750-7587/10 $8.00 Copyright © 2010 the American Physiological Societyhttp://www.jap.org 567\\nDownloaded from journals.physiology.org/journal/jappl (110.138.091.074) on January 26, 2023.\\n</SENT 01_10>\\n\\n\\n<SENT 02_01>\\nfrom all volunteers.Athletes filled out a short questionnaire on\\npersonal identifiers; age, sex, and current height and weight; whether\\nrunning was their preferred sport (and, if not, which sport was their\\npreferred one); how many kilometers they had run per week in the last\\nweek, month, and year before the competition; the highest altitude\\nthey had ever reached; and the longest distance they had ever ran\\nwithout stopping.A saliva sample was finally collected for DNA\\nanalysis.The data collection for the study occurred at the 2007 and\\n2008 annual events.\\n</SENT 02_01>\\n<SENT 02_02>\\nThe study protocol was reviewed by the University Hospital of Ioannina Scientific Committee, and ethical approval\\nwas granted.Race course description.The race course length is 43.8 km.It\\nentails a gradual ascent from the start line located a t 3 m above sea\\nlevel, reaches a maximum altitude of 2,690 m, and then has a downhill\\nroute to the finish line at an altitude of 300 m. The race provides an\\nexcellent event for assessing variability in endurance performance.\\n</SENT 02_02>\\n<SENT 02_03>\\nThe winner usually completes the race in/H110115 h, but times of people\\nthat have completed the race vary substantially and can exceed 14 h.\\nGenotyping.Saliva samples (2 ml) were collected from all volunteers using the Oragene DNA Self Collection Kits (DNA Genotek)\\naccording to the manufacturer’s instructions; the Oragene collection\\ntube also contained 2 ml of stabilizing agent.All samples were\\ncollected at the competition site in Greece and were then sent for\\nanalysis at Erasmus University Rotterdam (Rotterdam, The Netherlands).Samples were processed using a PUREGENE DNA Purification kit (DNA Genotek’s Oragene DNA Purification protocol, DNA\\nGenotec, Ottawa, Ontario, Canada).\\n</SENT 02_03>\\n<SENT 02_04>\\nFor DNA isolation, 3 ml of the\\nsaliva mixture were used, and 120 ml of Oragene purifier were added.After a 10-min incubation on ice, the vial was spun down, and the\\npellet was discarded.Subsequently, an equal volume of ethanol was\\nadded to precipitate the DNA.After a 10-min incubation at room\\ntemperature, the vial was spun down, the supernatant was discarded,\\nand the pellet was air dried.The DNA pellet was then dissolved in 300\\nml MilliQ water.Polymorphisms under study.The selected genes, their respective\\nSNP identification numbers, and the implicated function of these\\ngenes are shown in Table 1.For all genes under study, one variant has\\nbeen investigated for associations with endurance performance and\\nrelated phenotypes except for the peroxisome proliferator-activated\\nreceptor (PPAR)-/H9254gene ( PPARD), where three variants have been\\nproposed to affect endurance.\\n</SENT 02_04>\\n<SENT 02_05>\\nFor all SNPs, Taqman assays were\\ngenerated and applied according to the manufacturers’s specifications.Results were analyzed with the ABI Taqman 7900HT using sequence\\ndetection system 2.22 software (Applied Biosystems, Foster City,\\nCA).To confirm the accuracy of the genotyping results, 5% of the\\nrandomly selected samples were regenotyped with the same method.No inconsistencies were observed.All primers and probes used in the\\npresent study are available on request.Phenotypes.We defined a priori the following phenotypes: 1) the\\nevent completion time at the specific Olympus Marathon where the\\nathlete was enrolled in the study,2) the fastest reported event\\ncompletion time ever achieved in the Olympus Marathon race, and\\n3) how many kilometers per week the athlete ran during the 12 mo\\nbefore enrollment in the study.For athletes who ran both years and were included in our sample\\nsize, we used for the time to finish the time in the year during which\\nthey were first enrolled.\\n</SENT 02_05>\\n<SENT 02_06>\\nHowever, if they only completed one of the\\ntwo races, then we used the time when they completed the race.All\\nathletes that exceeded the time limit for finishing the race set by the\\norganizing committee (12 h in 2007 and 10 h in 2008) were personally\\ncontacted to provide a self-reported time finish, since the official time\\nwas not available.Statistical analyses.Genotypes for each tested polymorphism were\\ntested for compliance with the Hardy-Weinberg equilibrium (HWE)\\nlaw using an exact test.Deviations from HWE may be due to different\\nreasons (31), but, in this design, they may also be used as a crude test\\nof association (36), since the analyzed cohort is a selected population\\nTable 1.\\n</SENT 02_06>\\n<SENT 02_07>\\nPolymorphisms under study\\nGene Name Major Role rs Number(s) Genotypes, number (%)\\nACTN3 /H92513-Actinin Skeletal muscle component rs1815739 CT: 214 (48.9) CC: 132 (30.1) TT: 85 (19.4) Undetermined: 7 (1.6)\\nAMPD1 AMP deaminase-1 Skeletal muscle metabolism rs17602729 GG: 336 (76.7) AG: 93 (21.2) AA: 4 (0.9) Undetermined: 5 (1.1)\\nBDKRB2 Bradykinin B2 receptor Bradykinin receptor rs1799722 CC: 169 (38.6) CT: 183 (41.8) TT: 80 (18.3) Undetermined: 6 (1.4)\\nADRB2 /H92522-Adrenergic receptor Adrenergic receptor rs1042713 GG: 165 (37.7) GA: 207 (47.3) AA: 60 (13.7) Undetermined: 6 (1.4)\\nPPARGC1A PPAR coactivator-1/H9251Regulation of energy\\nmetabolism genes\\nrs8192678 AA: 297 (67.8) GA: 119 (27.2) GG: 17 (3.9) Undetermined: 5 (1.1)\\nPPARA PPAR-/H9251/H9252 -Oxidation of fatty acids rs4253778 GG: 284 (64.8) GC: 135 (30.8) CC: 15 (3.4) Undetermined: 4 (0.9)\\nPPARD PPAR-/H9254/H9252 -Oxidation of fatty acids rs6902123 TT: 351 (80.1) TC: 78 (17.8) CC: 2 (0.5) Undetermined: 7 (1.6)\\nrs1053049 TT: 221 (50.5) CT: 188 (42.9) CC: 23 (5.3) Undetermined: 6 (1.4)rs2267668 AA: 297 (67.8) GA: 119 (27.2) GG: 17 (3.9) Undetermined: 5 (1.4)\\nAPOE Apolipoprotein E Binding and catabolism of\\nlipoproteins\\nrs7412 and rs429358 other/other: 366 (83.5) ε4/other: 59 (13.5) ε4/ ε4: 4 (0.9) Undetermined: 9 (2.1)\\nPPAR, peroxisome proliferator-activated receptor.For APOE, the two polymorphisms are linked and define six possible genotypes ( ε2/ε2, ε2/ε3, ε3/ε3, ε3/ε4, ε4/ε4, and ε2/ε4).568 POLYMORPHISMS IN OLYMPUS MARATHON RUNNERS\\nJ Appl Physiol\\x7f VOL 108 \\x7f MARCH 2010 \\x7f www.jap.org\\nDownloaded from journals.physiology.org/journal/jappl (110.138.091.074) on January 26, 2023.\\n</SENT 02_07>\\n',\n",
       "  ['1. What specific muscle- or metabolism-related genes were associated with performance in Mount Olympus marathon runners?',\n",
       "   '2. How were the athletes recruited for the study, and what were the inclusion criteria?',\n",
       "   '3. What were the three phenotypes of interest in the study, and how were they measured or calculated?',\n",
       "   '4. Which polymorphisms in the eight genes showed nominal statistical significance for any of the three phenotypes, and what were the directions of effect?',\n",
       "   '5. How did the researchers account for potential confounding factors such as population stratification in their analysis?']),\n",
       " ('<SENT 03_01>\\nwith selection based on a phenotype (participation in a very demanding endurance event) that reflects overall endurance performance.The main association analyses were performed using model-free\\nANOVA and allele-based linear regression models for continuous\\ntraits including only males.The major allele was considered as a\\nreference.For the apolipoprotein E gene (APOE), the ε4/ε4 versus\\nε4/other versus other/other genotypes were considered for the\\nANOVA.\\n</SENT 03_01>\\n<SENT 03_02>\\nAlso, a contrast of ε2 versus other alleles was considered .The analysis of the event completion time at the specific Olympus\\nMarathon was adjusted for age and year of the race (to allow for\\ndifferent event completion times due to any different weather conditions in different years).The best time ever achieved in the Olympus\\nMarathon and the kilometers per week analysis were adjusted for age\\nonly.A Shapiro-Wilk test was performed to evaluate if the phenotypes\\nwere normally distributed.\\n</SENT 03_02>\\n<SENT 03_03>\\nGiven that we observed significant deviations from normality, a natural logarithm transformation was performed on all three phenotypes.For significant associations, a pairwise interaction analysis was performed.We performed additional secondary analyses: a sex-stratified analysis where both men and women were considered and an analysis\\nwhere we considered only male athletes who stated in the enrollment\\nquestionnaire that running was their preferred sport.Moreover, in the\\nanalysis of the event completion time at the specific Olympus Marathon, whenever an athlete did not complete the race the athlete was\\nexcluded from the main analysis; however, we also performed a\\nsensitivity analysis where the timing of such dropout athletes was\\nimputed by the time of the slowest athlete finishing the race.\\n</SENT 03_03>\\n<SENT 03_04>\\nFinally,\\nin another analysis, we tested if a specific variant affected the\\nperformance of an athlete by testing athletes that finished the race\\nversus athletes who did not complete the event using a per-allele\\nmodel.Our study had 80% power to detect at /H9251/H110050.05 associations where\\nthe per-allele effect would be 1/10th the magnitude of the standard\\ndeviation of the phenotype of interest in the population, when the\\nminor allele frequency would be 10%.\\n</SENT 03_04>\\n<SENT 03_05>\\nAssociations that reach formal statistical significance at P /H110210.05\\ndo not necessarily mean that they are true.One may consider an\\napproach to correct theP values for eight genes being evaluated and\\nclaim significance for P /H110210.05/8 /H110050.006 or, if all variants are\\nconsidered independent, for P /H110210.05/11 /H110050.0045.However, such a\\ncorrection would be debatable given that these gene variants have\\nalready been proposed and have at least one other study suggesting\\nthat they may be important.To avoid this controversy, we present\\ninferences based on an alternative, Bayesian approach.\\n</SENT 03_05>\\n<SENT 03_06>\\nSpecifically,\\nwe estimated the Bayes factors according to a spike and smear model\\n(10), if the mean effect of true associations might be 0.04 in the log\\nscale (corresponding to a 9.6% difference in performance per allele,\\ne.g., completing the marathon in 456 vs. 500 min).The inverse of\\nthe Bayes factor tells how many fold the odds of an association to be\\ntrue increase based on the results of the study compared with what one\\nthought before the study.\\n</SENT 03_06>\\n<SENT 03_07>\\nThus, if the odds of an association to be true\\nwere 1:9 [1/(1 /H110019) /H110051/10 /H1100510% likely to be true] before the study\\nand the Bayes factor from the study data is 0.2 (inverse /H110051/0.2 /H110055),\\nthen the odds of the association to be true become 5:9 [5/(5 /H110019) /H11005\\n5/14 /H1100536% likely to be true].Given that different investigators may\\nhave different beliefs, we also present credibility estimates (the\\nlikelihood of an association being true) for prior evidence of 0.1 (with\\na 1:10 prior chance of the association being true), 0.001 (a more\\nconservative, sceptical approach, with a 1:1,000 chance of the association being true), and 0.000001 (a most sceptical approach, with a 1\\nin a million chance of the association being true, e.g., if the prior\\nevidence mostly suggests that the association would have an effect in\\nthe opposite direction than what is seen in the present study).\\n</SENT 03_07>\\n<SENT 03_08>\\nAll analyses were performed with STATA (version 10.0, College\\nStation, TX).P values were two tailed.Reporting of the study data\\nfollows STREGA guidelines (17).RESULTS\\nCohort characteristics.A total of 438 athletes (417 men and\\n21 women) volunteered for the study during the 2 yr. Two\\nhundred sixty-eight athletes enrolled in 2007 and 229 athletes\\ncompleted the race; in 2008, 170 athletes enrolled and 156\\nathletes finished the race.Nine athletes who did not finish the\\nrace in the year of enrolment (2007) completed the race in\\n2008.Hence, there were 44 athletes that did not ever finish the\\nOlympus marathon.Population characteristics are shown in\\nTable 2.Three hundred thirty-three athletes (76%) stated that\\ntheir preferred sport was running, whereas the other participants mentioned climbing (n /H1100518), cycling (n /H1100516), or other\\nsports (n /H1100572).\\n</SENT 03_08>\\n<SENT 03_09>\\nThe training of athletes that do not compete in\\nrunning may not be focused in the development of an endurance phenotype.Runners completed the race in faster times\\nthan the other group of athletes; however, this difference was\\nnot statistically different.Runners versus other athletes had a\\nsignificantly higher number of kilometers run per week in the\\nprevious year (mean: 56.4 vs. 37.2,P /H110210.0001) and previous\\nmonth (mean: 63.4 vs. 46.8, P /H110210.0001) but did not differ in\\nage, sex, weight, and height.Distribution of alleles and genotypes.The frequencies of the\\npolymorphisms are shown in Table 1.\\n</SENT 03_09>\\n<SENT 03_10>\\nThe undetermined genotypes were 2% or less for all polymorphisms.HWE analyses.With one exception, the distribution of\\ngenotypes for all polymorphisms did not deviate significantly\\nfrom HWE.Specifically, bradykinin B2 receptor ( BDKRB2)\\nrs1799722 deviated from the equilibrium (CC:n /H11005162, CT:n /H11005\\n173, and TT: n /H1100576, exact P value /H110050.018) for male athletes\\nTable 2.Characteristics of the athletes that participated in the study\\nAll 2007 2008 Runners Only Male Runners\\nn 438 268 170 333 316\\nAge, yr 38.4 (8.3) 38.3 (8.8) 38.6 (7.6) 38.8 (8.3) 38.9 (8.4)\\nHeight, cm 177 (6.8) 177 (6.8) 177 (6.8) 176.6 (6.9) 177.3 (6.3)\\nWeight, kg 75.2 (9.0) 75.6 (9.1) 74.5 (8.8) 74.5 (8.9) 75.6 (7.8)\\nRunners, % 333 (76) 200 (74.6) 133 (78.2) 333 (100) 316 (100)\\nDistance run, km/wk in the last month 59.4 (30.4) 57.9 (32.5) 61.7 (26.5) 63.4 (30.1) 64.04 (30.5)\\nDistance run, km/wk in the last year 51.8 (28.6) 50.5 (29.9) 53.8 (26.6) 56.4 (28.8) 57.1 (29.0)\\nHighest altitude, m 2,690 (1,231) 2,757 (1,226) 2,585 (1,236) 2,645 (1,183) 2,642 (1,180)\\nLongest distance, km 52.1 (33) 51.6 (31.6) 52.9 (35.4) 53.8 (34.3) 54.5 (34.8)\\nOM time, min 537.3 (93.8) 558 (102.5) 506.7 (68.9) 533.0 (93.5) 531.9 (93.1)\\nBest OM time, min 514.9 (92.9) 523.9 (102.5) 508.5 (75.4) 513.3 (91.9) 511.4 (91.1)\\nValue are means (SD); n, number of participants.Of the 438 athletes, 417 were men (257 men in 2007 and 160 men in 2008).OM, Olympus Marathon.569POLYMORPHISMS IN OLYMPUS MARATHON RUNNERS\\nJ Appl Physiol\\x7f VOL 108 \\x7f MARCH 2010 \\x7f www.jap.org\\nDownloaded from journals.physiology.org/journal/jappl (110.138.091.074) on January 26, 2023.\\n</SENT 03_10>\\n\\n\\n<SENT 04_01>\\nbecause of an excess of the TT genotype.The deviation was\\nalso statistically significant when we analyzed men that stated\\nthat running was their preferred sport (CC:n /H11005127, CT: n /H11005\\n125, TT: n /H1100559, exact P value /H110050.006) as well as all (both\\nmale and female) athletes who stated that running was their\\npreferred sport (CC:n /H11005133, CT: n /H11005132, TT: n /H1100563, exact\\nP value /H110050.005), always with an excess of the TT genotype.\\n</SENT 04_01>\\n<SENT 04_02>\\nHWE did not deviate significantly among athletes that did not\\ncomplete the race.Association analyses.When 417 male athletes were considered, there were generally formally no statistically significant\\nfindings observed for the polymorphisms under study regardless of whether a model-free ANOVA or an allele-based model\\nwas used, as shown in Table 3.The results were largely\\nunaltered when both men and women were considered and\\nwhen the missing values were imputed with the slowest time\\nreported (not shown).When we performed the analysis with male athletes that\\nstated that running was their preferred sport of interest (Table 4),\\nan association in one polymorphism [/H92522-adrenergic receptor\\n(ADRB2) rs1042713] reached nominal statistical significance,\\nwith faster times for the A allele for the fastest time ever ( P /H11005\\n0.01).\\n</SENT 04_02>\\n<SENT 04_03>\\nFor the specific association, we performed a sensitivity\\nanalysis where the training volume (expressed as kilometers per\\nweek the athlete ran during the 12 mo before enrollment in the study)\\nwas used as a covariate.The association remained significant for\\nrs1042713 (P /H110050.035).Also, a nominal significantP value of 0.04\\nwas found for AMP deaminase-1 (AMPD1) rs17602729 for the\\nfastest time ever using model-free ANOVA (Table 4).These two variants ( AMPD1 rs17602729 and ADRB2\\nrs1042713) reached statistical significance, both for the time at\\nthe specific event and for the fastest time ever, and with similar\\nper-allele effects for the two phenotypes when we performed\\nan analysis including both men and women.\\n</SENT 04_03>\\n<SENT 04_04>\\nForAMPD1\\nrs17602729, the P values were 0.021 and 0.03 for event\\ncompletion time and for the fastest time ever, respectively.For\\nADRB2rs1042713, the P values were 0.015 and 0.003, respectively.In a secondary analysis, a pairwise interaction analysis of the\\nsignificant SNPs did not provide any significant findings (not\\nshown in detail).Moreover, an analysis of athletes that completed the race versus athletes that did not finish the race did\\nnot yield any significant findings (data not shown).\\n</SENT 04_04>\\n<SENT 04_05>\\nTable 3.Results of the association analyses for male athletes\\nANOVA F Value P Value Allele-Based Coefficient (SE) P Value\\nACTN3 rs1815739\\nOM time 1.66 0.20 /H110020.016 (0.013) 0.20\\nBest OM time ever 0.61 0.52 /H110020.014 (0.013) 0.29\\nDistance run, km/wk 0.59 0.56 0.034 (0.046) 0.46\\nAMPD1 rs17602729\\nOM time 0.91 0.34 /H110020.019 (0.020) 0.34\\nBest OM time ever 0.74 0.48 /H110020.016 (0.020) 0.42\\nDistance run, km/wk 0.28 0.86 /H110020.001 (0.072) 0.99\\nBDKRB2 rs1799722\\nOM time 1.89 0.17 0.017 (0.012) 0.17\\nBest OM time ever 0.80 0.45 0.010 (0.012) 0.41\\nDistance run, km/wk 1.31 0.27/H110020.046 (0.044) 0.29\\nADRB2 rs1042713\\nOM time 1.55 0.21 /H110020.016 (0.013) 0.21\\nBest OM time ever 1.06 0.35 /H110020.019 (0.013) 0.16\\nDistance run, km/wk 0.01 0.99 /H110020.001 (0.047) 0.98\\nPPARGC1 rs8192678\\nOM time 0.14 0.71 /H110020.006 (0.016) 0.71\\nBest OM time ever 0.37 0.69 0.004 (0.016) 0.82\\nDistance run, km/wk 0.04 0.96 0.013 (0.057) 0.82\\nPPARA rs4253778\\nOM time 1.35 0.25 0.018 (0.016) 0.25\\nBest OM time ever 0.28 0.76 0.007 (0.016) 0.66\\nDistance run, km/wk 0.01 0.99 0.005 (0.057) 0.93\\nPPARD rs6902123\\nOM time 1.40 0.24 0.025 (0.022) 0.24\\nBest OM time ever 1.71 0.18 0.024 (0.022) 0.28\\nDistance run, km/wk 0.12 0.89/H110020.024 (0.079) 0.76\\nPPARD rs1053049\\nOM time 1.62 0.20 0.019 (0.015) 0.20\\nBest OM time ever 1.58 0.21 0.023 (0.015) 0.12\\nDistance run, km/wk 0.33 0.72 0.015 (0.054) 0.79\\nPPARD rs2267668\\nOM time 0.14 0.71 /H110020.006 (0.016) 0.71\\nBest OM time ever 0.37 0.69 0.004 (0.016) 0.82\\nDistance run, km/wk 0.04 0.96 0.013 (0.057) 0.82\\nAPOE ε4*\\nOM time 2.39 0.12 /H110020.036 (0.023) 0.12\\nBest OM time ever 0.10 0.91 /H110020.005 (0.023) 0.82\\nDistance run, km/wk 1.01 0.36 0.060 (0.082) 0.47\\nn /H11005417 male athletes total.*Analysis of ε2 genotypes did not alter the results.570 POLYMORPHISMS IN OLYMPUS MARATHON RUNNERS\\nJ Appl Physiol\\x7f VOL 108 \\x7f MARCH 2010 \\x7f www.jap.org\\nDownloaded from journals.physiology.org/journal/jappl (110.138.091.074) on January 26, 2023.\\n</SENT 04_05>\\n',\n",
       "  ['1. What specific phenotype was selected for the study, and why was it chosen?',\n",
       "   '2. How were the major and minor alleles determined for each gene variant?',\n",
       "   '3. Were there any differences in the distribution of genotypes between male and female athletes?',\n",
       "   \"4. How were the athletes' training volumes accounted for in the association analyses?\",\n",
       "   '5. Were there any significant findings in the pairwise interaction analysis of the significant SNPs?']),\n",
       " ('<SENT 05_01>\\nIn the per-allele models, for the analysis including only male\\nrunners, the estimated Bayes factor was 0.22 for ADRB2\\nrs1042713 for the fastest time ever achieved.When the analysis of the runners included both sexes, the estimated Bayes\\nfactors were 0.27 and 0.24 for the time in the specific event and\\n0.34 and 0.06 for the fastest time ever achieved forAMPD1\\nrs17602729 and ADRB2 rs1042713, respectively.These correspond, respectively, to 5-, 4-, 4-, 3-, and 16-fold increases in\\nthe odds that the association is true compared with the prior\\nbelief before the study.For a prior credibility of 10% (1:10\\nlikely to be true), the credibility of the associations was 31%\\nfor the fastest time ever achieved in the analysis of male\\nrunners.\\n</SENT 05_01>\\n<SENT 05_02>\\nFor all runners, the credibility was 27% and 29% for\\nthe time in the specific event and 23% and 62% for the fastest\\ntime ever achieved for the two polymorphisms, respectively.For a prior credibility of 1:1,000 or 1:1,000,000, the credibility\\nof the associations would be negligible (/H110210.1% for all four\\nanalyses).DISCUSSION\\nWe investigated 11 polymorphisms in 8 muscle- and/or\\nmetabolism-related genes in a large group of athletes that\\ncompeted in a special athletic event that is highly demanding\\nfor endurance.In this selected cohort, BDKRB2 rs1799722\\ndeviated significantly from HWE with an excess of TT individuals, suggesting that this genotype may be associated with\\nendurance performance in general.\\n</SENT 05_02>\\n<SENT 05_03>\\nMoreover, we found associations with nominal statistical significance forADRB2\\nrs1042713 for the best time ever among male athletes where\\nrunning is their preferred sport and associations forAMPD1\\nrs17602729 and ADRB2 rs1042713 for the time of the specific\\nevent and best time ever among all (both male and female)\\nrunners.Given that these genes and variants have clear functional roles and have also been proposed as potentially important by previous studies, modest signals should not be dismissed, if they also agree with other prior evidence.\\n</SENT 05_03>\\n<SENT 05_04>\\nOur findings are in agreement with prior evidence for\\nADRB2 rs1042713 (Arg16Gly).ADRB2 rs1042713 encodes\\nthe /H92522-adrenergic receptor that is primarily responsible for\\nincreases in bronchodilation, ventricular function, and vasodilation, all of which have direct implications on cardiovascular\\nand pulmonary responses to exercise (36).In a case-control\\nstudy (n /H11005313 endurance athletes competing in 8 endurance\\nsporting disciplines vs. n /H11005297 sedentary controls), the A\\nTable 4.Results of the association analyses for habitual male runners only\\nANOVA F Value P Value Allele-Based Coefficient (SE) P Value\\nACTN3 rs1815739\\nOM time 1.42 0.23 /H110020.017 (0.014) 0.24\\nBest OM time ever 1.10 0.32 /H110020.018 (0.014) 0.22\\nDistance run, km/wk 0.75 0.87 0.017 (0.048) 0.72\\nAMPD1 rs17602729\\nOM time 3.61 0.06 /H110020.044 (0.023) 0.06\\nBest OM time ever 3.14 0.04 /H110020.041 (0.024) 0.08\\nDistance run, km/wk 0.75 0.86 0.037 (0.079) 0.64\\nBDKRB2 rs1799722\\nOM time 0.51 0.48 0.009 (0.013) 0.48\\nBest OM time ever 1.92 0.15 0.007 (0.014) 0.63\\nDistance run, km/wk 0.78 0.46/H110020.069 (0.045) 0.13\\nADRB2 rs1042713\\nOM time 3.38 0.07 /H110020.026 (0.014) 0.07\\nBest OM time ever 5.92 0.01 /H110020.037 (0.015) 0.01\\nDistance run, km/wk 0.88 0.35 0.048 (0.049) 0.33\\nPPARGC1 rs8192678\\nOM time 0.22 0.64 0.009 (0.018) 0.64\\nBest OM time ever 1.67 0.19 0.022 (0.019) 0.24\\nDistance run, km/wk 0.04 0.97 0.016 (0.062) 0.80\\nPPARA rs4253778\\nOM time 0.79 0.38 0.015 (0.017) 0.38\\nBest OM time ever 0.21 0.81 0.008 (0.017) 0.66\\nDistance run, km/wk 0.02 0.98 0.014 (0.058) 0.98\\nPPARD rs6902123\\nOM time 1.34 0.25 0.027 (0.023) 0.25\\nBest OM time ever 1.12 0.33 0.021 (0.024) 0.37\\nDistance run, km/wk 0.47 0.62/H110020.005 (0.079) 0.95\\nPPARD rs1053049\\nOM time 2.87 0.09 0.028 (0.017) 0.09\\nBest OM time ever 3.13 0.05 0.031 (0.017) 0.07\\nDistance run, km/wk 0.27 0.77 0.023 (0.057) 0.69\\nPPARD rs2267668\\nOM time 0.22 0.64 0.029 (0.018) 0.64\\nBest OM time ever 1.67 0.19 0.022 (0.018) 0.24\\nDistance run, km/wk 0.03 0.97 0.016 (0.062) 0.79\\nAPOE ε4*\\nOM time 2.20 0.14 /H110020.037 (0.025) 0.14\\nBest OM time ever 0.48 0.62 /H110020.021 (0.026) 0.42\\nDistance run, km/wk 0.88 0.42 0.070 (0.084) 0.41\\nn /H11005316 habitual male runners total.*Analysis of ε2 genotypes did not alter the results.571POLYMORPHISMS IN OLYMPUS MARATHON RUNNERS\\nJ Appl Physiol\\x7f VOL 108 \\x7f MARCH 2010 \\x7f www.jap.org\\nDownloaded from journals.physiology.org/journal/jappl (110.138.091.074) on January 26, 2023.\\n</SENT 05_04>\\n\\n\\n<SENT 06_01>\\n(Arg16) allele was significantly more prevalent in the endurance athletes (45).ARDB2 regulates the cardiopulmonary response to exercise (36), and this polymorphism has been\\ninvestigated in many phenotypes, with some evidence oflower\\nmean arterial blood pressure at rest, during and after exercise for\\nArg16 homozygotes (37), decreased exercise performance in\\npatients with heart failure with Gly16 (42), and decreased thermogenic responses to/H92522-adrenergic stimulation with Arg16 (26).\\n</SENT 06_01>\\n<SENT 06_02>\\nAMPD1 (rs17602729), which functions in skeletal muscle\\nmetabolism (salvage of adenine nucleotides) and is involved in\\nthe regulation of muscle glycolysis during rigorous exercise\\n(29), was evaluated in a case-control study (n /H11005104 endurance\\nathletes vs. 100 controls).The minor (A) allele was actually\\nsignificantly less common in endurance athletes than in controls, but endurance indexes such as V˙O2max, ventilatory threshold, and respiratory compensation threshold were not significantly different between athlete carriers and noncarriers of the\\nminor allele (29).\\n</SENT 06_02>\\n<SENT 06_03>\\nFurthermore,AMPD1 rs17602729 was evaluated in a study assessing physiological responses in 503\\nindividuals undergoing a 20-wk training regime, and the minor\\nallele had smaller ventilatory training responses pertaining to\\nendurance (28).Therefore, our results are in the opposite\\ndirection for this polymorphism than what prior studies had\\nsuggested.Discrepancies may be due to different study design,\\nsetting, types of athletes, and phenotype definitions, but most\\nlikely they simply represent false positives in opposite directions, and this polymorphism is not credibly associated with\\nendurance performance.Incidentally, there is a case report (19)\\non a world-class athlete with exceptionally favorable laboratory indexes for endurance performance (V˙O2max) despite car rying the A allele (19), which further documents that these\\ngenetic effects, even if present, are subtle and do not doom\\none’s athletic performance.\\n</SENT 06_03>\\n<SENT 06_04>\\nVariation at the BDKRB2 (rs1799722) gene locus, which\\nencodes a receptor for bradykinin and is implicated in the\\nincrease of skeletal muscle glucose uptake during exercise\\n(34), has also been associated with endurance performance in\\nprevious studies.In one study (43), athletes (n /H1100581) competing in longer distance events had a significantly higher frequency of the/H110029 allele, a 9-bp insertion/deletion polymorphism in exon 1.In a study of 453 athletes who completed the\\nSouth Africa Ironman Triathlon, the athletes had a significantly\\nhigher frequency of the/H110029//H110029 genotype compared with the\\ncontrols (n /H11005203) (34).In that same study (34), the /H110029 allele\\nwas not significantly different in three groups of athletes who\\ncompleted the event in fast, medium, and slow times (34).\\n</SENT 06_04>\\n<SENT 06_05>\\nThe\\n/H110029 allele has been demonstrated to result in increased transcription rates of the gene (6).In our study, we focused only on\\nSNPs rather than insertion/deletion polymorphisms, and we\\ninstead genotyped the promoter variant rs1799722 (also known\\nas/H1100258C/T), which is also known to have a functional impact\\non the gene, with increased transcription rates for the T allele\\nin luciferase experiments (15).We found results consistent\\nwith previous evidence: the high transcription allele was overrepresented in this group of endurance athletes and, even more\\nso, among those who are habitual runners.The same promoter\\nvariant has also been associated with hypertension, left ventricular hypertrophy, and baroreflex sensitivity (7, 22).\\n</SENT 06_05>\\n<SENT 06_06>\\nAll of the other polymorphisms that showed no nominally\\nsignificant signals in our study have been examined in the past\\nin at least one other study, but the evidence was often inconclusive or may be stronger for associations with phenotypes\\nthat have only indirect bearing on athletic endurance performance.Moreover, we only examined specific polymorphisms\\nfor each one of these genes, so we cannot exclude that other\\nvariants in the same gene locus may have an effect on athletic\\nperformance in particular populations.\\n</SENT 06_06>\\n<SENT 06_07>\\nFinally, our study was\\npowered to detect modest effects, but subtle effects for variants\\nwith minor allele frequencies/H1102110% may have been missed.Specifically, in /H92513-actinin ( ACTN3; rs1815739), which\\ncodes for the synthesis of actinin-3 (a major skeletal muscle\\nconstituent) and is implicated in fast contraction ability by\\nmuscles (46), the T allele was reported at somewhat higher\\nfrequencies in endurance athletes than controls in two studies\\n(25, 46), but the differences were not formally significant.\\n</SENT 06_07>\\n<SENT 06_08>\\nMoreover, rs1815739 was not associated with a continuous\\nendurance outcome (based on competitor finish time) during\\nthe South Africa Éronman Triathlon competition (n /H11005457\\nathletes vs. 143 controls) (33), and no differences were observed in a case-control study (18) (n /H1100552 Olympic-level\\nrunners vs. 50 professional cyclists vs. 123 controls) between\\ngroups and when laboratory endurance performance indexes\\nwere evaluated.PPAR-/H9251(PPARA) rs4253778 is an intronic variant that is\\ninvolved in the regulation of liver, heart, and skeletal muscle\\nlipid metabolism as well as glucose homeostasis (39).\\n</SENT 06_08>\\n<SENT 06_09>\\nThis\\nvariant was investigated in a multisport case-control study (2)\\n(n /H11005786 Russian athletes vs. 1,242 controls) stratified by\\nperformance (endurance, sprint, and mixed) in which endurance athletes had significantly higher frequency of the G allele\\nthan controls.Thesame polymorphism was found to be associated with triglyceride and apolipoprotein CIII levels in Africadescended people but not in Caucasians (35).It is unknown\\nwhether ancestry-specific genetic effects (11) may also be\\nimportant in athletic performance.PPARD (rs1053049, rs6902123, and rs2267668) is involved\\nin fatty acid /H9252-oxidation, glucose utilization, mitochondrial\\nbiogenesis, angiogenesis, and muscle fiber type (39), and\\nPPAR-/H9253coactivator-1/H9251(PPARGC1A; rs8192678) regulates\\ngenes involved in energy metabolism and is associated with\\nmitochondrial biogenesis and skeletal muscle fiber type conversion (20).\\n</SENT 06_09>\\n<SENT 06_10>\\nBoth genes seem to have independent effects on\\nthe effectiveness of aerobic exercise training to increase aerobic physical fitness and insulin sensitivity (39).Furthermore, a\\nwhole body MRI study (38) found differential changes after\\nlifestyle intervention in overall adiposity, hepatic fat storage,\\nand relative muscle mass for the rs1053049, rs6902123, and\\nrs2267668 variants ofPPARD (38).Moreover, in a multisport\\ncase-control study (1) ( n /H110051,256 athletes vs. 610 controls),\\nPPARD allelic differences were seen in endurance athletes but\\nnot in controls (1), and another case-control study (20) found\\nthat the minor allele ofPPARGC1A rs8192678 was significantly more common in unfit controls ( n /H11005100) than in\\nendurance athletes ( n /H11005104).However, it is unclear whether\\nsuch associations would result also in noticeable differences in\\nperformance among endurance athletes besides differentiating\\nathletes from controls.\\n</SENT 06_10>\\n<SENT 06_11>\\nSimilarly, for apolipoprotein E (APOE;\\nrs7412 and rs429358), which facilitates triglyceride clearance\\nby mediating lipoprotein binding to hepatic receptors, thus\\ncontributing to the variability in individual response to exercise\\ntraining (41), theε4 allele in previous studies (8, 41) was found\\nto be associated with better physiological responses after ex572 POLYMORPHISMS IN OLYMPUS MARATHON RUNNERS\\nJ Appl Physiol\\x7f VOL 108 \\x7f MARCH 2010 \\x7f www.jap.org\\nDownloaded from journals.physiology.org/journal/jappl (110.138.091.074) on January 26, 2023.\\n</SENT 06_11>\\n',\n",
       "  ['1. What is the functional role of ADRB2 rs1042713 in the human body?',\n",
       "   '2. How does the AMPD1 rs17602729 polymorphism affect skeletal muscle metabolism during rigorous exercise?',\n",
       "   \"3. What are the potential reasons for the discrepancies between your study's results and previous studies regarding AMPD1 rs17602729 and its association with endurance performance?\",\n",
       "   '4. Can you explain the functional impact of the BDKRB2 rs1799722 promoter variant on the gene?',\n",
       "   '5. How might ancestry-specific genetic effects influence the association between certain polymorphisms and athletic performance?']),\n",
       " ('<SENT 07_01>\\nercise training, but this may not necessarily translate to better\\nendurance performance.In all, more extensive replication\\nwould be essential (10, 15) before making claims that any of\\nthese variants have a substantial effect specifically on endurance performance.Furthermore, three case-control studies (30, 32, 23) examined the representation in athletes and controls of seven polymorphisms, three of which (ACTN3 rs1815739, AMPD1\\nrs17602729, and PPARGC1A rs8192678) were also investigated in our study.In the first study (3) ( n /H1100546 endurance\\nathletes vs. 123 controls), a point was given for each “favorable\\nallele,” and the overall mean total score was higher in athletes\\nthan in controls.\\n</SENT 07_01>\\n<SENT 07_02>\\nThe second study (32) (n /H1100539 world-level\\nathletes vs. 15 national-level athletes) found no differences in\\ngenetic profiles between the two groups of athletes.The third\\nstudy (23) compared many groups (n /H1100550 professional cyclists vs. 52 Olympic-level runners vs. 39 world-level rowers\\nvs. 123 controls), and no significant differences among the\\ngroups were found.Given the relatively limited sample size, it\\nis difficult to draw conclusive inferences.Some limitations should be discussed.\\n</SENT 07_02>\\n<SENT 07_03>\\nWe used a cohort\\ndesign and did not make a comparison against matched nonathlete controls from the general population.Therefore, we\\nperformed two analyses for association: a first crude screening\\nbased on the Hardy-Weinberg test and another based on association for continuous outcomes of endurance performance\\nwithin the cohort of athletes.Hardy-Weinberg testing has low\\npower as an association test, and subtle associations may have\\nbeen missed in this screening.The analysis of comparative\\nperformance is based on a continuous outcome, and it has good\\npower to detect modest differences, but most genetic effects for\\ncommon variants represent effects with very subtle magnitude\\n(13), andsuch effects could still have been missed.\\n</SENT 07_03>\\n<SENT 07_04>\\nConversely, nominally significant associations should be interpreted with caution since they may still represent false\\npositives, and we encourage further replication of our findings in additional studies and settings(14).Third, genes and\\npolymorphisms were selected based on considerable prior\\nknowledge on the function of the proteins encoded by these\\ngenes and also some prior epidemiological evidence.With the\\nadvent of high-throughput genotyping, the field of athletic\\nperformance should also consider the conduct of agnostic\\ngenome-wide association studies (21), although for variants\\nemerging from such studies it is often difficult to establish their\\nphysiological functional role (10).Overall, human physical performance is a multifactorial phenotype in which numerous genetic and environmental factors have\\njoint effects.Athletic performance is very much influenced by\\ntraining programmes and opportunities, residing altitude, recovery, nutrition, sporting equipment, and psychological factors during competition.\\n</SENT 07_04>\\n<SENT 07_05>\\nSome of these factors may also be\\nreciprocally determined by genetic input to some extent, e.g.,\\nthe propensity to exercise may also be a genetic trait (38).In\\nour study, we also tried to perform analyses limited to athletes\\nthat had a strong preference for running, but there was still\\ndiversity in some other exposures, e.g., the amount of training\\nfor these athletes.Dissecting the complex effects on athletic\\nexcellence will require large-scale studies with meticulous\\nmeasurements and replication of proposed associations in diverse datasets.ACKNOWLEDGMENTS\\nThe authors thank F. Kavvoura, D. Panagiotopoulos, M. Matziaris, T.\\nThanopoulos, and K. Gousis for the invaluable assistance during data collection.\\n</SENT 07_05>\\n<SENT 07_06>\\nDISCLOSURES\\nNo conflicts of interest are declared by the author(s).REFERENCES\\n1.Ahmetov II, Astranenkova IV, Rogozkin VA.Association of PPARD\\ngene polymorphism with human physical performance.Mol Biol (Mosk)\\n41: 852–857, 2007.2.Ahmetov II, Mozhayskaya IA, Flavell M, Astratenkova IV, Komkova\\nAI, Lyubaeva EV, Tarakin PP, Shenkman BS, Vdovina AB, Netreba\\nAI, Popov DV, Vinogradova OL, Montgomery HE, Rogozkin VA.\\nPPARalpha gene variation and physical performance in Russian athletes.Eur J Appl Physiol97: 103–108, 2006.3.\\n</SENT 07_06>\\n<SENT 07_07>\\nBouchard C, Daw EW, Rice T, Perusse L, Gagnon J, Province MA,\\nLeon AS, Rao DC, Skinner JS, Wilmore JH.Familial resemblance for\\nV˙O2max in the sedentary state: the HERITAGE Family Study.Med Sci\\nSports Exerc 30: 252–258, 1998.4.Bouchard C, An P, Rice T, Skinner JS, Wilmore JH, Gagnon J,\\nPerusse L, Leon AS, Rao DC.Familial aggregation of V˙O2max response\\nto exercise training: results from the HERITAGE Family Study.J Appl\\nPhysiol 87, 1003–1008, 1999.5.Bray MS, Hagberg JM, Pérusse L, Rankinen T, Roth SM, Wolfarth B,\\nBouchard C. The human gene map for performance and health-related\\nfitness phenotypes: the 2006 –2007 update.Med Sci Sports Exerc 41:\\n35–73, 2009.\\n</SENT 07_07>\\n<SENT 07_08>\\n6.Braun A, Kammerer S, Maier E, Böhme E, Roscher AA.Polymorphisms in the gene for the human B 2-bradykinin receptor.New tools in\\nassessing a genetic risk for bradykinin-associated diseases.Immunopharmacology 33: 32–35, 1996.7.Fu Y, Katsuya T, Matsuo A, Yamamoto K, Akasaka H, Takami Y,\\nIwashima Y, Sugimoto K, Ishikawa K, Ohishi M, Rakugi H, Ogihara\\nT.Relationship of bradykinin B 2 receptor gene polymorphism with es sential hypertension and left ventricular hypertrophy.Hypertens Res 27:\\n933–938, 2004.8.\\n</SENT 07_08>\\n<SENT 07_09>\\nHagberg JM, Ferrell RE, Katzel LI, Dengel DR, Sorkin JD, Goldberg\\nAP.Apolipoprotein E genotype and exercise training-induced increases in\\nplasma high-density lipoprotein (HDL)- and HDL2-cholesterol levels in\\noverweight men.Metabolism 48: 943–945, 1999.9.Hirschhorn JN, Lohmueller K, Byrne E, Hirschhorn K.A comprehensive review of genetic association studies.Genet Med 4: 45–61, 2002.10.Ioannidis JP, Thomas G, Daly MJ.Validating, augmenting and refining\\ngenome-wide association signals.Nat Rev Genet10: 318 –329, 2009.\\n</SENT 07_09>\\n<SENT 07_10>\\n11.Ioannidis JP, Ntzani EE, Trikalinos TA.“Racial” differences in genetic\\neffects for complex diseases.Nat Genet 36: 1312–1318, 2004.12.Ioannidis JP.Calibration of credibility of agnostic genome-wide associations.Am J Med Genet B Neuropsychiatr Genet147B: 964 –972, 2008.13.Ioannidis JP, Trikalinos TA, Khoury MJ.Implications of small effect\\nsizes of individual genetic variants on the design and interpretation of\\ngenetic association studies of complex diseases.Am J Epidemiol 164:\\n609 –614, 2006.\\n</SENT 07_10>\\n<SENT 07_11>\\n14.Ioannidis JP, Ntzani EE, Trikalinos TA, Contopoulos-Ioannidis DG.Replication validity of genetic association studies.Nat Genet 29: 306 –\\n309, 2001.15.Kammerer S, Braun A, Arnold N, Roscher AA.The human bradykinin\\nB2 receptor gene: full length cDNA, genomic organization and identifi cation of the regulatory region.Biochem Biophys Res Commun 211:\\n226 –233, 1995.16.Li M, Li C.Assessing departure from Hardy-Weinberg equilibrium in the\\npresence of disease association.Genet Epidemiol 32: 589 –599, 2008.17.Little J, Higgins JP, Ioannidis JP, Moher D, Gagnon F, von Elm E,\\nKhoury MJ, Cohen B, Davey-Smith G, Grimshaw J, Scheet P, Gwinn\\nM, Williamson RE, Zou GY, Hutchings K, Johnson CY, Tait V, Wiens\\nM, Golding J, van Duijn C, McLaughlin J, Paterson A, Wells G,\\nFortier I, Freedman M, Zecevic M, King R, Infante-Rivard C, Stewart\\nA, Birkett N.Strengthening the reporting of genetic association studies\\n(STREGA)–an extension of the STROBE statement.Eur J Clin Invest39:\\n247–266, 2009.18.Lucia A, Gómez-Gallego F, Santiago C, Bandrés F, Earnest C,\\nRabadán M, Alonso JM, Hoyos J, Córdova A, Villa G, Foster C.\\n573POLYMORPHISMS IN OLYMPUS MARATHON RUNNERS\\nJ Appl Physiol\\x7f VOL 108 \\x7f MARCH 2010 \\x7f www.jap.org\\nDownloaded from journals.physiology.org/journal/jappl (110.138.091.074) on January 26, 2023.\\n</SENT 07_11>\\n\\n\\n<SENT 08_01>\\nACTN3 genotype in professional endurance cyclists.Int J Sports Med27:\\n880 –884, 2006.19.Lucia A, Martin MA, Esteve-Lanao J, San Juan AF, Rubio JC, Oliván\\nJ, Arenas J.C34T mutation of the AMPD1 gene in an elite white runner.Br J Sports Med40: e7, 2006.20.Lucia A, Gómez-Gallego F, Barroso I, Rabadán M, Bandrés F, San\\nJuan AF, Chicharro JL, Ekelund U, Brage S, Earnest CP, Wareham\\nNJ, Franks PW.PPARGC1A genotype (Gly482Ser) predicts exceptional\\nendurance capacity in European men.J Appl Physiol99: 344 –348, 2005.\\n</SENT 08_01>\\n<SENT 08_02>\\n21.McCarthy MI, Abecasis GR, Cardon LR, Goldstein DB, Little J,\\nIoannidis JP, Hirschhorn JN.Genome-wide association studies for\\ncomplex traits: consensus, uncertainty and challenges.Nat Rev Genet9:\\n356 –369, 2008.22.Milan A, Mulatero P, Williams TA, Carra R, Schiavone D, Martuzzi\\nR, Rabbia F, Veglio F.Bradykinin B2 receptor gene (/H1100258T/C) polymorphism influences baroreflex sensitivity in never-treated hypertensive patients.J Hypertens 23: 63–69, 2005.23.Muniesa CA, González-Freire M, Santiago C, Lao JI, Buxens A,\\nRubio JC, Martín MA, Arenas J, Gomez-Gallego F, Lucia A.Worldclass performance in lightweight rowing: Is it genetically influenced?\\n</SENT 08_02>\\n<SENT 08_03>\\nA\\ncomparison with cyclists, runners and non-athletes.Br J Sports Med.I n\\npress.24.Myburgh KH.What makes an endurance athlete world-class?Not simply\\na physiological conundrum.Comp Biochem Physiol A Mol Integr Physiol\\n136: 171–190, 2003.25.Niemi AK, Majamaa K.DNA and ACTN3 genotypes in Finnish elite\\nendurance and sprint athletes.Eur J Hum Genet13: 965–969, 2005.26.Oomen JM, van Rossum CT, Hoebee B, Saris WH, van Baak MA./H92522-Adrenergic receptor polymorphisms and salbutamol-stimulated energy\\nexpenditure.J Clin Endocrinol Metab90: 2301–2307, 2005.27.Rankinen T, Bray MS, Hagberg JM, Pérusse L, Roth SM, Wolfarth B,\\nBouchard C. The human gene map for performance and health-related\\nfitness phenotypes: the 2005 update.Med Sci Sports Exerc38: 1863–1888,\\n2006.\\n</SENT 08_03>\\n<SENT 08_04>\\n28.Rico-Sanz J, Rankinen T, Joanisse DR, Leon AS, Skinner JS, Wilmore JH, Rao DC, Bouchard C.Associations between cardiorespiratory\\nresponses to exercise and the C34T AMPD1 gene polymorphism in the\\nHERITAGE Family Study.Physiol Genomics 14: 161–166, 2003.29.Rubio JC, Martín MA, Rabadán M, Gómez-Gallego F, San Juan AF,\\nAlonso JM, Chicharro JL, Pérez M, Arenas J, Lucia A.Frequency of\\nthe C34T mutation of the AMPD1 gene in world-class endurance athletes:\\ndoes this mutation impair performance?J Appl Physiol98: 2108 –2112,\\n2005.\\n</SENT 08_04>\\n<SENT 08_05>\\n30.Ruiz JR, Gómez-Gallego F, Santiago C, González-Freire M, Verde Z,\\nFoster C, Lucia A.Is there an optimum endurance polygenic profile?J\\nPhysiol 587: 1527–1534, 2009.31.Salanti G, Amountza G, Ntzani EE, Ioannidis JP.Hardy-Weinberg\\nequilibrium in genetic association studies: an empirical evaluation of\\nreporting, deviations, and power.Eur J Hum Genet13: 840 –848, 2005.32.Santiago C, Ruiz JR, Muniesa CA, González-Freire M, GómezGallego F, Lucia A.Does the polygenic profile determine the potential for\\nbecoming a world-class athlete?Insights from the sport of rowing.\\n</SENT 08_05>\\n<SENT 08_06>\\nScand\\nJ Med Sci Sports.In press.33.Saunders CJ, September AV, Xenophontos SL, Cariolou MA, Anastassiades LC, Noakes TD, Collins M.No association of the ACTN3 gene\\nR577X polymorphism with endurance performance in Ironman Triathlons.Ann Hum Genet71: 777–781, 2007.34.Saunders CJ, Xenophontos SL, Cariolou MA, Anastassiades LC,\\nNoakes TD, Collins M.The bradykinin beta 2 receptor (BDKRB2) and\\nendothelial nitric oxide synthase 3 (NOS3) genes and endurance performance during Ironman Triathlons.Hum Mol Genet15: 979 –987, 2006.35.\\n</SENT 08_06>\\n<SENT 08_07>\\nShin MJ, Kanaya AM, Krauss RM.Polymorphisms in the peroxisome\\nproliferator activated receptor alpha gene are associated with levels of\\napolipoprotein CIII and triglyceride in African-Americans but not Caucasians.Atherosclerosis 198: 313–319, 2008.36.Snyder EM, Johnson BD, Joyner MJ.Genetics of /H92522-adrenergic receptors and the cardiopulmonary response to exercise.Exerc Sport Sci Rev36:\\n98 –105, 2008.37.Snyder EM, Beck KC, Dietz NM, Eisenach JH, Joyner MJ, Turner\\nST, Johnson BD.Arg16Gly polymorphism of the /H92522-adrenergic receptor\\nis associated with differences in cardiovascular function at rest and during\\nexercise in humans.J Physiol 571: 121–130, 2006.38.Stubbe JH, Boomsma DI, Vink JM, Cornes BK, Martin NG, Skytthe\\nA, Kyvik KO, Rose RJ, Kujala UM, Kaprio J, Harris JR, Pedersen\\nNL, Hunkin J, Spector TD, de Geus EJ.Genetic influences on exercise\\nparticipation in 37,051 twin pairs from seven countries.\\n</SENT 08_07>\\n<SENT 08_08>\\nPLoS ONE1: e22,\\n2006.39.Stefan N, Thamer C, Staiger H, Machicao F, Machann J, Schick F,\\nVenter C, Niess A, Laakso M, Fritsche A, Häring HU.Genetic\\nvariations in PPARD and PPARGC1A determine mitochondrial function\\nand change in aerobic physical fitness and insulin sensitivity during\\nlifestyle intervention.J Clin Endocrinol Metab92: 1827–1833, 2007.40.Thamer C, Machann J, Stefan N, Schäfer SA, Machicao F, Staiger H,\\nLaakso M, Böttcher M, Claussen C, Schick F, Fritsche A, Haring HU.Variations in PPARD determine the change in body composition during\\nlifestyle intervention: a whole-body magnetic resonance study.J Clin\\nEndocrinol Metab 93: 1497–1500, 2008.41.\\n</SENT 08_08>\\n<SENT 08_09>\\nThompson PD, Tsongalis GJ, Seip RL, Bilbie C, Miles M, Zoeller R,\\nVisich P, Gordon P, Angelopoulos TJ, Pescatello L, Bausserman L,\\nMoyna N.Apolipoprotein E genotype and changes in serum lipids and\\nmaximal oxygen uptake with exercise training.Metabolism 53: 193–202,\\n2004.42.Wagoner LE, Craft LL, Singh B, Suresh DP, Zengel PW, McGuire N,\\nAbraham WT, Chenier TC, Dorn GW 2nd, Liggett SB.Polymorphisms\\nof the /H92522-adrenergic receptor determine exercise capacity in patients with\\nheart failure.\\n</SENT 08_09>\\n<SENT 08_10>\\nCirc Res 86: 834 –840, 2000.43.Williams AG, Dhamrait SS, Wootton PT, Day SH, Hawe E, Payne JR,\\nMyerson SG, World M, Budgett R, Humphries SE, Montgomery HE.Bradykinin receptor gene variant and human physical performance.J Appl\\nPhysiol 96: 938 –942, 2004.44.Williams AG, Folland JP.Similarity of polygenic profiles limits the\\npotential for elite human physical performance.J Physiol 586: 113–121,\\n2008.45.Wolfarth B, Rankinen T, Mühlbauer S, Scherr J, Boulay MR, Pérusse\\nL, Rauramaa R, Bouchard C. Association between a /H92522-adrenergic\\nreceptor polymorphism and elite endurance performance.Metabolism 56:\\n1649 –1651, 2007.46.Yang N, MacArthur DG, Gulbin JP, Hahn AG, Beggs AH, Easteal S,\\nNorth K. ACTN3 genotype is associated with human elite athletic\\nperformance.Am J Hum Genet73: 627–631, 2003.574 POLYMORPHISMS IN OLYMPUS MARATHON RUNNERS\\nJ Appl Physiol\\x7f VOL 108 \\x7f MARCH 2010 \\x7f www.jap.org\\nDownloaded from journals.physiology.org/journal/jappl (110.138.091.074) on January 26, 2023.\\n</SENT 08_10>\\n',\n",
       "  ['1. What specific exercise training methods were studied in relation to endurance performance?',\n",
       "   '2. How do the genetic variants investigated in the study potentially affect endurance performance?',\n",
       "   '3. What were the sample sizes for the three case-control studies mentioned in the context?',\n",
       "   '4. What were the limitations of using a cohort design without a comparison against matched non-athlete controls?',\n",
       "   \"5. How does the Hardy-Weinberg test's low power as an association test impact the study's findings?\"]),\n",
       " (\"<SENT 09_01>\\nORIGINAL RESEARCH ARTICLE\\nThe interaction betweenFTO rs9939609 and physical\\nactivity is associated with a 2-fold reduction in the risk\\nof obesity in Korean population\\nHye-Won Cho1 | Hyun-Seok Jin2 | Yong-Bin Eom1,3\\n1Department of Medical Sciences, College\\nof Medical Sciences, Soonchunhyang\\nUniversity, Asan, Republic of Korea\\n2Department of Biomedical Laboratory\\nScience, College of Life and Health\\nSciences, Hoseo University, Asan,\\nRepublic of Korea\\n3Department of Biomedical Laboratory\\nScience, College of Medical Sciences,\\nSoonchunhyang University, Asan,\\nRepublic of Korea\\nCorrespondence\\nYong-Bin Eom, Department of Biomedical\\nLaboratory Science, College of Medical\\nSciences, Soonchunhyang University,\\n22 Soonchunhyang-ro, Asan-si,\\nChungcheongnam-do 31538,\\nRepublic of Korea.\\n</SENT 09_01>\\n<SENT 09_02>\\nEmail: omnibin@sch.ac.kr\\nFunding information\\nNational Research Foundation of Korea,\\nGrant/Award Number: NRF2017R1D1A1B03032960; Soonchunhyang\\nUniversity, Grant/Award Number: SCH20200312\\nAbstract\\nObjective: FTO (fat mass and obesity-associated) gene is a well-known genetic\\nrisk factor for obesity.We investigated whether physical activity modulates the\\neffect ofFTO rs9939609 on obesity in Korean population.Methods: The study analyzed the correlation between physical activity and\\nobesity in 8840 individuals representing the Korea Association Resource\\n(KARE).The association between obesity-related traits and single-nucleotide\\npolymorphisms (SNPs) was assessed using linear regression models.\\n</SENT 09_02>\\n<SENT 09_03>\\nPhysical\\nactivity was defined as 3 hours or more of daily intense activity.Results: Participants carrying rs9939609 (AT+AA) genotypes showed higher\\nBMI compared with those carrying the wild-type (TT) homozygote.The\\nhighest significant association was observed between obesity-related traits\\n(ß = .334,P value = 1.76× 10−6).FTO rs9939609 (AT+AA) increased the risk\\nof obesity (OR = 1.42, CI [1.13-1.79]), which was correlated with BMI correlations.However, active exercise by subjects carrying the same genotype reduced\\nthe risk of obesity by nearly 2-fold (OR = 0.62, CI [0.25-0.84]).In contrast, TT\\ngenotype was not statistically significant in reducing the risk of obesity in the\\nactive exercise group.Conclusions: Our results support a previous finding correlating FTO and\\nobesity-related traits and suggest that the interaction with genetic variation\\nand physical activity is an important risk factor for obesity.1 | INTRODUCTION\\nThe World Health Organization (WHO) reported that the\\nprevalence of obesity worldwide has nearly tripled since\\n1975, and more than 1.9 billion adults aged 18 years and\\nover (39% of men and 40% of women) were overweight in\\n2016.\\n</SENT 09_03>\\n<SENT 09_04>\\nAccording to the key facts reported by WHO, most\\nof the world's population lives in countries where overweight and obesity kill more people than underweight\\n(World Health Organization, 2018).Obesity and\\noverweight are attributed to an increase in physical inactivity due to the increasingly sedentary nature of many\\nforms of work, changing modes of transportation, and\\nincreasing urbanization.Obesity is becoming a global\\npublic health concern as a result of low physical activity\\nand genetic predisposition in individuals of all ages and\\nethnic groups, as it contributes to the increasing burden\\nof cardiovascular disease and type 2 diabetes (Kim\\net al., 2016; Ogden et al., 2006).Genetic variants of the FTO (fat mass and obesity\\nassociated) gene, located on chromosome 16q12.2, are\\ncorrelated with obesity and higher body mass indexHye-Won Cho and Hyun-Seok Jin contributed equally to this study.Received: 12 March 2020 Revised: 29 June 2020 Accepted: 29 July 2020\\nDOI: 10.1002/ajhb.23489\\nAm J Hum Biol.2020;e23489.wileyonlinelibrary.com/journal/ajhb © 2020 Wiley Periodicals LLC 1o f1 0\\nhttps://doi.org/10.1002/ajhb.23489\\n</SENT 09_04>\\n\\n\\n<SENT 10_01>\\n(BMI) in humans (Cornes et al., 2009; Frayling\\net al., 2007; Koike, Richards, Wong, & Hardy, 2018;\\nLoos & Yeo, 2014).Heritability estimates for obesityrelated traits indicate that a substantial proportion of\\nphenotypic variation still needs to be explained\\n(O'Rahilly & Farooqi, 2008).It is proposed that the interaction between FTO gene and lifestyle factors may contribute to the variability of those traits, since many\\nstudies have shown controversial results or negligible\\ninteractive effects.Previous studies have demonstrated a link between\\nFTO and higher BMI in humans.\\n</SENT 10_01>\\n<SENT 10_02>\\nIndividuals with first\\nintron rs9939609 FTO obesity-risk allele (homozygous = AA and heterozygous = AT) were found to\\nexhibit higher BMI than those with wild-type alleles\\n(TT) (Qi et al., 2014; Saber-Ayad et al., 2019).Recent\\nstudies reported an interaction betweenFTO genotypes,\\nphysical inactivity, and BMI (Leonska-Duniec\\net al., 2018; Liaw, Liaw, & Lan, 2019).Higher levels of\\nphysical activity reduced the likelihood of being overweight or obese (Bernstein, Costanza, & Morabia, 2004;\\nKing et al., 2001; Payne, Cahill, Sun, Loredo-Osti, &\\nAbarin, 2014; Yoshioka et al., 2005).However, none of\\nthese studies considered the effect of genetic factors as\\nmodifiers.\\n</SENT 10_02>\\n<SENT 10_03>\\nPhysical activity has been thought to exacerbate the effect of theFTO gene (Andreasen et al., 2008).The effect of genetic susceptibility on obesity caused by\\nFTO variants can actually be attenuated by increased\\nphysical activity (Vimaleswaran et al., 2009).A replication study is essential to establish the\\nreported genetic association, especially among\\npopulations of different ethnic backgrounds.However,\\nstudies involving Oceanic population (Ohashi\\net al., 2007), African Americans (Hinney et al., 2007),\\nHan Chinese (Li et al., 2008), and Japanese (Horikoshi\\net al., 2007) failed to detect the association between previously reported SNPs and obesity or obesity-related traits\\n(Chang et al., 2008).\\n</SENT 10_03>\\n<SENT 10_04>\\nAlthough the limited sample size\\nand power of these studies is the most likely reason for\\nthe lack of association, evidence showed that otherFTO\\nSNPs not in linkage disequilibrium (LD) with rs9939609\\nmay be causative variants in non-European populations\\n(Grant et al., 2008).Taken together, the significant role\\nplayed by physical activity in the relation betweenFTO\\ngenetic variant and obesity across different cultures or\\nethnicities needs to be established.\\n</SENT 10_04>\\n<SENT 10_05>\\nAlthoughFTO genotypes have been reported to interact with physical activity\\nand correlate with BMI (Lee et al., 2010), the direct effect\\nof the risk alleles in theFTO gene and physical activity\\non obesity have yet to be reported in the Korean\\npopulation.Although physical activity has been shown to facilitate weight loss and weight maintenance in obese\\nsubjects, a tremendous interindividual variation exists\\n(Perusse et al., 2000).It is unknown whether lifestyle factors, such as physical activity, attenuate obesity in individuals with an underlyingFTO genetic predisposition.Thus, the objective of this study was to further\\nexplore the relationships between physical activity,FTO\\nSNPs, and obesity within a Korean population.\\n</SENT 10_05>\\n<SENT 10_06>\\nWe\\nbelieve that our results pertaining to obesity and physical\\nactivity are important and widely applicable to the obesity health promotion program.2 | MATERIALS AND METHODS\\n2.1 | Subjects\\nA total of 8840 subjects aged 40 to 69 years were recruited\\nfrom the Ansung and Ansan cohorts of the Korean\\nGenome and Epidemiology Study (KoGES), which represent rural and urban communities in Korea, respectively.The baseline survey of the KoGES involved a total of\\n10 038 adults from 2001 to 2002, surveyed biannually.The Korea Association Resource (KARE) project facilitated the availability of DNA in the large-scale genomewide analysis of 10 004 participants, following which\\n8840 genotypes were released into the public domain\\n(Lee et al., 2010).\\n</SENT 10_06>\\n<SENT 10_07>\\nA detailed analysis of Ansung and\\nAnsan cohorts has been published previously (Cho\\net al., 2007).The participant survey involved not only\\ntheir parents' health history and anthropometric data\\nsuch as height, weight, and waist and hip circumference,\\nbut also education and income, physical activity, and dietary intake.We categorized the participants as nonobese\\n(18.5 ≤ BMI < 25 kg/m2) or obese (30 kg/m2 ≤ BMI).We\\nfollowed the WHO cutoffs to delineate obesity (BMI of\\n25.0-29.99 for overweight [preobesity] and≥30.0 for obesity) (World Health Organization, 2000).\\n</SENT 10_07>\\n<SENT 10_08>\\nThe genetic\\ninformation used in this study was distributed by the\\nHuman Resources Bank of Korea Centers for Disease\\nControl and Prevention (KBN-2017-046) and analyzed\\nafter approval of the research ethics committee of the\\nKorea National Institute of Health (KNIH) and Hoeso\\nUniversity (1041231170418-HR-056-02).Written informed\\nconsent was obtained from all subjects.2.2 | Anthropometry and physical\\nactivity\\nHeight, weight, and circumference of waist and hips were\\nmeasured by qualified medical staff, and BMI (weight (kg)/\\nheight (m2)) and waist-to-hip ratios (WHR = waist measurement (cm)/hip measurement (cm)) were calculated.2o f1 0 CHO ET AL.\\n</SENT 10_08>\\n\",\n",
       "  ['1. What specific physical activities were considered in the study?',\n",
       "   '2. How was the intensity of physical activity measured and categorized?',\n",
       "   '3. Were there any differences in the association between FTO rs9939609 and obesity in rural vs. urban populations within the Korean population?',\n",
       "   '4. Did the study control for potential confounding factors such as diet and socioeconomic status when examining the relationship between physical activity, FTO rs9939609, and obesity?',\n",
       "   '5. How did the researchers define \"active exercise\" in the context of their study?']),\n",
       " (\"<SENT 11_01>\\nBody fat percentage (BFP) and abdominal fat percentage\\n(AbFP) were measured via tetrapolar bioelectrical impedance analysis (Inbody 3.0; Biospace, Seoul, Korea).Obesity\\n(case) was defined by BMI greater than 30 kg/m2 and nonobesity (control) was defined by BMI 18.5 kg/m2 or higher,\\nbut less than 25 kg/m2.To assess the index of physical activity, questionnaires were administered to subjects to determine the number of hours per day the adult participated in\\nintense activity (athletic sports, hiking, running, tree felling,\\nforestry, mining industry, and farming).According to the\\nlevel of activity, we classified subjects into two groups: inactive (<0.5 hour per day of intense activity) and active\\n(≥3 hours per day of intense activity).\\n</SENT 11_01>\\n<SENT 11_02>\\nIn addition, by applying standardized protocols throughout the entire process of\\ninvestigation, in-depth data quality control and measurement were achieved.Through this systematic and standardized protocol, we confirmed the validity and reliability of\\nthe assessment in terms of physical activity measures.2.3 | Genotyping\\nGenomic DNA isolated from participants' whole blood\\nwas genotyped with Affymetrix Genome-Wide Human\\nSNP array 5.0 (Affymetrix Inc., Santa Clara, CA).Subjects\\nwith a genotyping accuracy less than 98%, high missing\\ngenotype call rates greater than 4%, heterozygosity\\ngreater than 30%, or gender mismatch were excluded.\\n</SENT 11_02>\\n<SENT 11_03>\\nThe Affymetrix 500K arrays contain 87 SNPs within the\\nregion of FTO gene.We selected and analyzed only\\nrs9939609, which is well-known and had the lowest\\nPvalue in all of BMI, BFP, and AbFP.The location of\\nSNPs on the chromosome is based on National Center for\\nBiotechnology Information (NCBI) human genome build\\n36 (hg18).The intragenotypic variations of rs9939609 in\\nall outcome measures were analyzed using the dominant\\nmodel (A allele carriers (AA/AT; n = 151, 1927 respectively) vs homozygous genotype (TT; n = 6751).\\n</SENT 11_03>\\n<SENT 11_04>\\n2.4 | Statistical analysis\\nIn most statistical analysis, the PLINK version 1.07\\n(http://pngu.mgh.harvard.edu//C24purcell/plink) and the\\nPASW Statistics version 18.0 (SPSS Inc., Chicago, IL)\\nwere used.Logistic regression analysis was used to correlate the genetic variation between obesity and nonobesity, based on the dominant genetic model for the\\nestimation of odd ratios (ORs) (95% confidence interval\\n[CI]), in which area, age, and gender were controlled as\\ncovariates.\\n</SENT 11_04>\\n<SENT 11_05>\\nThe significance level of the analytical value\\nwas based on 0.05 or less.LocusZoom (http://locuszoom.org/) was used to determine the correlation between\\nresults and recombination rate.HaploReg database\\n(https://pubs.broadinstitute.org/mammals/haploreg/\\nhaploreg.php) was also used to identify the potential\\nfunctional effects of the associated SNPs.3 | RESULTS\\n3.1 | Participant characteristics\\nThe average value of body measurements obtained by\\ncase-control analysis is shown in Table 1.Finally, 8840\\nparticipants were divided into obese (case) and nonobese\\n(control) categories: 441 (4.98%) were obese group and\\n4888 (55.2%) were nonobese group.As shown in Table 1,\\nBMI, an indicative marker of obesity, differed among the\\ncase-control groups and increased in the obese group.The mean value of BMI in the obese group was 9.15 kg/\\nm2 higher than in the nonobese group.\\n</SENT 11_05>\\n<SENT 11_06>\\nThe mean values\\nof waist and hip circumference of the obese group were\\nhigher than in the nonobese group and were 18.41 and\\n12.92, respectively.Finally, the WHR of the obese group\\nwas 0.09-fold higher than that of the nonobese group.3.2 | FTO variants and anthropometric\\nand biochemical phenotypes\\nAmong the indicators related to obesity, we selected BMI,\\nBFP, AbFP, and WHR to analyze the association with\\nFTO genetic variation.Twelve SNPs were associated with\\nBMI, 8 SNPs with BFP, and 12 SNPs with AbFP (data not\\nshown).\\n</SENT 11_06>\\n<SENT 11_07>\\nThe BMI, BFP, and AbFP showed the lowest\\nP values in rs9939609, rs7193144, and rs9940128 (Table 2).Also, the parameters BMI, BFP, and AbFP were significantly associated with theFTO rs9939609 genotype\\n(1.76 × 10−6,1 .7 7× 10−4, and 1.91 × 10−5, respectively).However, no association was found between WHR and\\nFTOrs9939609, rs7193144, and rs9940128 genotypes.3.3 | Interaction of theFTO variant\\nrs9939609 and physical activity with obesity\\nThe genotypes of rs9939609 have varying effects on the\\nimpact of exercise.In the dominant model, the association with exercise was highly significant (P value = .012).Among A carriers, the risk of obesity among individuals\\nwho exercised (OR = 0.62) was approximately more than\\n2-fold lower than in those who did not exercise\\n(OR = 1.42), suggesting that the interaction with genotype and exercise reduced the risk (Figure 1).However,\\nthere was no statistically significant association between\\nCHO ET AL.3o f1 0\\n</SENT 11_07>\\n\\n\\n<SENT 12_01>\\nindividuals carrying homozygous genotype of major\\nalleles and exercise (OR = 0.87, CI [0.62-1.28],\\nPvalue = 0.446), although exercise showed a tendency\\ntoward a lower odds ratio.It also demonstrated a significant association with sex, which was one of the covariants (Table S1, Supporting Information).The association\\nbetween sex and obesity was apparent in both minor\\nallele carriers (AA, AT) and the homozygous major allele\\ngroup (TT) (P value = 0.001, 0, respectively).In the bar\\ngraph (Figure 1), the A allele groups associated with exercise were associated with a reduced risk of obesity compared with those carrying the A alleles without exposure\\nto exercise.\\n</SENT 12_01>\\n<SENT 12_02>\\nIn addition, the probability of reduced risk of\\nobesity was greater than the probability of reduced risk of\\nobesity among subjects carrying TT genotype and\\nexposed to exercise (Figure 1).Finally, to test the\\nTABLE 1 Characteristics of subjects in the Korean population\\nQuantitative\\ntrait analysis\\nCase-control analysisa\\nP\\nvalueb\\nObese\\ngroup\\nNonobese\\ngroup\\nNumber of subjects 8840 441 4888\\nAge (M years ± SD) 52.22 ± 8.91 52.43 ± 8.53 52.08 ± 9.11 .432\\nSex (men (%)) 4182 (47.31%) 117 (26.53%) 443 (53.72%) <.001\\nHeight (cm ± SD) 160 ± 8.66 157.10 ± 8.47 160.37 ± 8.49 <.001\\nWeight (cm ± SD) 63.08 ± 10.11 78.50 ± 8.91 58.35 ± 7.40 <.001\\nBMI (M kg/m2 ± SD) 24.58 ± 3.13 31.77 ± 1.79 22.62 ± 1.61 <.001\\nWaist (cm ± SD) 82.65 ± 8.81 96.77 ± 7.24 78.36 ± 6.78 <.001\\nHip (cm ± SD) 93.63 ± 5.92 103.83 ± 5.37 90.91 ± 4.58 <.001\\nWHR (M ± SD) 0.88 ± 0.08 0.93 ± 0.06 0.84 ± 0.44 <.001\\nBFP (M ± SD) 27.02 ± 7.09 37.15 ± 5.58 24.36 ± 6.21 <.001\\nAbFP (M ± SD) 0.90 ± 0.04 0.99 ± 0.04 0.87 ± 0.03 <.001\\nAbbreviations: AbFP, abdominal fat percentage; BFP, body fat percentage; BMI, body mass index; M, mean value; SD, standard deviation;\\nWHR, waist-to-hip ratio.\\n</SENT 12_02>\\n<SENT 12_03>\\naObese group > BMI 30 kg/m2; nonobese group, 18.5 < BMI≤ 25 kg/m2.bSignificant differences in obese and nonobese groups were obtained by the Studentt test.TABLE 2 Results of association\\nanalysis between the three singlenucleotide polymorphisms (SNPs) in\\nthe FTO gene and obesity-related traits\\nin the KARE study cohort\\nPhenotype SNP A1\\nAdditive model\\nEffect sizeβ ± SE P value\\nBMI rs9939609 A 0.334 ± 0.070 1.76 × 10−6\\nrs7193144 C 0.328 ± 0.070 2.80 × 10−6\\nrs9940128 A 0.205 ± 0.063 1.12 × 10−3\\nBFP rs9939609 A 0.468 ± 0.12 1.77 × 10−4\\nrs7193144 C 0.453 ± 0.12 2.90 × 10−4\\nrs9940128 A 0.313 ± 0.11 5.49 × 10−3\\nAbFP rs9939609 A 0.0047 ± 0.0011 1.91 × 10−5\\nrs7193144 C 0.0046 ± 0.0011 2.66 × 10−5\\nrs9940128 A 0.0031 ± 0.0010 1.56 × 10−3\\nWHR rs9939609 A 0.0019 ± 0.0013 .120\\nrs7193144 C 0.0021 ± 0.0014 .137\\nrs9940128 A 0.0019 ± 0.0014 .162\\nNote: Age, sex, and area were included as covariants in linear regression analysis.Abbreviations: A1, minor allele; AbFP, abdominal fat percentage; BFP, body fat percentage;\\nBMI, body mass index;β, regression coefficient; SE, standard error; WHR, waist-to-hip ratio.4o f1 0 CHO ET AL.\\n</SENT 12_03>\\n\",\n",
       "  ['1. How was the body fat percentage (BFP) and abdominal fat percentage (AbFP) measured in this study?',\n",
       "   '2. What were the criteria used to define obesity and non-obesity in this study?',\n",
       "   '3. How were the participants categorized based on their level of physical activity?',\n",
       "   '4. What was the purpose of using standardized protocols throughout the investigation process?',\n",
       "   '5. How were the genotyping accuracy and missing genotype call rates determined in this study?']),\n",
       " ('<SENT 13_01>\\nsignificance of the interactive effect between FTO\\nrs9939609 and physical activity, linear regression analysis\\nwas performed (Table S2).It was confirmed that exercise\\nstatus controls the effect of minor allele carriers (AA, AT)\\non obesity.TheR2 value was 0.001 (0.1%) in stage 1 and\\n0.002 (0.2%) in stage 2, but increased to 0.893 (89.2%) in\\nstage 3.According to the significance probability, it was\\nfound to be less than 0.001, indicating that there is an\\nadjustment effect (Table S2).\\n</SENT 13_01>\\n<SENT 13_02>\\nThese results indicate that\\nthe effect of exercise was significantly associated with the\\nFTO rs9939609 genotype in Korean adults.3.4 | Recombination rate of SNPs near\\nthe FTO gene loci and functional\\nannotation using HaploReg\\nWe used the LocusZoom andHaploReg programs to determine the recombination rates of SNPs and functional\\nannotation of the FTO genetic variation.Eight SNPs\\nshowed a high recombination rate near theFTO gene in\\nLocusZoom (Figure 2).Among these, the SNP most closely\\nrelated to BMI was rs9939609 (P value = 1.76 × 10−6).\\n</SENT 13_02>\\n<SENT 13_03>\\nThese results suggest that in KARE, regardless of\\nrs9939609 allele, rs9926289 (D0 =1 , r2 = 1), rs8050136\\n(D0 =0 .9 9 ,r2 = 0.99), rs7193144 (D0 =0 .9 9 ,r2 =0 .9 8 ) ,a n d\\nrs1121980 (D0 =0 .9 9 ,r2 = 0.88) define a haplotype that is\\nassociated with increased BMI.To identify the potential\\nrole of the associated SNP (rs9939609) inFTO gene, we\\nscanned the SNP site using the HaploReg database.Using HaploReg, we identified 91 genetic variants\\ntagged by rs9939609 variant withr2 ≥ 0.8.These\\n91 genetic variants are located around the FTO.T h e\\nmost significantly associated rs9939609 site inFTO gene\\nwas predicted to change the motif for Nanog and Pou5f1\\ntranscription factor binding site in Table 3.\\n</SENT 13_03>\\n<SENT 13_04>\\nFurthermore, the SNP site (rs9939609) was also associated with\\nhistones marks associated with enhancers in breast cancer loci (HaploReg v 4.1).4 | DISCUSSION\\nEarly detection risk factors are important for the prevention of childhood obesity and its complications, especially\\ngiven the social and habitual conditions associated with\\nthe increased prevalence of obesity.The interactions\\nFIGURE 1 Relative odds ratio of obesity according to the\\nphysical activity andFTO rs9939609 genotypes.The OR (95% CI) of\\ngenetic interaction of dominant (AA+AT)/homozygous\\n(TT) genotypes ofFTO rs9939609 with variable exercise.\\n</SENT 13_04>\\n<SENT 13_05>\\nThe\\nhomozygous model (TT) was set as the reference allele.P values\\nwere adjusted for age, gender, and area by analysis of covariance.*P < .05.OR, odds ratio; CI, confidence interval\\nFIGURE 2 Association plot with the\\nrecombination rate of SNPs near theFTO gene\\nloci of chromosome 16 with obesity and BMI.LocusZoom plot, which analyzes the SNPs\\nlocated in the BMI associated region near the\\nFTOgene and the recombination rate LD (r2)\\nbased on genotyping data obtained from 8840\\nKARE subjects are presented.FTO gene location\\non chromosome 16 and the number of examined\\nSNPs is 87.Results are based on the minimum\\nP value (rs9939609).The color of each SNP\\nsymbol represents its LD (using ther2\\nalgorithm) with the top SNP (purple diamond)\\nwithin the association locus.The image above is\\ngenerated using the LocusZoom program\\n(http://locuszoom.org/)\\nCHO ET AL.5o f1 0\\n</SENT 13_05>\\n\\n\\n<SENT 14_01>\\nbetween genes, environment and physical activity determine the risk of obesity and its early detection for prompt\\nintervention (Lee et al., 2010).A growing number of studies have shown that the influence of theFTO gene on\\nobesity may be modified by lifestyle factors such as diet\\nand physical activity.Polymorphisms of the FTO gene\\nare the only genetic variants that have been associated\\nwith obesity in populations of European ancestry (Dina\\net al., 2007; Frayling et al., 2007; Hinney et al., 2007; Scuteri et al., 2007).Also, several studies reported strong\\nassociations ofFTO genetic variants with obesity and\\nBMI in East Asian population (Chang et al., 2008; Hotta\\net al., 2008; Lee et al., 2010; Omori et al., 2008).\\n</SENT 14_01>\\n<SENT 14_02>\\nA significant association between rs9939609 and obesity-related\\ntraits (BMI, BFP, and AbFP) was detected using linear\\nregression analysis (Table 2).However, there was no significant association with WHR and SNPs, nor any statistically significant relationship between different genders\\n(data not shown).In our study, we replicated the significant association between rs9939609 and obesity-related traits, such\\nas BMI, BFP, and AbFP.Individuals carrying the minor\\nallele (A) of rs9939609 had higher BMI than those carrying the homozygous major allele (TT).\\n</SENT 14_02>\\n<SENT 14_03>\\nThe frequency of\\nminor allele carriers (AA/AT) of rs9939609 associated\\nwith the risk of overweight was about 12.62% in this\\nstudy, and the frequency of homozygotes with the A\\nallele was only 1.7% in Koreans.Thus, our results are\\nsimilar to those of Lee et al.(2010).This risk allele was\\nrare in Asians (12.6%) compared with Europeans (45%)\\nand West Africans (52%), and the variance in BMI\\nattributed to rs9939609 was also much lower (/C240.5%)\\nthan in Europeans (/C241%) (Chang et al., 2008; Frayling\\net al., 2007).\\n</SENT 14_03>\\n<SENT 14_04>\\nAlthough variants in FTO causally linked to obesity\\nare less common in Korean population, many studies\\nshow that theFTO gene affects obesity in East Asians.Several genome-wide association studies (GWAS) performed in Asian countries have also demonstrated significant associations of well-known FTO variants with\\nobesity and BMI (Chang et al., 2008; Cho et al., 2009;\\nHuang, Sun, & Sun, 2011; Ng et al., 2008; Sull, Lee, &\\nJee, 2013; Takeuchi et al., 2011; Tan et al., 2008).\\n</SENT 14_04>\\n<SENT 14_05>\\nCho\\net al.(2009) reported that rs9939609 was strongly associated with BMI using the KARE sample in Korea,\\nsuggesting that rs9939609 showed a strong correlation\\nwith the underlying causative variant.Similarly, Luglio\\nand Sulistyoningrum (2014) analyzed samples from\\n“Caucasian” in Europe and North America and Asians\\nfrom East and Southeast Asia and observed that minor\\nallele frequency in Asians was lower than in individuals\\nof European descent (Luglio & Sulistyoningrum, 2014).Interestingly, the OR for obesity was similar between the\\ntwo sets of populations.\\n</SENT 14_05>\\n<SENT 14_06>\\nThey concluded that although\\nminor allele frequency in Europeans was higher than\\nAsians, the influence on obesity was similar, supporting\\nthe higher predominance of obesity in Europeans\\n(Luglio & Sulistyoningrum, 2014).On the other hand, in seeking the biological basis for\\nthe interaction, one study reported that obesity-related\\nFTO gene restricts functions of the hippocampus\\n(Spychala & Rüther, 2019).The loss of FTO increases\\nstress parameters such as corticosterone in the blood\\nplasma, which suggests hypersensitivity of the\\nhypothalamic-pituitary-adrenal (HPA) axis.In addition,\\nSpychala and Rüther pointed out that blockingFTO gene\\ncan have negative effects on the regular function of the\\nhippocampus (Spychala & Rüther, 2019).\\n</SENT 14_06>\\n<SENT 14_07>\\nIn parallel,FTO\\ngene was related to depression, which is characterized by\\na deficient of reward-seeking behavior (Samaan\\net al., 2013; Shankman, Klein, Tenke, & Bruder, 2007).Samaan et al.provided the meta-analysis of the association between obesity and depression and suggested that\\ndepression improves the effect of FTO variants on\\nobesity.Numerous studies involving various ethnic and age\\ngroups investigated whether the interaction between\\nphysical activity and FTO genetic variants affected obesity risk.\\n</SENT 14_07>\\n<SENT 14_08>\\nOur study focused on the genetic association\\nbetweenFTO polymorphisms in KARE cohort and obesity and evaluated the effects of lifestyle factors such as\\nphysical activity.In contrast to studies related to food\\nintake (Park et al., 2013) or increased appetite (Wardle\\net al., 2008), the results related to the effects of physical\\nTABLE 3 Results of the HaploReg\\nof three SNPs in theFTO gene\\nSNP A1 A2\\nHaploReg\\nDNase Enhancer histone marks Motifs changed\\nrs9939609 A T BRST Nanog, Pou5f1\\nrs7193144 C T 13 tissues 4 altered motifs\\nrs9940128 A G 14 tissues 14 tissues Ets, NF-AT1, TATA\\nNote: HaploReg is a tool used to explore annotations of candidate regulatory SNPs (http://\\narchive.broadinstitute.org/mammals/haploreg/haploreg_v3.php).Abbreviations: A1, minor allele; A2, major allele; SNP, single-nucleotide polymorphism.6o f1 0 CHO ET AL.\\n</SENT 14_08>\\n',\n",
       "  ['1. What is the significance of the interactive effect between FTO rs9939609 and physical activity in the context of obesity?',\n",
       "   '2. How does exercise status control the effect of minor allele carriers (AA, AT) on obesity?',\n",
       "   '3. What is the difference in R2 values between stages 1, 2, and 3, and what does this indicate about the adjustment effect?',\n",
       "   '4. How do the results of the study indicate that the effect of exercise is significantly associated with the FTO rs9939609 genotype in Korean adults?',\n",
       "   '5. What were the recombination rates of SNPs near the FTO gene loci, and how were they determined using the LocusZoom and HaploReg programs?']),\n",
       " (\"<SENT 15_01>\\nactivity varied among studies, largely depending on\\nwhether or not A allele, which is a risk allele of\\nrs9939609, was related to exercise.The key finding of our study was that the association\\nbetween FTO gene polymorphism and exercise was significant only in subjects with the A allele (AA/AT) (Figure 1).Lee et al.observed the effect ofFTO variants on fat mass\\nand its changes by physical activity in Korean population.Also, West et al.showed that the difference in patterns of\\nphysical activity betweenFTO genotypes may further offset\\nthe adiposity-increasing effect ofFTO within the European\\ncohort (West et al., 2018).Other studies have reported that\\nphysical inactivity confers an increased risk ofFTO genetic\\npredisposition to obesity while physically active participants who carry risk alleles ofFTO SNPs exhibited a lower\\nrisk (Demerath et al., 2011; Kilpelainen et al., 2011; Kim\\net al., 2016).Finally, Payne et al.\\n</SENT 15_01>\\n<SENT 15_02>\\ndemonstrated the interactions between genetic susceptibility of obesity caused by\\ntwo SNPs (rs9939609 and rs1421085) in theFTO gene and\\nphysical activity (Payne et al., 2014).With consistent replications of the association betweenFTO genetic variants\\nand obesity-related phenotypes, it is of growing interest to\\ndetermine whether these genetic effects on obesity are\\nmodified by lifestyle factors such as physical activity.In this regard, a few studies failed to detect the effect\\nof any interaction between physical activity and FTO\\nSNPs on obesity (Cho et al., 2009; Huang et al., 2011).Nonetheless, physical activity levels betweenFTO genotypes conflict with previous studies suggesting that the\\nFTO genotype does not affect the physical activity level\\n(Hakanen et al., 2009).\\n</SENT 15_02>\\n<SENT 15_03>\\nBerentzen et al.(2008) reported a\\nlack of effect of theFTO variant on physical activity in\\nindividuals classified by the authors as highly active, and\\nthe effect of FTO genotype on activity levels in cohorts\\nwith even greater levels of physical activity is unknown.By contrast, our data represent unequivocal evidence that\\nphysical activity attenuates the effect of theFTO genetic\\nvariants on obesity, based on data from 8840 individuals\\nin the KARE database.While believing our results to be informative, this study\\nhas some limitations.\\n</SENT 15_03>\\n<SENT 15_04>\\nThe contrasting results of the current\\nstudy and other evidence may be attributed to the high\\nlevels of intense physical activity of individuals within the\\ncurrent study.Consequently, future studies are required to\\nexamineFTO-mediated differences in physical activity\\namong individuals with varying activity.Further measures\\nof body adiposity and precise standards of physical activity\\nrequired to replicate the current study's findings.Currently, the strongest associations between FTO\\nSNPs and obesity belong to intronic SNPs, which play a\\nrole in the regulation of genes close toFTO.\\n</SENT 15_04>\\n<SENT 15_05>\\nWe found\\nthat multiple neighboring SNPs (Figure 2) also show\\nhighly significant association with BMI.All GWAS\\nidentified SNPs belong to the same highly correlated cluster, with an LD ofr2 > 0.80, and consequently are associated with BMI at similar significance levels, as well as\\nother obesity-related traits (Loos & Yeo, 2014).Our\\nresults demonstrate that, in the KARE cohort, rs9926289\\n(D0 =1 , r2 = 1), rs8050136 ( D0 = 0.99, r2 = 0.99),\\nrs7193144 (D0 = 0.99,r2 = 0.98), and rs1121980 (D0 = 0.99,\\nr2 = 0.88) are associated with increased BMI associated\\nwith rs9939609 (Figure 2).Shabana et al.\\n</SENT 15_05>\\n<SENT 15_06>\\nshowed that the\\nFTO variants rs9939609 and rs9926289 exhibit a strong\\ncodominant, obesity-predisposing effect of genotypes\\nhomozygous for minor alleles (Shabana &\\nHasnain, 2015).Interestingly, Shabana et al.showed that\\nrs9939609 and rs9926289 were significantly associated\\nwith BMI and waist circumference (P < .0125) but not\\nwith WHR.However, rs8050136 and rs9939690 were not\\nassociated with BMI either, although one variant,\\nrs8050136, was associated with the risk of type 2 diabetes\\nrs8050136 in a Japanese (Horikoshi et al., 2007).\\n</SENT 15_06>\\n<SENT 15_07>\\nTo further examine the LD block that is significantly\\nassociated with obesity, we used the HaploReg (v4.1)\\ndatabase, in which the most significant rs9939609 SNP\\nwas identified using an imputation method based on\\nAsian (Japanese in Tokyo (JPT), Han Chinese in Beijing\\n(CHB), Southern Han Chinese (CHS)) reference panels\\nacquired from the 1000 Genomes Project and based on\\nan association analysis.We found that 91 genetic variants\\nsignificantly regulated the expression of the nearbyFTO\\ngene.\\n</SENT 15_07>\\n<SENT 15_08>\\nAmong the 91 SNPs, 80 of them carried functional\\nmotifs including rs9939609, suggesting that they may\\nplay a role in the expression and/or regulation of the\\nFTOgene (Table S3).HaploReg database further suggests\\nthat rs7193144 and rs9940128 carry a significant regulatory potential due to the presence of histone marks associated with enhancers in various tissues (Table 3).Our\\nstudy showed similar results (Table 2), demonstrating the\\nsignificant association of rs7193144 and rs9940128 with\\nobesity as reported previously (Hinney et al., 2007; Peng\\net al., 2011; Zhang et al., 2012).In the current study, higher physical activity reduced\\nthe risk of obesity via interaction with specific genotypes.\\n</SENT 15_08>\\n<SENT 15_09>\\nOur results are consistent with the basic concept that\\nphysical activity attenuated obesity mediated via the A\\nallele carriers.However, in individuals with specific\\ngenetic predisposition, particularly homozygous major\\nallele carriers, a higher level of physical activity is\\nrequired to significantly reduce the risk of obesity.5 | CONCLUSION\\nOur study found that theFTO variant rs9939609 is significantly associated with weight, BMI, and other obesityCHO ET AL.7o f1 0\\n</SENT 15_09>\\n\\n\\n<SENT 16_01>\\nrelated phenotypes in a KARE cohort.Moreover, we\\nfound significant association between rs9939609 and\\nphysical activity for any obesity-related phenotypes.Consequently, our study suggests that lifestyle factors such as\\nphysical activity substantially modulate the association\\nbetweenFTO variant rs9939609 and obesity in Korea.ACKNOWLEDGEMENTS\\nThis work was supported by the Soonchunhyang University research fund and the Basic Science Research Program through the National Research Foundation of\\nKorea (NRF) funded by the Ministry of Education (NRF2017R1D1A1B03032960).\\n</SENT 16_01>\\n<SENT 16_02>\\nAUTHOR CONTRIBUTIONS\\nHye-Won Cho:Data curation; formal analysis; visualization; writing-original draft.Hyun-Seok Jin:Conceptualization; data curation; formal analysis; visualization;\\nwriting-original draft.Yong-Bin Eom: Conceptualization; data curation; formal analysis; funding acquisition;\\ninvestigation; methodology; project administration;\\nsupervision; validation; visualization; writing-original\\ndraft; writing-review and editing.CONFLICT OF INTEREST\\nThe authors declare no potential conflict of interest.\\n</SENT 16_02>\\n<SENT 16_03>\\nDATA AVAILABILITY STATEMENT\\nThe data that support the findings of this study are available on request from the corresponding author.The data\\nare not publicly available due to privacy or ethical\\nrestrictions.ETHICS STATEMENT\\nThis study was approved by the Institutional Review Board\\nof the Korean National Institute of Health (KNIH, KBN2017-046) and Hoeso University (1041231170418-HR-056-02).Written informed consent was obtained from all subjects.ORCID\\nYong-Bin Eom https://orcid.org/0000-0002-1569-7248\\nREFERENCES\\nAndreasen, C. H., Stender-Petersen, K. L., Mogensen, M. S.,\\nTorekov, S. S., Wegner, L., Andersen, G.,… Hansen, T. (2008).Low physical activity accentuates the effect of the FTO\\nrs9939609 polymorphism on body fat accumulation.Diabetes,\\n57,9 5–101.Berentzen, T., Kring, S. I., Holst, C., Zimmermann, E., Jess, T.,\\nHansen, T., … Sorensen, T. I.(2008).\\n</SENT 16_03>\\n<SENT 16_04>\\nLack of association of\\nfatness-related FTO gene variants with energy expenditure or\\nphysical activity.Journal of Clinical Endocrinology and Metabolism, 93, 2904–2908.Bernstein, M. S., Costanza, M. C., & Morabia, A.(2004).Association\\nof physical activity intensity levels with overweight and obesity\\nin a population-based sample of adults.Preventive Medicine, 38,\\n94–104.Chang, Y. C., Liu, P. H., Lee, W. J., Chang, T. J., Jiang, Y. D., Li, H. Y.,\\n… Chuang, L. M. (2008).Common variation in the fat mass and\\nobesity-associated (FTO) gene confers risk of obesity and modulates BMI in the Chinese population.Diabetes, 57,2 2 4 5–2252.Cho, N. H., Jang, H. C., Choi, S. H., Kim, H. R., Lee, H. K.,\\nChan, J. C., & Lim, S. (2007).Abnormal liver function test predicts type 2 diabetes: A community-based prospective study.\\n</SENT 16_04>\\n<SENT 16_05>\\nDiabetes Care, 30, 2566–2568.Cho, Y. S., Go, M. J., Kim, Y. J., Heo, J. Y., Oh, J. H., Ban, H. J.,…\\nKim, H. L. (2009).A large-scale genome-wide association study\\nof Asian populations uncovers genetic factors influencing eight\\nquantitative traits.Nature Genetics, 41, 527–534.Cornes, B. K., Lind, P. A., Medland, S. E., Montgomery, G. W.,\\nNyholt, D. R., & Martin, N. G. (2009).Replication of the association of common rs9939609 variant ofFTO with increased BMI\\nin an Australian adult twin population but no evidence for gene\\nby environment (G x E) interaction.International Journal of\\nObesity, 33,7 5–79.Demerath, E. W., Lutsey, P. L., Monda, K. L., Linda Kao, W. H.,\\nBressler, J., Pankow, J. S.,… Folsom, A. R. (2011).Interaction\\nof FTO and physical activity level on adiposity in AfricanAmerican and European-American adults: The ARIC study.Obesity (Silver Spring), 19, 1866–1872.\\n</SENT 16_05>\\n<SENT 16_06>\\nDina, C., Meyre, D., Gallina, S., Durand, E., Korner, A.,\\nJacobson, P.,… Froguel, P. (2007).Variation inFTO contributes\\nto childhood obesity and severe adult obesity.Nature Genetics,\\n39, 724–726.Frayling, T. M., Timpson, N. J., Weedon, M. N., Zeggini, E.,\\nFreathy, R. M., Lindgren, C. M.,… McCarthy, M. I.(2007).A\\ncommon variant in theFTO gene is associated with body mass\\nindex and predisposes to childhood and adult obesity.Science,\\n316, 889–894.Grant, S. F., Li, M., Bradfield, J. P., Kim, C. E., Annaiah, K.,\\nSanta, E.,… Hakonarson, H. (2008).Association analysis of the\\nFTO gene with obesity in children of Caucasian and African\\nancestry reveals a common tagging SNP.PLoS One, 3, e1746.Hakanen, M., Raitakari, O. T., Lehtimaki, T., Peltonen, N.,\\nPahkala, K., Sillanmaki, L.,… Ronnemaa, T. (2009).FTO genotype is associated with body mass index after the age of seven\\nyears but not with energy intake or leisure-time physical activity.\\n</SENT 16_06>\\n<SENT 16_07>\\nJournal of Clinical Endocrinology and Metabolism , 94,\\n1281–1287.Hinney, A., Nguyen, T. T., Scherag, A., Friedel, S., Bronner, G.,\\nMuller, T. D.,… Hebebrand, J.(2007).Genome wide association\\n(GWA) study for early onset extreme obesity supports the role\\nof fat mass and obesity associated gene (FTO) variants.PLoS\\nOne, 2, e1361.Horikoshi, M., Hara, K., Ito, C., Shojima, N., Nagai, R., Ueki, K.,…\\nKadowaki, T. (2007).Variations in the HHEX gene are associated with increased risk of type 2 diabetes in the Japanese population.Diabetologia, 50, 2461–2466.Hotta, K., Nakata, Y., Matsuo, T., Kamohara, S., Kotani, K.,\\nKomatsu, R., … Nakamura, Y.(2008).Variations in theFTO\\ngene are associated with severe obesity in the Japanese.Journal\\nof Human Genetics, 53, 546–553.8o f1 0 CHO ET AL.\\n</SENT 16_07>\\n\",\n",
       "  ['1. How did the activity vary among the studies mentioned in the context?',\n",
       "   '2. What were the specific findings of Lee et al. regarding the effect of FTO variants on fat mass and its changes by physical activity in the Korean population?',\n",
       "   \"3. In the context, what were the results of West et al.'s study on the difference in patterns of physical activity between FTO genotypes within the European cohort?\",\n",
       "   \"4. What were the conclusions of Demerath et al., Kilpelainen et al., and Kim et al.'s studies regarding the relationship between physical inactivity, FTO genetic predisposition to obesity, and risk?\",\n",
       "   '5. How did Payne et al. demonstrate the interactions between genetic susceptibility of obesity caused by two SNPs (rs9939609 and rs1421085) in the FTO gene and physical activity?']),\n",
       " (\"<SENT 17_01>\\nHuang, W., Sun, Y., & Sun, J.(2011).Combined effects ofFTO\\nrs9939609 and MC4R rs17782313 on obesity and BMI in Chinese Han populations.Endocrine, 39,6 9–74.Kilpelainen, T. O., Qi, L., Brage, S., Sharp, S. J., Sonestedt, E.,\\nDemerath, E.,… Loos, R. J.(2011).Physical activity attenuates\\nthe influence ofFTO variants on obesity risk: A meta-analysis\\nof 218,166 adults and 19,268 children.PLoS Medicine , 8,\\ne1001116.Kim, J. Y., DeMenna, J. T., Puppala, S., Chittoor, G., Schneider, J.,\\nDuggirala, R., … Coletta, D. K. (2016).\\n</SENT 17_01>\\n<SENT 17_02>\\nPhysical activity and\\nFTO genotype by physical activity interactive influences on\\nobesity.BMC Genetics, 17, 47.King, G. A., Fitzhugh, E. C., Bassett, D. R., Jr., McLaughlin, J. E.,\\nStrath, S. J., Swartz, A. M., & Thompson, D. L. (2001).Relationship of leisure-time physical activity and occupational activity\\nto the prevalence of obesity.International Journal of Obesity\\nand Related Metabolic Disorders, 25, 606–612.Koike, S., Richards, M., Wong, A., & Hardy, R. (2018).Fat mass and\\nobesity-associated (FTO) rs9939609 polymorphism modifies the\\nrelationship between body mass index and affective symptoms\\nthrough the life course: A prospective birth cohort study.Translational Psychiatry, 8, 62.Lee, H. J., Kim, I. K., Kang, J. H., Ahn, Y., Han, B. G., Lee, J. Y., &\\nSong, J.(2010).\\n</SENT 17_02>\\n<SENT 17_03>\\nEffects of commonFTO gene variants associated with BMI on dietary intake and physical activity in\\nKoreans.Clinica Chimica Acta, 411, 1716–1722.Leonska-Duniec, A., Jastrzebski, Z., Zarebska, A., Maciejewska, A.,\\nFicek, K., & Cieszczyk, P. (2018).Assessing effect of interaction\\nbetween theFTO A/T polymorphism (rs9939609) and physical\\nactivity on obesity-related traits.Journal of Sport and Health\\nScience, 7, 459–464.Li, H., Wu, Y., Loos, R. J., Hu, F. B., Liu, Y., Wang, J.,… Lin, X.(2008).Variants in the fat mass- and obesity-associated (FTO)\\ngene are not associated with obesity in a Chinese Han population.Diabetes, 57, 264–268.\\n</SENT 17_03>\\n<SENT 17_04>\\nLiaw, Y. C., Liaw, Y. P., & Lan, T. H. (2019).Physical activity might\\nreduce the adverse impacts of theFTO gene variant rs3751812\\non the body mass index of adults in Taiwan.Genes (Basel), 10\\n(5), 354.Loos, R. J., & Yeo, G. S. (2014).The bigger picture ofFTO: The first\\nGWAS-identified obesity gene.Nature Reviews.Endocrinology,\\n10,5 1–61.Luglio, F. H., & Sulistyoningrum, C. D. (2014).Single nucleotide\\npolymorphism in FTO rs9939609 and obesity among Asians\\nand Caucasians: A meta-analysis.\\n</SENT 17_04>\\n<SENT 17_05>\\nImmunology, Endocrine &\\nMetabolic Agents in Medicinal Chemistry (Formerly Current\\nMedicinal Chemistry-Immunology, Endocrine and Metabolic\\nAgents), 14,4 6–53.Ng, M. C., Park, K. S., Oh, B., Tam, C. H., Cho, Y. M., Shin, H. D.,\\n… Cho, N. H. (2008).Implication of genetic variants near\\nTCF7L2, SLC30A8, HHEX, CDKAL1, CDKN2A/B, IGF2BP2,\\nand FTO in type 2 diabetes and obesity in 6,719 Asians.Diabetes, 57, 2226–2233.Ogden, C. L., Carroll, M. D., Curtin, L. R., McDowell, M. A.,\\nTabak, C. J., & Flegal, K. M. (2006).\\n</SENT 17_05>\\n<SENT 17_06>\\nPrevalence of overweight\\nand obesity in the United States, 1999–2004.JAMA, 295,\\n1549–1555.Ohashi, J., Naka, I., Kimura, R., Natsuhara, K., Yamauchi, T.,\\nFurusawa, T., … Ohtsuka, R. (2007).FTO polymorphisms in\\noceanic populations.Journal of Human Genetics , 52,\\n1031–1035.Omori, S., Tanaka, Y., Takahashi, A., Hirose, H., Kashiwagi, A.,\\nKaku, K., … Maeda, S. (2008).Association of CDKAL1,\\nIGF2BP2, CDKN2A/B, HHEX, SLC30A8, and KCNJ11 with\\nsusceptibility to type 2 diabetes in a Japanese population.Diabetes, 57, 791–795.O'Rahilly, S., & Farooqi, I. S. (2008).\\n</SENT 17_06>\\n<SENT 17_07>\\nHuman obesity: A heritable\\nneurobehavioral disorder that is highly sensitive to environmental conditions.Diabetes, 57, 2905–2910.Park, S. L., Cheng, I., Pendergrass, S. A., Kucharska-Newton, A. M.,\\nLim, U., Ambite, J. L.,… Le Marchand, L. (2013).Association\\nof the FTO obesity risk variant rs8050136 with percentage of\\nenergy intake from fat in multiple racial/ethnic populations:\\nThe PAGE study.American Journal of Epidemiology , 178,\\n780–790.Payne, A., Cahill, F., Sun, G., Loredo-Osti, J. C., & Abarin, T.\\n(2014).\\n</SENT 17_07>\\n<SENT 17_08>\\nEffect ofFTO gene and physical activity interaction on\\ntrunk fat percentage among the Newfoundland population.Genetics & Epigenetics, 6,2 1–30.Peng, S., Zhu, Y., Xu, F., Ren, X., Li, X., & Lai, M. (2011).FTO gene\\npolymorphisms and obesity risk: A meta-analysis.BMC Medicine, 9, 71.Perusse, L., Rice, T., Province, M. A., Gagnon, J., Leon, A. S.,\\nSkinner, J. S., … Bouchard, C. (2000).Familial aggregation of\\namount and distribution of subcutaneous fat and their\\nresponses to exercise training in the HERITAGE family study.Obesity Research, 8, 140–150.Qi, Q., Kilpelainen, T. O., Downer, M. K., Tanaka, T., Smith, C. E.,\\nSluijs, I., … Qi, L. (2014).FTO genetic variants, dietary intake\\nand body mass index: Insights from 177,330 individuals.Human Molecular Genetics, 23, 6961–6972.\\n</SENT 17_08>\\n<SENT 17_09>\\nSaber-Ayad, M., Manzoor, S., Radwan, H., Hammoudeh, S.,\\nWardeh, R., Ashraf, A.,… Hamoudi, R. (2019).TheFTO genetic\\nvariants are associated with dietary intake and body mass index\\namongst Emirati population.PLoS One, 14, e0223808.Samaan, Z., Anand, S., Zhang, X., Desai, D., Rivera, M., Pare, G.,…\\nEngert, J.(2013).The protective effect of the obesity-associated\\nrs9939609 A variant in fat mass-and obesity-associated gene on\\ndepression.Molecular Psychiatry, 18, 1281–1286.Scuteri, A., Sanna, S., Chen, W. M., Uda, M., Albai, G., Strait, J.,…\\nAbecasis, G. R. (2007).Genome-wide association scan shows\\ngenetic variants in theFTO gene are associated with obesityrelated traits.PLoS Genetics, 3, e115.\\n</SENT 17_09>\\n<SENT 17_10>\\nShabana, S., & Hasnain, S. (2015).Effect of the common fat mass\\nand obesity associated gene variants on obesity in Pakistani\\npopulation: A case-control study.BioMed Research International, 2015, 852920.Shankman, S. A., Klein, D. N., Tenke, C. E., & Bruder, G. E. (2007).Reward sensitivity in depression: A biobehavioral study.Journal of Abnormal Psychology, 116,9 5–104.Spychala, A., & Rüther, U.(2019).FTO affects hippocampal function by regulation of BDNF processing.PLoS One , 14,\\ne0211937.Sull, J. W., Lee, M., & Jee, S. H. (2013).Replication of genetic effects\\nof MC4R polymorphisms on body mass index in a Korean population.Endocrine, 44, 675–679.Takeuchi, F., Yamamoto, K., Katsuya, T., Nabika, T., Sugiyama, T.,\\nFujioka, A., … Kato, N. (2011).Association of genetic variants\\nCHO ET AL.9o f1 0\\n</SENT 17_10>\\n\\n\\n<SENT 18_01>\\nfor susceptibility to obesity with type 2 diabetes in Japanese\\nindividuals.Diabetologia, 54, 1350–1359.Tan, J. T., Dorajoo, R., Seielstad, M., Sim, X. L., Ong, R. T.,\\nChia, K. S.,… Tai, E. S. (2008).FTO variants are associated with\\nobesity in the Chinese and Malay populations in Singapore.Diabetes, 57, 2851–2857.Vimaleswaran, K. S., Li, S., Zhao, J. H., Luan, J., Bingham, S. A.,\\nKhaw, K. T.,… Loos, R. J.(2009).Physical activity attenuates the\\nbody mass index-increasing influence of genetic variation in the\\nFTO gene.American Journal of Clinical Nutrition, 90,4 2 5–428.\\n</SENT 18_01>\\n<SENT 18_02>\\nWardle, J., Carnell, S., Haworth, C. M., Farooqi, I. S.,\\nO'Rahilly, S., & Plomin, R. (2008).Obesity associated genetic\\nvariation in FTO is associated with diminished satiety.Journal\\nof Clinical Endocrinology and Metabolism, 93, 3640–3643.West, N. R., Dorling, J., Thackray, A. E., Hanson, N. C.,\\nDecombel, S. E., Stensel, D. J., & Grice, S. J.(2018).Effect of\\nobesity-linkedFTO rs9939609 variant on physical activity and\\ndietary patterns in physically active men and women.Journal\\nof Obesity, 2018, 7560707.World Health Organization.(2000).Obesity: Preventing and managing the global epidemic: Report of a WHO consultation (WHO\\nTechnical Report Series 894)(pp.1–253).\\n</SENT 18_02>\\n<SENT 18_03>\\nWorld Health Organ\\nTech Rep Ser.Geneva: WHO.World Health Organization.(2018).Obesity and overweight.Fact\\nSheet No.311.Yoshioka, M., Ayabe, M., Yahiro, T., Higuchi, H., Higaki, Y., StAmand, J., … Tanaka, H. (2005).Long-period accelerometer\\nmonitoring shows the role of physical activity in overweight\\nand obesity.International Journal of Obesity, 29, 502–508.Zhang, X., Qi, Q., Zhang, C., Smith, S. R., Hu, F. B., Sacks, F. M.,…\\nQi, L. (2012).FTO genotype and 2-year change in body composition and fat distribution in response to weight-loss diets: The\\npounds lost trial.Diabetes, 61, 3005–3011.SUPPORTING INFORMATION\\nAdditional supporting information may be found online\\nin the Supporting Information section at the end of this\\narticle.How to cite this article:Cho H-W, Jin H-S,\\nEom Y-B.The interaction betweenFTO rs9939609\\nand physical activity is associated with a 2-fold\\nreduction in the risk of obesity in Korean\\npopulation.Am J Hum Biol.2020;e23489.https://\\ndoi.org/10.1002/ajhb.23489\\n10 of 10 CHO ET AL.\\n</SENT 18_03>\\n\",\n",
       "  ['1. What are the combined effects of FTO rs9939609 and MC4R rs17782313 on obesity and BMI in Chinese Han populations?',\n",
       "   '2. How does physical activity attenuate the influence of FTO variants on obesity risk in adults and children?',\n",
       "   '3. What are the interactive influences of FTO genotype and physical activity on obesity?',\n",
       "   '4. What is the relationship between leisure-time physical activity, occupational activity, and the prevalence of obesity?',\n",
       "   '5. Does the FTO rs9939609 polymorphism modify the relationship between body mass index and affective symptoms through the life course?']),\n",
       " ('<SENT 19_01>\\nFood Reinforcement, the Dopamine D2 Receptor Genotype, and Energy\\nIntake in Obese and Nonobese Humans\\nLeonard H. Epstein, Jennifer L. Temple, Brad J. Neaderhiser, Robbert J. Salis, Richard W. Erbe, and\\nJohn J. Leddy\\nUniversity at Buffalo, The State University of New York\\nThe authors measured food reinforcement, polymorphisms of the dopamine D2 receptor (DRD2) and\\ndopamine transporter (DAT1) genes, and laboratory energy intake in 29 obese and 45 nonobese humans\\n18–40 years old.\\n</SENT 19_01>\\n<SENT 19_02>\\nFood reinforcement was greater in obese than in nonobese individuals, especially in\\nobese individuals with theTaqI A1 allele.Energy intake was greater for individuals high in food\\nreinforcement and greatest in those high in food reinforcement with theTaqI A1 allele.No effect of the\\nDAT1 genotype was observed.These data show that individual differences in food reinforcement may be\\nimportant for obesity and that theDRD2 genotype may interact with food reinforcement to influence\\nenergy intake.\\n</SENT 19_02>\\n<SENT 19_03>\\nKeywords: food reinforcement, energy intake, obesity, dopamine, dopamine receptor\\nEating is a highly reinforcing activity (Wise, 2006), and there\\nare individual differences in the reinforcing efficacy of food that\\nmay relate to differences in eating and energy intake (Epstein,\\nLeddy, Temple, & Faith, in press).In the same way that the\\nreinforcing efficacy of a drug is related to drug consumption\\n(Bickel, Marsch, & Carroll, 2000), subjects who find food highly\\nreinforcing may consume more energy in an ad libitum eating\\nsituation than those who are low in food reinforcement (Epstein et\\nal., 2004a).\\n</SENT 19_03>\\n<SENT 19_04>\\nIf individual differences in food reinforcement are\\nrelated to differences in energy intake, then obesity, which is\\ncharacterized by excess energy intake, may be related to food\\nreinforcement.Obese individuals may find food more reinforcing\\nand may be more motivated to eat than normal weight individuals\\n(Saelens & Epstein, 1996).There have been limited experimental tests of the hypothesis\\nthat high levels of food reinforcement lead to greater energy intake\\nand that obese persons are more motivated to obtain food than\\nnonobese persons.Initial research showed that smokers high in\\nfood reinforcement consumed more food than smokers low in food\\nreinforcement in an ad libitum snack food eating task (Epstein et\\nal., 2004a).\\n</SENT 19_04>\\n<SENT 19_05>\\nThis study had two important limitations.First, the\\nstudy used a questionnaire version of a food reinforcement task\\n(Goldfield, Epstein, Davidson, & Saad, 2006) rather than directly\\nmeasuring the amount of work subjects would do to gain access to\\nfood, which is the more sensitive methodology for measuring food\\nreinforcement.Second, only smokers were studied, and smokers\\ngenerally weigh less and therefore may consume less food than\\nnonsmokers (Klesges, Meyers, Klesges, & LaVasque, 1989); further, smokers may have a different level of food reinforcement\\nthan nonsmokers.There has been one experimental test of the\\nhypothesis that food is more reinforcing for obese than for lean\\npersons (Saelens & Epstein, 1996).\\n</SENT 19_05>\\n<SENT 19_06>\\nThis study showed that obese\\nyoung women responded more for access to food than leaner\\nyoung women, but the study was small (N /H1100520) and did not\\ninclude men or older subjects.Because men and women may differ\\nin how food reinforcement relates to energy intake (Epstein et al.,\\n2004a), it is important to study how individual differences in food\\nreinforcement are related to obesity in both men and women.The reinforcing value of food is related to activity of the\\ndopaminergic system.Food consumption increases brain dopamine levels in animals (Hernandez & Hoebel, 1988, 1990) and\\nhumans (Small, Jones-Gotman, & Dagher, 2003).\\n</SENT 19_06>\\n<SENT 19_07>\\nModifying brain\\ndopamine levels influences energy intake, with dopamine agonists\\nreducing energy intake (Leddy et al., 2004) and dopamine antagonists increasing energy intake and body weight (Wellman, 2005).Individual differences in food reinforcement may be related to\\nindividual differences in dopaminergic activity.Dopamine activity\\nis related to both the density of dopamine receptors and the amount\\nof the dopamine transporter.Thus, one way to indirectly study\\nindividual differences in brain dopamine levels is by studying\\npolymorphisms in dopamine receptor and transporter genes.\\n</SENT 19_07>\\n<SENT 19_08>\\nFor\\nexample, the presence of theTaqI A1 allele has been associated\\nwith a 30%–40% reduction in the density of the dopamine D2\\nreceptor (DRD2) and weaker dopamine signaling (Jonsson et al.,\\n1999; Pohjalainen et al., 1998; Ritchie & Noble, 2003).The\\n10-repeats allele (10R) of the dopamine transporter (DAT1) variLeonard H. Epstein, Jennifer L. Temple, Robbert J. Salis, and Richard\\nW. Erbe, Department of Pediatrics, University at Buffalo, The State University of New York; Brad J. Neaderhiser, Department of Rehabilitation\\nMedicine, University at Buffalo, The State University of New York; John\\nJ. Leddy, Department of Orthopaedics, University at Buffalo, The State\\nUniversity of New York.\\n</SENT 19_08>\\n<SENT 19_09>\\nLeonard H. Epstein is a consultant to Kraft foods.The other authors do\\nnot have any potential conflicts of interest.This research was funded in part\\nby a grant from the National Institute of Child Health and Human Development (R01 HD 39778) to Leonard H. Epstein.Correspondence concerning this article should be addressed to Leonard\\nH. Epstein, Department of Pediatrics, School of Medicine and Biomedical\\nSciences, University at Buffalo, The State University of New York, Farber\\nHall, Room G56, 3435 Main Street, Building 26, Buffalo, NY 14214-3000.\\n</SENT 19_09>\\n<SENT 19_10>\\nE-mail: lhenet@acsu.buffalo.edu\\nCORRECTED NOVEMBER 21, 2007; SEE LAST PAGE\\nBehavioral Neuroscience Copyright 2007 by the American Psychological Association\\n2007, Vol.121, No.5, 877–886 0735-7044/07/$12.00 DOI: 10.1037/0735-7044.121.5.877\\n877\\nThis document is copyrighted by the American Psychological Association or one of its allied publishers.This article is intended solely for the personal use of the individual user and is not to be disseminated broadly.\\n</SENT 19_10>\\n\\n\\n<SENT 20_01>\\nable number of tandem repeats (VNTR) region (SLC6A3 3 /H11032untranslated region) is related to increased expression ofDAT1\\nprotein, leading to greater dopamine reuptake and lower synaptic\\nlevels of dopamine (Heinz et al., 2000).Preliminary data on the interaction of these dopamine genotypes\\nas markers of individual differences in dopaminergic activation\\nand food reinforcement showed that smokers, most of whom were\\nnonobese, who were high in food reinforcement in combination\\nwith theTaqI A1 allele of theDRD2 or the absence of the 9R allele\\nof DAT1 had increased energy intake (Epstein et al., 2004b).\\n</SENT 20_01>\\n<SENT 20_02>\\nTo\\nour knowledge, this was the first demonstration that the interaction\\nof a dopamine genotype and a food reinforcement phenotype\\ninfluences energy intake.However, this interaction needs to be\\nextended beyond nonobese smokers.Nicotine produces reliable\\nincreases in brain dopamine levels and reduces food intake (M. D.\\nLi, Kane, & Konu, 2003; Miyata, Meguid, Fetissov, Torelli, &\\nKim, 1999).Long-term exposure to smoking up-regulates dopamine transporter activity (S. Li et al., 2004) andDRD2 (Bahk, Li,\\nPark, & Kim, 2002), but it lessens the up-regulation ofDRD2 by\\nother factors (Janson, Hedlund, Fuxe, & von Euler, 1994).\\n</SENT 20_02>\\n<SENT 20_03>\\nTherefore, it is possible that the interaction of the dopamine genotypes\\nand the food reinforcement is the result of smoking’s influence on\\ndopaminergic systems and may not replicate in nonsmokers.The primary aim of this study was to test two hypotheses\\nderived from reinforcement theory: whether obesity is related to\\nindividual differences in food reinforcement and whether individual differences in food reinforcement are related to differences in\\nenergy intake in an ad libitum eating task.\\n</SENT 20_03>\\n<SENT 20_04>\\nA secondary aim of the\\nstudy was to extend previous findings (Epstein et al., 2004b) on the\\ninteraction of food reinforcement with theDRD2 and DAT1 dopamine genotypes in nonsmokers to nonobese and obese nonsmokers.Another secondary aim was to compare food reinforcement\\nand food hedonics as predictors of energy intake.Our preliminary\\ndata in smokers suggest that the reinforcing value of food is a\\nstronger determinant of energy intake than food liking (Epstein, et\\nal., 2004a) and we want to determine if the same is true for\\nnonsmokers.Method\\nParticipants\\nSeventy-four participants were studied, 29 obese (body mass\\nindex [BMI] /H1135030) and 45 nonobese (BMI/H1102130) nonsmoking\\nadults between the ages of 18 and 40 (M/H1100525.7 years,SD /H110057.0).\\n</SENT 20_04>\\n<SENT 20_05>\\nOf the participants, 18.9% were minorities and 44.5% were male.The distribution ofDRD2 genotypes was equal (presence of A1\\nallele /H1100537, absence of A1 allele/H1100537); distribution of theDAT1\\ngenotypes was 31 participants with a 9R allele and 41 without a 9R\\nallele.DAT1 genotypes were unable to be determined for 2 participants.Participants were excluded if they were taking medication associated with a loss of appetite, were smokers, had diabetes,\\nhad previously been diagnosed with an eating disorder, were\\nallergic to ingredients in the study foods, were currently dieting,\\nfound the study foods aversive, or did not normally eat breakfast\\nand lunch.Procedures\\nParticipants visited the laboratory for two sessions separated by\\n2–7 days (see Table 1).\\n</SENT 20_05>\\n<SENT 20_06>\\nThe first session involved completion of\\nconsent and demographic forms and the ad libitum snack-eating\\ntask; the second session involved the food reinforcement task.A\\nsubset of participants (n/H1100520) completed a third session 2–7 days\\nlater identical to the second session to determine the test–retest\\nreliability of the food reinforcement phenotype.Experimental sessions were run during a typical lunch period, at least 3 hr postprandial.All participants consumed the same breakfast each day\\nand were provided with a 150-kcal preload (18% fat, 17% protein,\\n65% carbohydrates) to minimize the effects of hunger on food\\nreinforcement.\\n</SENT 20_06>\\n<SENT 20_07>\\nThe inclusion of a standard preload increases the\\nability to show individual differences in food reinforcement (Reiss\\n& Havercamp, 1996).At the beginning of each session, participants completed a same-day dietary recall to ensure adherence to\\nthe dietary instructions.Ad libitum eating task.The ad libitum food consumption task\\nwas presented as a taste test.Participants were presented 480–\\n500-kcal servings of six palatable high-fat, high-carbohydrate,\\nlow-protein snack foods (percentages of fat, carbohydrates, and\\nprotein shown in parentheses): Lay’s Potato Chips (57%, 38%,\\n5%); Doritos (44%, 50.4%, 5.6%); M&M’s (38.8%, 57.4%, 3.8%);\\nTwix (45%, 50.0%, 5.0%); Kit Kat (45.2%, 49.3%, 5.5%); and\\nButterfinger (35.2%, 60.3%, 4.5%).Water was provided ad libitum.Participants rated each food on a number of different characteristics using 9-point Likert-type scales, including pleasurability, sweetness, saltiness, blandness, flavorfulness, and bitterness.They were told that they could consume as little or as much of the\\nfood as they wished as long as they tasted each one so that they\\ncould rate its characteristics.\\n</SENT 20_07>\\n<SENT 20_08>\\nParticipants were then given three\\neating questionnaires to complete.Food from the taste test was left\\nin the room, and participants were told they could continue to eat\\nif they wished, as the food would be discarded after the session.When participants indicated that they were finished, they were\\nTable 1\\nStudy Design\\nPhase\\nSession\\n1 23\\nN 74 74 20 (of 74)\\nBreakfast Same breakfast each day\\nPreload Same 150-kcal preload each day\\nLunch task Ad libitum eating taskReinforcement task (1) Reinforcement task (2)\\nNote.Sessions were 2–7 days apart.878 EPSTEIN ET AL.This document is copyrighted by the American Psychological Association or one of its allied publishers.This article is intended solely for the personal use of the individual user and is not to be disseminated broadly.\\n</SENT 20_08>\\n',\n",
       "  ['1. How does the TaqI A1 allele affect the density of the dopamine D2 receptor and weaker dopamine signaling?',\n",
       "   '2. What is the relationship between the presence of the 10-repeats allele (10R) of the dopamine transporter (DAT1) variLeonard H. Epstein, Jennifer L. Temple, Robbert J. Salis, and Richard',\n",
       "   'W. Erbe, Department of Pediatrics, University at Buffalo, The State University of New York; Brad J. Neaderhiser, Department of Rehabilitation',\n",
       "   'Medicine, University at Buffalo, The State University of New York; John',\n",
       "   'J. Leddy, Department of Orthopaedics, University at Buffalo, The State']),\n",
       " ('<SENT 21_01>\\nasked to identify their favorite food from among the six and told\\nthat this was the food that would be used in the next session.Once\\nthe questionnaires were complete, participants’ cheek epithelial\\ncells were collected using a buccal brush for DNA analysis, and\\nparticipants were given a reminder for their next visit.Food reinforcement task.The reinforcing value of food was\\nmeasured by determining the number of responses on a concurrent\\nschedule task that participants made for food or food alternatives.\\n</SENT 21_01>\\n<SENT 21_02>\\nThe experimental environment included two computer stations\\nwith a swivel chair in the middle.At one station participants could\\nearn points toward food, and at the other station they could earn\\npoints for time to spend reading theBuffalo News.This alternative\\nactivity was provided to reduce the likelihood that participants\\nwould engage in responding out of boredom.Participants earned access to food or the alternative on concurrent variable-ratio (VR) schedules of reinforcement.\\n</SENT 21_02>\\n<SENT 21_03>\\nThe reinforcement schedule for food was a progressive VR schedule with\\nresponse requirements of 4, 8, 16, 32, 64, 128, 256, 512, 1,024, and\\nso forth, on the average, for each point.After 5 points were earned,\\nthe participant received a 100-kcal portion of his or her preferred\\nsnack food selected during Session 1.The task was identical on the\\ncomputer on which reading time could be earned, but the schedule\\nremained at VR4 throughout, and after 5 points were earned, the\\nparticipant received access to reading for 2 min.The reinforcement\\ntask was similar to a slot machine.\\n</SENT 21_03>\\n<SENT 21_04>\\nWhen the left button on the\\nmouse was pressed, three boxes containing different colored\\nshapes revolved, and when all of the shapes matched in shape and\\ncolor, the participant earned 1 point.Participants were instructed\\nthat the session would end when they no longer wished to earn\\npoints for access to food or reading.Water was provided ad\\nlibitum.Laboratory environment.The laboratory was specially constructed for eating experiments.It was equipped with an air delivery system that circulated new air through each room approximately 10 times per hour.The laboratory rooms were also\\nequipped with high-efficiency particulate air purifiers containing a\\ncarbon–permanganate–zeolite filter to remove airborne odorants.Genotyping.\\n</SENT 21_04>\\n<SENT 21_05>\\nDNA was extracted from the buccal samples\\nusing a commercially available genomic DNA quick preparation\\nkit (Gentra Systems, Qiagen, Inc., Valencia, CA), yielding 20/H9262l of\\nDNA at a concentration of 15–40 ng//H9262l.After DNA purification,\\neach sample was assigned an accession number and stored at –20\\n°C for later analysis.For detection of theTaqI A1 polymorphism in theDRD2 gene,\\na region of 304 base pairs (bp) was amplified.The primers first\\ndescribed by Grandy et al.\\n</SENT 21_05>\\n<SENT 21_06>\\n(Grandy et al., 1989) were modified to\\nsense 5/H11032-CCC TTC CTG AGT GTC ATC A-3/H11032and antisense\\n5/H11032-CGG CTG GCC AAG TTG TCT-3/H11032.The presence of the\\namplicon was confirmed by electrophoresis on a 1% agarose gel.The restriction endonucleaseTaqI A1 digests the 304-bp amplicon,\\nand subsequently the fragments are separated on a 6% polyacrylamide gel by electrophoresis.TheTaqI A1 polymorphism in the\\nDRD2 gene at Position 32806 T to C creates a restriction site,\\nresulting in partition of the 304 bp into a fragment of 177 and 127\\nbp.\\n</SENT 21_06>\\n<SENT 21_07>\\nThe A1/A1 or TT variant therefore is represented by an uncut\\namplicon of 304 bp; the A1/A2 or TC heterozygous form digests\\nin three fragments of 304, 177, and 127 bp; and the A2/A2 or CC\\nvariant is characterized by two fragments of 177 and 127 bp.TheDRD2 was coded for the allele patterns of A1/A1, A1/A2,\\nand A2/A2.Analyses were performed combining the A1/A1 and\\nA1/A2 patterns, comparing presence or absence of the A1 allele.Analysis of DAT1 gene focused on the VNTR of a 40-bp\\nsequence, which recurs 3 to 11 times.\\n</SENT 21_07>\\n<SENT 21_08>\\nPrimers first described by\\nVandenbergh et al.(Vandenbergh et al., 1992) were modified to\\nsense 5/H11032-GGT GTA GGG AAC GGC CTG AG-3/H11032and antisense\\n5/H11033-CTG GAG GTC ACG GCT CAA GG-3/H11032.The amplicon was\\nanalyzed on a 10% polyacrylamide gel by electrophoresis.The\\nlarge increment of 40 bp provides distinct typing of the VNTRs.Each run of participant DNA samples included sequenced control\\nDNA samples and a negative control.TheDAT1 was coded for the\\nallele patterns of 9R/9R, 9R/10R, and 10R/10R.Analyses were\\nperformed combining the 9R/9R and 9R/10R patterns, comparing\\npresence or absence of the 9R allele.Dietary recalls.\\n</SENT 21_08>\\n<SENT 21_09>\\nAt the beginning of each session, participants\\nwere asked to recall what foods and beverages they had consumed\\nthe day of the experiment to ensure adherence to the experimental\\nprotocol.Participants were guided through the recall process and\\nrecorded each food item recalled as well as the portion size,\\ncondiments, added fats, and added sugars.Measuring cups and\\nspoons were provided to help the participants estimate portion\\nsizes.Demographics.A general demographics questionnaire was\\nused to assess participants’ education status, race, and ethnicity.Anthropometrics.Height (cm) and weight (lb) were measured\\nusing a Digi-Kit digital stadiometer (Digi-Kit, North Bend, WA)\\nand a Tanita digital weight scale (Tanita, Arlington Heights, IL),\\nand used to calculate BMI (kg/m2).Individuals were considered\\nobese if their BMI was at least 30 kg/m2 and nonobese if their BMI\\nwas less than 30 kg/m2 (NHLBI Obesity Education Initiative\\nExpert Panel, 1998).\\n</SENT 21_09>\\n<SENT 21_10>\\nEating questionnaires.Participants completed the ThreeFactor Eating Questionnaire (French, Jeffery, & Wing, 1994;\\nStunkard & Messick, 1985), the Questionnaire of Eating and\\nWeight Patterns (Spitzer et al., 1992), and the Binge Eating\\nScale (Gormally, Black, Daston, & Rardin, 1982).The ThreeFactor Eating Questionnaire has three subscales that assess\\ndietary restraint, hunger, and disinhibition.The Questionnaire\\nof Eating and Weight Patterns and Binge Eating Scale are used\\nto assess binge eating disorder.\\n</SENT 21_10>\\n<SENT 21_11>\\nParticipants were classified as\\ndietary restrained if they scored higher than 12 on the Dietary\\nRestraint subscale of the Three-Factor Eating Questionnaire.Participants were identified as potentially having binge eating\\ndisorder if they scored higher than 27 on the Binge Eating Scale\\nor were indicated as having the disorder by the Questionnaire\\non Eating and Weight Patterns.Any participant identified as\\npotentially having binge eating disorder was required to complete the Eating Disorders Examination (Bryant-Waugh, Cooper, Taylor, & Lask, 1996), administered by trained personnel\\nin an additional session.No participants met criteria for binge\\neating disorder.\\n</SENT 21_11>\\n<SENT 21_12>\\nFood liking, hunger, and fullness.Both before and after the\\nfood reinforcement task, participants rated how hungry they felt on\\na 100-mm visual analogue scale anchored bynot hungry at alland\\nextremely hungry, and how much they liked the snack food reinforcer on a 100-mm visual analogue scale anchored bynot like at\\nall and like very much.879FOOD REINFORCEMENT AND ENERGY INTAKE\\nThis document is copyrighted by the American Psychological Association or one of its allied publishers.This article is intended solely for the personal use of the individual user and is not to be disseminated broadly.\\n</SENT 21_12>\\n\\n\\n<SENT 22_01>\\nAnalytic Plan\\nDifferences in participant characteristics by genotypes and by\\nobesity were assessed using one-way analysis of variance for\\ncontinuous variables and chi-square tests for categorical variables.Differences in patterns of operant responding for food were determined using hierarchical mixed-effects regression models (MRM),\\nwhich allow for regression analyses to test predictors of repeated\\ndependent measures (Hedeker & Gibbons, 2006).The first model\\nused responding for each schedule of reinforcement as the dependent variable, with obesity, gender, age, and binge eating score as\\ntime-invariant independent predictors and schedule of reinforcement as the time-variant predictor.Both linear (Obesity/H11003Schedule of Reinforcement) and quadratic (Obesity /H11003Schedule /H11003\\nSchedule of Reinforcement) patterns were tested.\\n</SENT 22_01>\\n<SENT 22_02>\\nQuadratic patterns were tested because the pattern of responding represents an\\nincrease in responding as the schedules increase, followed by a\\nreduction in responding as the break point for each participant is\\nreached.Age, gender, and binge eating score were tested as moderators of the obesity effect by interacting the potential moderator\\nwith obesity using methods recommended by Kraemer and colleagues (Kraemer, Frank, & Kupfer, 2006).The next step was to\\nadd the dopamine genotypes/H11003schedules of reinforcement (including linear and quadratic terms) as predictors of responding in\\nseparate analyses and assess whether adding information on either\\ngenotype tested separately improved the fit of the model by using\\nchi-square to test the change in the log-likelihood ratio.The final\\nmodels tested the interaction of obesity and each of the two\\ndopamine genotypes/H11003schedules of reinforcement as predictors of\\nresponding.\\n</SENT 22_02>\\n<SENT 22_03>\\nThus, the first model tested whether obesity was\\nrelated to responding for food, whereas the subsequent models\\ntested whether knowledge about the genotype(s) improved the fit\\nof the model.Changes in hunger and food liking were tested using mixed\\nanalysis of covariance (ANCOVA), with obesity and theDRD2\\nand DAT1 genotypes as between-subjects variables, and time of\\nassessment (pre- or posttask) as the within-subject variable.Variables that were different between groups, age and binge eating\\nscore, were used as covariates, in addition to gender.Linear\\ncontrasts were used to compare differences in the pattern of change\\nover time.\\n</SENT 22_03>\\n<SENT 22_04>\\nDifferences in energy intake for food were determined\\nusing a factorial ANCOVA with food reinforcement and theDRD2\\nand DAT1 genotypes as the between-subjects variables, with age,\\nbinge eating, and sex as covariates.Responding at or below the\\nVR32 schedule was used to differentiate low and high responding\\non the basis of the median of the distribution of responding.Contrasts were used to compare differences in energy intake by\\ngroup.Outliers were determined using exploratory data analysis,\\nand one outlier (greater than three standard deviations from the\\nmean) for energy intake was removed from the analysis.\\n</SENT 22_04>\\n<SENT 22_05>\\nInitial\\nanalyses used both genotypes to examine the interaction between\\nthe genotypes.Subsequent analyses were performed separately for\\neach genotype, as there was no interaction between the genotypes,\\nand to maximize the sample for each genotype because there were\\ntwoDAT1 genotypes that could not be determined.Food reinforcement and food liking were assessed as determinants of energy\\nintake using a multiple regression model including both of these\\npredictors, as well as age, sex, and binge eating.The test–retest\\nreliability of food reinforcement was assessed by a correlation of\\nthe break points in each measure.\\n</SENT 22_05>\\n<SENT 22_06>\\nResults\\nParticipant Characteristics\\nThere were no differences between the groups for education,\\npercentage of minority participants, food pleasurability, or hunger\\n(p /H11022.05; see Table 2).There was an effect of weight status on age\\nand binge eating score, where the obese individuals were older,\\nF(1, 72)/H1100514.60, p /H11021.0001, and scored higher on the Binge\\nEating Scale, F(1, 72) /H1100519.81, p /H11021.0001, than the nonobese\\nindividuals.\\n</SENT 22_06>\\n<SENT 22_07>\\nParticipants with the A2/A2 genotype had higher\\nbinge eating scores,F(1, 72)/H110054.79, p /H11021.05, than those with the\\nA1/A1 or A1/A2 genotype.Weight Status, Genotype, and Food Reinforcement\\nMRM showed that obesity was a significant predictor of responding for food for both linear (p /H11005.004) and quadratic (p /H11005\\n.033) trends, which improved the fit of the basic model (/H92732(3) /H11005\\n15.49, p /H11005.0014), controlling for age, gender, and binge eating\\nstatus.\\n</SENT 22_07>\\n<SENT 22_08>\\nAnalyses did not show that these variables interacted with\\nobesity to moderate responding for food.The relationships between obesity and responding for food across the schedules of\\nreinforcement are shown in the top graph of Figure 1.TheDRD2\\ngenotype was related to linear (p /H11005.022) but not quadratic\\nresponding for food (p /H11005.009; middle graph, Figure 1), which\\nimproved the fit of the basic model (/H92732(3) /H110059.15, p /H11005.027), as\\nparticipants with the A1 allele responded more for food than those\\nwithout the A1 allele.In addition, theDRD2 genotype interacted\\nwith obesity to predict linear (p /H11021.001) and quadratic (p /H11005.011)\\ntrends for responding for food, improving fit of the obesity/H11003\\nschedules model (/H92732(5) /H1100525.70, p /H11021.0001).\\n</SENT 22_08>\\n<SENT 22_09>\\nThese relationships,\\nshown in the bottom graph of Figure 1, were explored by separate\\nMRM analyses of participants with and without the A1 allele.Model fit was not improved by adding the obesity/H11003schedule\\ninteraction for participants without the A1 allele (/H92732(3) /H110051.63, p\\n/H11022.05), but both linear (p /H11021.001) and quadratic (p /H11005.011)\\ndifferences in responding were observed for obese and nonobese\\nparticipants with the A1 allele, improving fit of the model (/H92732(3)\\n/H1100520.55, p /H11021.0001), as obese subjects with the A1 allele responded more for food.\\n</SENT 22_09>\\n<SENT 22_10>\\nAdding theDAT1 genotype /H11003schedule\\ninteraction did not improve fit of the basic model,/H92732(3) /H110054.12, p\\n/H11022.05, and adding the interaction ofDAT1 genotype /H11003obesity /H11003\\nschedules did not improve the fit of the obesity /H11003schedules\\nmodel, /H92732(5) /H110059.32, p /H11022.05.Genotype, Food Reinforcement, and Energy Intake\\nThe results for the ad libitum energy intake task (Figure 2)\\nshowed a significant main effect for food reinforcement, F(1,\\n66) /H110059.40, p /H11005.003, as those with high levels of food reinforcement consumed more energy (686.6 /H11006336.4 kcal, M /H11006SD)\\ncompared with those with low levels of food reinforcement\\n(494.4/H11006194.5 kcal).\\n</SENT 22_10>\\n<SENT 22_11>\\nIn addition, an interaction between the food\\nreinforcement phenotype and theDRD2 genotype, F(1, 66)/H110056.22,\\np /H11005.015, was observed.Contrasts showed that those high in food\\nreinforcement with the A1 allele consumed more energy (774.4/H11006\\n880 EPSTEIN ET AL.This document is copyrighted by the American Psychological Association or one of its allied publishers.This article is intended solely for the personal use of the individual user and is not to be disseminated broadly.\\n</SENT 22_11>\\n',\n",
       "  [\"1. How were the participants' favorite foods chosen for the next session?\",\n",
       "   '2. What was the purpose of providing an alternative activity during the food reinforcement task?',\n",
       "   '3. Can you explain the progressive variable-ratio schedule used for earning food points?',\n",
       "   \"4. How was the laboratory environment designed to minimize external factors that may influence participants' food choices?\",\n",
       "   '5. What was the purpose of collecting cheek epithelial cells from participants?']),\n",
       " ('<SENT 23_01>\\n283.2 kcal) than those low in food reinforcement either with\\n(430.1 /H11006158.5 kcal,p /H11005.0002) or without (558.7/H11006227.7 kcal,\\np /H11005.0005) the A1 allele and those high in food reinforcement\\nwithout the A1 allele (598.8/H11006362.1 kcal,p /H11005.037).In analyses\\nincluding the DAT1 genotype, no interaction ofDAT1 genotype\\nwith food reinforcement (p /H11005.92), main effect ofDAT1 ( p /H11005.83),\\ninteraction betweenDRD2 and DAT1 ( p /H11005.96), or interaction of\\nDRD2, DAT1, and food reinforcement (p /H11005.83) was observed.\\n</SENT 23_01>\\n<SENT 23_02>\\nA significant decrease in hunger (51.9/H1100623.9 to 26.4/H1100623.1)\\nwas observed after food consumption,F(1, 64)/H110059.89, p /H11005.002,\\nbut no interaction of hunger reduction with weight status, or either\\nof the dopamine genotypes, was found.No significant changes in\\nfood liking were observed over time (75.3/H1100622.0 to 63.8/H1100625.2)\\nor by weight or genotype status.Reinforcing Value of Food, Food Liking, and Energy\\nIntake\\nRegression analysis showed that the reinforcing value of food\\nwas a significant predictor of energy intake (p /H11005.002) but that\\nself-reported liking of the favorite food was not (p /H11005.72), controlling for sex, age, and binge eating scores.The univariate\\ncorrelation between reinforcing value and energy intake was .40\\n(p /H11005.0004), whereas the univariate correlation between liking and\\nenergy intake was .02 (p /H11005.87).\\n</SENT 23_02>\\n<SENT 23_03>\\nReliability of Study Measures\\nThere was a significant positive correlation between the break\\npoints in the repeated measures of food reinforcement (n /H1100520, r /H11005\\n.80, p /H11021.05).The regression line for predicting the break point in\\nSession 2 from the break point in Session 1 had a slope of 1.00 and\\nan intercept of –0.18.Discussion\\nThe results support the hypotheses that individuals higher in\\nfood reinforcement will consume more food in an ad libitum eating\\nsituation than those lower in food reinforcement and that obese\\nindividuals are higher in food reinforcement than those who are\\nnonobese.These results are consistent with previous research in\\nsmokers showing that subjects high in food reinforcement consume more food than those lower in food reinforcement (Epstein et\\nal., 2004b) and that subjects who find food more reinforcing\\nconsume more energy than those less motivated to eat (Johnson,\\n1974; Saelens & Epstein, 1996).\\n</SENT 23_03>\\n<SENT 23_04>\\nThese results provide support for the importance of studying\\nfood reinforcement as a contributor to obesity.Food is a powerful\\nreinforcer that can be as reinforcing as drugs of abuse (Hursh &\\nBauman, 1987).The motivation to eat and food reinforcement can\\nbe increased by food deprivation (Raynor & Epstein, 2003), as\\nwell as by food variety (Temple, Giacomelli, Roemmich, & Epstein, in press).Chronic food deprivation may sensitize the brain to\\nincreases in food reinforcement (Carr, 1996).\\n</SENT 23_04>\\n<SENT 23_05>\\nConceptualizing\\novereating as increased motivation to eat because of increased\\nfood reinforcement is similar to conceptualizations of drug abuse\\nthat focus on the positive reinforcing effects of drug selfadministration in maintaining drug abuse (Bickel, Madden, &\\nPetry, 1998; Bickel et al., 2000).This approach provides the\\nopportunity to take advantage of a wealth of basic research that has\\nprovided new insights into how reinforcement may motivate behavior (Wise, 2006).\\n</SENT 23_05>\\n<SENT 23_06>\\nTable 2\\nParticipant Characteristics\\nVariable\\nObesity DRD2 SLC6A3\\nNonobese Obese A1/A1 & A1/A2 A2/A2 9/9 & 9/10 10/10\\nN (male/female) 45 (22/23) 29 (11/18) 37 (15/22) 37 (18/19) 31 (15/16) 41 (17/24)\\nAge (M /H11006SD) 23.4 /H110065.4a 29.3 /H110067.8a 26.1 /H110067.2 25.4 /H110066.8 26.7 /H110067.3 24.8 /H110066.6\\nBMI (kg/m2)( M /H11006SD) 23.2 /H110062.8 34.5 /H110064.6a 26.3 /H110066.7 29.0 /H110066.3 28.0 /H110066.2 27.0 /H110066.4\\nDietary restraint (M/H11006SD) 5.8 /H110063.9 6.9 /H110064.2 6.4 /H110064.1 6.1 /H110063.9 6.9 /H110064.2 5.9 /H110063.9\\nBinge Eating Scale (M/H11006SD) 6.8 /H110065.5 13.8 /H110068.0a 7.7 /H110066.5 11.4 /H110067.9b 10.3 /H110066.5 8.9 /H110067.7\\nBaseline hunger (M/H11006SD) 53.3 /H1100622.6 52.0 /H1100626.2 50.4 /H1100624.4 55.2 /H1100623.4 52.9 /H1100625.7 53.5 /H1100623.1\\nAverage food liking (M/H11006SD) 75.9 /H1100622.9 74.3 /H1100620.8 75.6 /H1100620.5 75.0 /H1100623.6 75.3 /H1100620.8 74.8 /H1100623.4\\nRace: n (%)\\nCaucasian 37 (82) 22 (76) 33 (89) 26 (70) 26 (84) 31 (76)\\nMinority 7 (16) 6 (21) 3 (8) 10 (27) 4 (13) 9 (22)\\nMixed race 1 (2) 1 (3) 1 (3) 1 (3) 1 (3) 1 (2)\\nEducation: n (%)\\nHigh school 28 (62) 17 (59) 20 (54) 25 (68) 18 (58) 25 (61)\\nCollege degree 17 (38) 12 (41) 17 (46) 12 (32) 13 (42) 16 (39)\\nDRD2 n (%)\\nA1/A1 and A1/A2 25 (56) 12 (41) 14 (45) 22 (54)\\nA2/A2 20 (44) 17 (59) 17 (55) 19 (46)\\nSLC6A3 n (%)\\n9/9 and 9/10 19 (43) 12 (43) 14 (39) 17 (47)\\n10/10 25 (57) 16 (57) 22 (61) 19 (53)\\nNote.\\n</SENT 23_06>\\n<SENT 23_07>\\nDietary restraint scores were derived from the Three-Factor Eating Questionnaire (Stunkard & Messick, 1985).DRD2 /H11005dopamine D2 receptor;\\nBMI /H11005body mass index.a Significant differences between obese and nonobese,p /H11021.001.b Significant differences betweenDRD2 genotypes, p /H11021.001.881FOOD REINFORCEMENT AND ENERGY INTAKE\\nThis document is copyrighted by the American Psychological Association or one of its allied publishers.This article is intended solely for the personal use of the individual user and is not to be disseminated broadly.\\n</SENT 23_07>\\n\\n\\n<SENT 24_01>\\nLean/Obese\\nResponses\\n0\\n100\\n200\\n300\\n400\\n500\\n600\\n700\\nLean\\nObese\\nDRD2 Genotype\\nResponses\\n0\\n100\\n200\\n300\\n400\\n500\\n600\\n700\\nA2/A2\\nA1/A1 and A1/A2\\nObesity x DRD2 Genotype\\nVariable Ratio Schedules of Reinforcement\\nResponses\\n0\\n100\\n200\\n300\\n400\\n500\\n600\\n700\\nLean A2/A2\\nObese A2/A2\\nLean A1/A1 and A1/A2\\nObese A1/A1 and A1/A2\\n4        8        16      32      64      128    256    512   1024\\n4        8        16      32      64      128    256    512   1024\\n4        8        16      32      64      128    256    512   1024\\n882 EPSTEIN ET AL.This document is copyrighted by the American Psychological Association or one of its allied publishers.This article is intended solely for the personal use of the individual user and is not to be disseminated broadly.\\n</SENT 24_01>\\n',\n",
       "  ['1. What is the relationship between food reinforcement and energy intake?',\n",
       "   '2. How does food deprivation affect food reinforcement and motivation to eat?',\n",
       "   '3. How does food variety influence food reinforcement and consumption?',\n",
       "   \"4. How does chronic food deprivation impact the brain's sensitivity to increases in food reinforcement?\",\n",
       "   '5. How does the conceptualization of overeating as increased motivation to eat due to increased food reinforcement compare to the conceptualization of drug abuse based on the positive reinforcing effects of drug self-administration?']),\n",
       " ('<SENT 25_01>\\nThere are several directions for research on food reinforcement\\nand obesity.One important question is whether individual differences in food reinforcement are risk factors for the development of\\nobesity, or whether a history of positive energy balance that leads\\nto obesity results in higher levels of food reinforcement.Prospective research on children is needed in which individual differences\\nin food reinforcement are measured in lean children and children\\nare followed over time.One of the biggest risk factors for the\\ndevelopment of obesity is parental obesity (Garn & Clark, 1976;\\nWhitaker, Wright, Pepe, Seidel, & Dietz, 1997), and one prediction is that individual differences in food reinforcement are shared\\nbetween parents and children.\\n</SENT 25_01>\\n<SENT 25_02>\\nConsistent with this hypothesis,\\nresearchers have recently shown a strong relationship between\\nparent and child levels of food reinforcement in 50 families with\\noverweight children (Epstein, Dearing, Temple, & Cavanaugh,\\n2007).Understanding individual differences in food reinforcement\\nsheds light on only one of the factors that influence eating, as\\neating represents a choice among behaviors (Epstein et al., in\\npress), and to understand how people make the choice to eat, it is\\nnecessary to understand the alternatives that are available.\\n</SENT 25_02>\\n<SENT 25_03>\\nOne\\npossibility is that the relative reinforcing value of eating is determined not only by the reinforcing value of food but also by the\\nreinforcing value of alternatives to eating (Epstein et al., in press).We have previously shown that nonfood alternatives and less\\npreferred foods can substitute for highly preferred foods when\\naccess to the preferred foods is reduced (Goldfield & Epstein,\\n2002).In order to better understand the role of food reinforcement\\nas a risk factor for obesity, it may be necessary to understand the\\nreinforcing value of alternatives to eating in obese people.It is\\npossible that for some people the reinforcing value of food may not\\nbe pronounced but that these people do not have reliable alternatives to food reinforcement, which results in greater relative food\\nreinforcement than reinforcement from alternatives to eating,\\nwhich could result in overeating.\\n</SENT 25_03>\\n<SENT 25_04>\\nThe results of this study replicate research indicating that the\\npresence of theTaqI A1 allele interacts with obesity to influence\\nfood reinforcement and interacts with food reinforcement to influence energy intake (Epstein et al., 2004b).The approach we have\\ntaken focuses on theoretical advances in understanding the neurobiology of the reinforcement process and studies genotypes that\\nrelate to or may be markers of basic neurobiological processes.\\n</SENT 25_04>\\n<SENT 25_05>\\nThe presence of theTaqI A1 allele of theDRD2 is associated with\\na lower density ofDRD2 (Jonsson et al., 1999; Pohjalainen et al.,\\nFigure 1 (opposite).Differences in motivated responding for snack foods across variable ratio schedules of\\nreinforcement for those who were obese (body mass index/H1102230) versus nonobese (top graph), for those with\\nversus without the dopamine D2 receptor (DRD2) A1 allele (middle graph), and for the combination of obesity\\nand the A1 allele (bottom graph).\\n</SENT 25_05>\\n<SENT 25_06>\\nMixed-effects regression models showed that obesity (p /H11005.0014) and the A1\\nallele (p /H11005.027) independently predicted responding for food.An interaction of obesity with theDRD2 genotype\\nwas observed, as obese participants with the A1 allele responded more for food than obese participants without\\nthe A1 allele (p /H11021.0001), whereas there were no significant differences in responding for nonobese participants\\nwith or without the A1 allele.\\n</SENT 25_06>\\n<SENT 25_07>\\nReinforcing value\\nhg i Hwo L\\nEnergy intake (kcals)\\n0\\n200\\n400\\n600\\n800\\n1000\\nA2/A2\\nA1/A1 and A1/A2\\nFigure 2.Participants’ energy consumption by food reinforcement value and presence or absence of the A1\\nallele ( p /H11005.003).Participants high in food reinforcement with the A1 allele consumed more food than\\nparticipants high in food reinforcement without the A1 allele ( p /H11005.037) and participants low in food\\nreinforcement with (p /H11021.0002) or without the A1 allele (p /H11005.0005).883FOOD REINFORCEMENT AND ENERGY INTAKE\\nThis document is copyrighted by the American Psychological Association or one of its allied publishers.This article is intended solely for the personal use of the individual user and is not to be disseminated broadly.\\n</SENT 25_07>\\n\\n\\n<SENT 26_01>\\n1998; Ritchie & Noble, 2003), and both theTaqI A1 allele (Comings et al., 1993; Noble et al., 1994) and reduced density ofDRD2\\n(Wang et al., 2001) have been associated with human obesity.Demonstrating that polymorphisms that are related to the density\\nofDRD2 receptors can affect food reinforcement, obesity, and\\nenergy intake provides a mechanistic explanation for how dopaminergic activity may influence ingestive behavior and obesity\\nrisk.An important limitation of the present study is the small\\nsample size and population admixture.Research designed to identify genes that are associated with or responsible for specific\\nbehaviors usually requires much larger sample sizes and control of\\npopulation stratification that can arise when multiple racial and\\nethnic groups are studied, and usually involves tests of a large\\nnumber of genes to provide data on the association between\\nspecific dopaminergic genes and food reinforcement.\\n</SENT 26_01>\\n<SENT 26_02>\\nA second limitation in the current study is that theTaqI A1\\nallele lies 10 kb downstream of theDRD2 gene and may reside in\\nthe coding region of a novel serine/threonine kinase gene (Neville,\\nJohnstone, & Walton, 2004).It is therefore unlikely that polymorphisms in theTaqI A1 have a direct influence onDRD2 expression.It is possible that theTaqI A1 allele is in linkage disequilibrium\\nwith functional variants of DRD2 and, thus, indirectly affects\\nDRD2 expression (Neville et al., 2004).The majority of studies\\nthat have examined associations between theTaqI A1 polymorphism and DRD2 density have shown significant reductions in\\nDRD2 in individuals who carry at least one copy of theTaqI A1\\n(Jonsson et al., 1999; Pohjalainen et al., 1998; Ritchie & Noble,\\n2003), though this is not universally accepted (Laruelle, Gelernter,\\n& Innis, 1998).\\n</SENT 26_02>\\n<SENT 26_03>\\nResearch showing that theTaqI A1 polymorphism\\nis not contained within theDRD2 gene constrains attributing food\\nreinforcement directly to the TaqI A1 polymorphism of DRD2\\ngenotype.Additional research is needed to identify the polymorphisms that directly lead to changes in the number ofDRD2\\nreceptors, and to link reductions in DRD2 density with energy\\nintake, food reinforcement, and weight status.Until these polymorphisms are identified, theTaqI A1 allele remains an indirect\\nmarker of DRD2 expression.Synaptic dopamine levels are related to dopamine transporter\\nactivity (Heinz et al., 2000).\\n</SENT 26_03>\\n<SENT 26_04>\\nIn previous research, smokers who\\nwere homozygous for the 10R allele and had high food reinforcement levels also consumed more energy than smokers who carried\\nat least one copy of the 9R allele or who had low food reinforcement levels (Epstein et al., 2004b).In the present study no interactions of thisDAT1 genotype and food reinforcement phenotype\\nwere observed, and there was no main effect of theDAT1 genotype\\non energy intake.The differences in effects of the dopamine\\ntransporter may be due to the use of smokers in the previous study.Smoking has effects on theDAT1 genotype (S. Li et al., 2004), and\\na long history of smoking may have altered the dopamine transport\\nsystem to increase sensitivity to food reinforcement.\\n</SENT 26_04>\\n<SENT 26_05>\\nWith current technology it is difficult to assess dynamic changes\\nin neurotransmitter activity in brains of behaving humans.Functional MRI studies can demonstrate changes in regional cerebral\\nblood flow during cognitive tasks in areas of the brain known to\\ncontain dopamine receptors but cannot yield information about\\nspecific neurotransmitter activity (Holsen et al., 2005).Positron\\nemission tomography (PET) scans provide the resolution to measure the density of availableDRD2 using receptor-specific radioligands (Elsinga, Hatano, & Ishiwata, 2006).However, this measurement is sensitive to potential fluctuations in endogenous\\ndopamine release that may occur as a result of the experimental\\nmanipulations and, thus, can differ depending on the context\\n(Adler et al., 2000; Leyton et al., 2002; Volkow et al., 2002).\\n</SENT 26_05>\\n<SENT 26_06>\\nIn\\naddition, raclopride, the most common radioligand used for the\\nstudy ofDRD2,i saDRD 2 antagonist and, thus, may have its own\\ninfluence on food reinforcement and impulsivity for reinforcers\\n(Hsiao & Smith, 1995; Wade, de Wit, & Richards, 2000).An\\nalternative approach to the direct measurement of dopaminergic\\nactivity is to measure individual differences in dopamine receptor\\ngenotypes as markers for dopaminergic activity in behaviorally\\nactive contexts.\\n</SENT 26_06>\\n<SENT 26_07>\\nPostmortem analysis of frontal cortex and caudate\\nnucleus (Ritchie & Noble, 2003) and PET studies have shown that\\nthe presence of at least one copy of theTaqI A1 allele is associated\\nwith a 30%–40% reduction in the density ofDRD2 (Jonsson et al.,\\n1999; Pohjalainen et al., 1998).Thus, it may be possible to use the\\ngenotype as a marker forDRD2 density and dopaminergic tone.This study did not consider physical activity and energy expenditure, but there may be separate or complementary biological\\nprocesses that interact with physical activity reinforcement phenotypes that may be associated with differences in energy expenditure.These patterns of behavioral and biological processes may be\\nindividual-difference characteristics, such that some people may be in\\npositive energy balance owing to overeating whereas others may be in\\npositive energy balance owing to relatively low energy expenditure\\nand still others may combine overeating with low activity levels.\\n</SENT 26_07>\\n<SENT 26_08>\\nThis study represents a novel, theoretically driven approach to\\nthe use of polymorphisms in genes that alter dopaminergic activity\\nto study biological influences on complex behavioral processes.The interaction ofDRD2 with other factors that control dopaminergic activity needs to be examined, along with how dopaminergic activity interacts with other neurotransmitters, such as opioids\\nor serotonin.The results of this study have important implications\\nfor the development and treatment of obesity.The ability to\\ncharacterize people as at risk for obesity on the basis of behavioral\\nand neurobiological factors provides the opportunity to develop\\ntreatment programs that are tailored to individuals with specific\\npatterns of risk factors.\\n</SENT 26_08>\\n<SENT 26_09>\\nCurrent obesity prevention programs have\\nnot been successful (Swinburn, Gill, & Kumanyika, 2005), and the\\napproach of focusing behavior change efforts on those at high risk\\nmay be better suited to prevention than less focused efforts at the\\ngeneral population, many of whom are at low risk.Obesity treatment programs generally provide a one-size-fits-all approach, with\\nthe same treatments for all participants.Weight loss and maintenance might be dramatically improved if treatment approaches\\nwould differentially focus on the individual characteristics of the\\nparticipants.It is easy to conceptualize that a prevention or treatment strategy for someone who is high in food reinforcement\\nwould be very different from the strategy for someone who is low\\nin food reinforcement but higher in activity reinforcement.\\n</SENT 26_09>\\n<SENT 26_10>\\nThe\\nidentification of individual differences in behavioral and neurobiological factors that are related to obesity may make the potential\\nfor individualizing prevention or treatment approaches to obesity a\\nreality.References\\nAdler, C. M., Elman, I., Weisenfeld, N., Kestler, L., Pickar, D., & Breier,\\nA.(2000).Effects of acute metabolic stress on striatal dopamine release\\nin healthy volunteers.Neuropsychopharmacology, 22, 545–550.884 EPSTEIN ET AL.This document is copyrighted by the American Psychological Association or one of its allied publishers.This article is intended solely for the personal use of the individual user and is not to be disseminated broadly.\\n</SENT 26_10>\\n',\n",
       "  ['1. How does the presence of the TaqI A1 allele interact with obesity to influence food reinforcement?',\n",
       "   '2. What are the alternatives to eating that could potentially substitute for highly preferred foods when access to the preferred foods is reduced?',\n",
       "   '3. How does the reinforcing value of alternatives to eating compare to the reinforcing value of food in obese people?',\n",
       "   '4. Are there any reliable alternatives to food reinforcement that could help reduce overeating in individuals with a pronounced reinforcing value of food?',\n",
       "   '5. What specific dopaminergic genes are associated with food reinforcement, obesity, and energy intake?']),\n",
       " ('<SENT 27_01>\\nBahk, J. Y., Li, S., Park, M. S., & Kim, M. O.(2002).Dopamine D1 and\\nD2 receptor mRNA up-regulation in the caudate-putamen and nucleus\\naccumbens of rat brains by smoking.Progress in Neuropsychopharmacology and Biological Psychiatry, 26,1095–1104.Bickel, W. K., Madden, G. J., & Petry, N. M. (1998).The price of change:\\nThe behavioral economics of drug dependence.Behavior Therapy, 29,\\n545–565.Bickel, W. K., Marsch, L. A., & Carroll, M. E. (2000).Deconstructing\\nrelative reinforcing efficacy and situating the measures of pharmacological reinforcement with behavioral economics: A theoretical proposal.Psychopharmacology, 153,44–56.\\n</SENT 27_01>\\n<SENT 27_02>\\nBryant-Waugh, R. J., Cooper, P. J., Taylor, C. L., & Lask, B. D. (1996).The use of the eating disorder examination with children: A pilot study.International Journal of Eating Disorders, 19,391–397.Carr, K. D. (1996).Feeding, drug abuse, and the sensitization of reward by\\nmetabolic need.Neurochemical Research, 21,1455–1467.Comings, D. E., Flanagan, S. D., Dietz, G., Muhleman, D., Knell, E., &\\nGysin, R. (1993).The dopamine D2 receptor (DRD2) as a major gene in\\nobesity and height.Biochemical Medicine and Metabolic Biology, 50,\\n176–185.\\n</SENT 27_02>\\n<SENT 27_03>\\nElsinga, P. H., Hatano, K., & Ishiwata, K. (2006).PET tracers for imaging\\nof the dopaminergic system.Current Medicinal Chemistry, 13,2139–\\n2153.Epstein, L. H., Dearing, K. K., Temple, J. L., & Cavanaugh, M. D. (2007).Food reinforcement and impulsivity in overweight children and their\\nparents.Manuscript submitted for publication.Epstein, L. H., Leddy, J. J., Temple, J. L., & Faith, M. S. (in press).Food\\nreinforcement and eating: A multilevel analysis.Psychological Bulletin.Epstein, L. H., Wright, S. M., Paluch, R. A., Leddy, J., Hawk, L. W.,\\nJaroni, J. L., et al.\\n</SENT 27_03>\\n<SENT 27_04>\\n(2004a).Food hedonics and reinforcement as determinants of laboratory food intake in smokers.Physiology & Behavior,\\n81, 511–517.Epstein, L. H., Wright, S. M., Paluch, R. A., Leddy, J. J., Hawk, L. W.,\\nJaroni, J. L., et al.(2004b).The relationship between food reinforcement\\nand dopamine genotypes on food intake in smokers.American Journal\\nof Clinical Nutrition, 80,82–88.French, S. A., Jeffery, R. W., & Wing, R. R. (1994).Food intake and\\nphysical activity: A comparison of three measures of dieting.Addictive\\nBehaviors, 19, 401–409.Garn, S. M., & Clark, D. C. (1976).\\n</SENT 27_04>\\n<SENT 27_05>\\nTrends in fatness and the origins of\\nobesity.Pediatrics, 57, 443–456.Goldfield, G. S., & Epstein, L. H. (2002).Can fruits and vegetables and\\nactivities substitute for snack foods?Health Psychology, 21,299–303.Goldfield, G. S., Epstein, L. H., Davidson, M., & Saad, F. G. (2006).Validation of a multiple choice questionnaire measure of the relative\\nreinforcing value of food.Eating Behaviors, 6,283–292.Gormally, J., Black, S., Daston, S., & Rardin, D. (1982).The assessment\\nof binge eating severity among obese persons.Addictive Behaviors, 7,\\n47–55.\\n</SENT 27_05>\\n<SENT 27_06>\\nGrandy, D. K., Litt, M., Allen, L., Bunzow, J. R., Marchionni, M., Makam,\\nH., et al.(1989).The human dopamine D2 receptor gene is located on\\nChromosome 11 at q22–q23 and identifies a TaqI RFLP.American\\nJournal of Human Genetics, 45,778–785.Hedeker, D., & Gibbons, R. D. (2006).Longitudinal data analysis.Hoboken, NJ: Wiley.Heinz, A., Goldman, D., Jones, D. W., Palmour, R., Hommer, D., Gorey,\\nJ. G., et al.(2000).Genotype influences in vivo dopamine transporter\\navailability in human striatum.Neuropsychopharmacology, 22, 133–\\n139.\\n</SENT 27_06>\\n<SENT 27_07>\\nHernandez, L., & Hoebel, B. G. (1988).Feeding and hypothalamic stimulation increase dopamine turnover in the accumbens.Physiology &\\nBehavior, 44, 599–606.Hernandez, L., & Hoebel, B. G. (1990).Feeding can enhance dopamine\\nturnover in the prefrontal cortex.Brain Research Bulletin, 25,975–979.Holsen, L. M., Zarcone, J. R., Thompson, T. I., Brooks, W. M., Anderson,\\nM. F., Ahluwalia, J. S., et al.(2005).Neural mechanisms underlying\\nfood motivation in children and adolescents.NeuroImage, 27,669–676.\\n</SENT 27_07>\\n<SENT 27_08>\\nHsiao, S., & Smith, G. P. (1995).Raclopride reduces sucrose preference in\\nrats.Pharmacology Biochemistry and Behavior, 50,121–125.Hursh, S. R., & Bauman, R. A.(1987).The behavioral analysis of demand.In L. Green & J. H. Kagel (Eds.),Advances in behavioral economics\\n(Vol.1, pp.117–165).Norwood, NJ: Ablex.Janson, A. M., Hedlund, P. B., Fuxe, K., & von Euler, G. (1994).Chronic\\nnicotine treatment counteracts dopamine D2 receptor upregulation induced by a partial meso-diencephalic hemitransection in the rat.Brain\\nResearch, 655, 25–32.Johnson, W. G. (1974).Effect of cue prominence and subject weight on\\nhuman food-directed performance.Journal of Personality and Social\\nPsychology, 29, 843–848.\\n</SENT 27_08>\\n<SENT 27_09>\\nJonsson, E. G., Nothen, M. M., Grunhage, F., Farde, L., Nakashima, Y.,\\nPropping, P., et al.(1999).Polymorphisms in the dopamine D2 receptor\\ngene and their relationships to striatal dopamine receptor density of\\nhealthy volunteers.Molecular Psychiatry, 4,290–296.Klesges, R. C., Meyers, A. W., Klesges, L. M., & LaVasque, M. D. (1989).Smoking, body weight, and their effects on smoking behavior: A comprehensive review of the literature.Psychological Bulletin, 106, 204–\\n230.Kraemer, H. C., Frank, E., & Kupfer, D. J.(2006).Moderators of treatment\\noutcomes: Clinical, research, and policy importance.\\n</SENT 27_09>\\n<SENT 27_10>\\nJournal of the\\nAmerican Medical Association, 296,1286–1289.Laruelle, M., Gelernter, J., & Innis, R. B.(1998).D2 receptors binding\\npotential is not affected by Taq1 polymorphism at the D2 receptor gene.Molecular Psychiatry, 3,261–265.Leddy, J. J., Epstein, L. H., Jaroni, J. L., Roemmich, J. N., Paluch, R. A.,\\nGoldfield, G. S., et al.(2004).The influence of methylphenidate on\\neating in obese men.Obesity Research, 12,224–232.Leyton, M., Boileau, I., Benkelfat, C., Diksic, M., Baker, G., & Dagher, A.(2002).Amphetamine-induced increases in extracellular dopamine, drug\\nwanting, and novelty seeking: A PET/[11C]raclopride study in healthy\\nmen.Neuropsychopharmacology, 27, 1027–1035.Li, M. D., Kane, J. K., & Konu, O.(2003).Nicotine, body weight and\\npotential implications in the treatment of obesity.Current Topics in\\nMedical Chemistry, 3,899–919.Li, S., Kim, K. Y., Kim, J. H., Park, M. S., Bahk, J. Y., & Kim, M. O.\\n</SENT 27_10>\\n<SENT 27_11>\\n(2004).Chronic nicotine and smoking treatment increases dopamine\\ntransporter mRNA expression in the rat midbrain.Neuroscience Letters,\\n363, 29–32.Miyata, G., Meguid, M. M., Fetissov, S. O., Torelli, G. F., & Kim, H. J.(1999).Nicotine’s effect on hypothalamic neurotransmitters and appetite\\nregulation.Surgery, 126, 255–263.Neville, M. J., Johnstone, E. C., & Walton, R. T. (2004).Identification and\\ncharacterization of ANKK1: A novel kinase gene closely linked to\\nDRD2 on chromosome band 11q23.1.Human Mutation, 23,540–545.NHLBI Obesity Education Initiative Expert Panel (1998).\\n</SENT 27_11>\\n<SENT 27_12>\\nClinical guidelines on the identification, evaluation, and treatment of overweight and\\nobesity in adults—The evidence report.Obesity Research, 6(Suppl.2),\\n51S–209S.Noble, E. P., Noble, R. E., Ritchie, T., Syndulko, K., Bohlman, M. C.,\\nNoble, L. A., et al.(1994).D2 dopamine receptor gene and obesity.International Journal of Eating Disorders, 15,205–217.Pohjalainen, T., Rinne, J. O., Nagren, K., Lehikoinen, P., Anttila, K.,\\nSyvalahti, E. K., et al.(1998).\\n</SENT 27_12>\\n<SENT 27_13>\\nThe A1 allele of the human D2 dopamine\\nreceptor gene predicts low D2 receptor availability in healthy volunteers.Molecular Psychiatry, 3,256–260.Raynor, H. A., & Epstein, L. H. (2003).The relative-reinforcing value of\\nfood under differing levels of food deprivation and restriction.Appetite,\\n40, 15–24.Reiss, S., & Havercamp, S. (1996).The sensitivity theory of motivation:\\n885FOOD REINFORCEMENT AND ENERGY INTAKE\\nThis document is copyrighted by the American Psychological Association or one of its allied publishers.This article is intended solely for the personal use of the individual user and is not to be disseminated broadly.\\n</SENT 27_13>\\n\\n\\n<SENT 28_01>\\nImplications for psychopathology.Behaviour Research and Therapy,\\n34, 621–632.Ritchie, T., & Noble, E. P. (2003).Association of seven polymorphisms of\\nthe D2 dopamine receptor gene with brain receptor-binding characteristics.Neurochemistry Research, 28,73–82.Saelens, B. E., & Epstein, L. H. (1996).The reinforcing value of food in\\nobese and non-obese women.Appetite, 27, 41–50.Small, D. M., Jones-Gotman, M., & Dagher, A.(2003).Feeding-induced\\ndopamine release in dorsal striatum correlates with meal pleasantness\\nratings in healthy human volunteers.NeuroImage, 19, 1709–1715.Spitzer, R. L., Devlin, M. J., Walsh, B., Hasin, D., Wing, R. R., Marcus,\\nM. D., et al.(1992).Binge eating disorder: A multisite field trial of the\\ndiagnostic criteria.International Journal of Eating Disorders, 11,191–\\n203.Stunkard, A. J., & Messick, S. (1985).The three-factor eating questionnaire to measure dietary restraint, disinhibition and hunger.Journal of\\nPsychosomatic Research, 29,71–83.Swinburn, B., Gill, T., & Kumanyika, S. (2005).\\n</SENT 28_01>\\n<SENT 28_02>\\nObesity prevention: A\\nproposed framework for translating evidence into action.Obesity Reviews, 6, 23–33.Temple, J. L., Giacomelli, A. M., Roemmich, J. N., & Epstein, L. H. (in\\npress).Dietary variety impairs habituation in children.Health Psychology.Vandenbergh, D. J., Persico, A. M., Hawkins, A. L., Griffin, C. A., Li, X.,\\nJabs, E. W., et al.(1992).Human dopamine transporter gene (DAT1)\\nmaps to chromosome 5p15.3 and displays a VNTR.Genomics, 14,\\n1104–1106.Volkow, N. D., Wang, G. J., Fowler, J. S., Logan, J., Jayne, M., Franceschi,\\nD., et al.\\n</SENT 28_02>\\n<SENT 28_03>\\n(2002).“Nonhedonic” food motivation in humans involves\\ndopamine in the dorsal striatum and methylphenidate amplifies this\\neffect.Synapse, 44, 175–180.Wade, T. R., de Wit, H., & Richards, J.B.(2000).Effects of dopaminergic\\ndrugs on delayed reward as a measure of impulsive behavior in rats.Psychopharmacology, 150,90–101.Wang, G. J., Volkow, N. D., Logan, J., Pappas, N. R., Wong, C. T., Zhu,\\nW., et al.(2001).Brain dopamine and obesity.Lancet, 357, 354–357.Wellman, P. J.(2005).Modulation of eating by central catecholamine\\nsystems.Current Drug Targets, 6,191–199.Whitaker, R. C., Wright, J.A., Pepe, M. S., Seidel, K. D., & Dietz, W. H.\\n(1997).\\n</SENT 28_03>\\n<SENT 28_04>\\nPredicting obesity in young adulthood from childhood and\\nparental obesity.New England Journal of Medicine, 337,869–873.Wise, R. A.(2006).Role of brain dopamine in food reward and reinforcement.Philosophical Transactions of the Royal Society, Series B, 361,\\n1149–1158.Received February 16, 2007\\nRevision received May 22, 2007\\nAccepted May 24, 2007/H18546\\n886 EPSTEIN ET AL.This document is copyrighted by the American Psychological Association or one of its allied publishers.This article is intended solely for the personal use of the individual user and is not to be disseminated broadly.\\n</SENT 28_04>\\n',\n",
       "  ['1. How does the up-regulation of dopamine D1 and D2 receptor mRNA in the caudate-putamen and nucleus accumbens of rat brains relate to smoking?',\n",
       "   '2. What is the significance of the behavioral economics of drug dependence in understanding the price of change?',\n",
       "   '3. How does the deconstruction of relative reinforcing efficacy contribute to situating the measures of pharmacological reinforcement within behavioral economics?',\n",
       "   '4. What is the purpose of using the eating disorder examination with children, and what were the findings of the pilot study conducted by Bryant-Waugh et al. (1996)?',\n",
       "   '5. How does metabolic need affect the sensitization of reward in feeding, drug abuse, and the dopaminergic system?']),\n",
       " ('<SENT 29_01>\\nCorrection to Epstein et al.(2007)\\nIn the article, “Food Reinforcement, the Dopamine D2 Receptor Genotype, and Energy Intake in\\nObese and Nonobese Humans,” by Leonard H. Epstein, Jennifer L. Temple, Brad J. Neaderhiser,\\nRobbert J. Salis, Richard W. Erbe, and John J. Leddy (Behavioral Neuroscience, 2007, Vol.121,\\nNo.5, pp.877–886), then values (and corresponding percentages) for the number of people with\\nthe A1/A1 & A1/A2 and A2/A2 genotypes were reversed in Table 2.The corrected table appears\\nbelow, with the revised numbers appearing in bold font.Table 2\\nParticipant Characteristics\\nVariable\\nObesity DRD2 SCL6A3\\nNonobese Obese A1/A1 & A1/A2 A2/A2 9/9 & 9/10 10/10\\nN (male/female) 45 (22/23) 29 (11/18) 37 (15/22) 37 (18/19) 31 (15/16) 41 (17/24)\\nAge (M/H11006SD) 23.4 /H110065.4a 29.3 /H110067.8a 26.1 /H110067.2 25.4 /H110066.8 26.7 /H110067.3 24.8 /H110066.6\\nBMI (kg/m2)( M /H11006SD) 23.2 /H110062.8 34.5 /H110064.6a 26.3 /H110066.7 29.0 /H110066.3 28.0 /H110066.2 27.0 /H110066.4\\nDietary restraint\\n(M /H11006SD) 5.8 /H110063.9 6.9 /H110064.2 6.4 /H110064.1 6.1 /H110063.9 6.9 /H110064.2 5.9 /H110063.9\\nBinge Eating Scale\\n(M /H11006SD) 6.8 /H110065.5 13.8 /H110068.0a 7.7 /H110066.5 11.4 /H110067.9b 10.3 /H110066.5 8.9 /H110067.7\\nBaseline hunger\\n(M /H11006SD) 53.3 /H1100622.6 52.0 /H1100626.2 50.4 /H1100624.4 55.2 /H1100623.4 52.9 /H1100625.7 53.5 /H1100623.1\\nAverage food liking\\n(M /H11006SD) 75.9 /H1100622.9 74.3 /H1100620.8 75.6 /H1100620.5 75.0 /H1100623.6 75.3 /H1100620.8 74.8 /H1100623.4\\nRace: n (%)\\nCaucasian 37 (82) 22 (76) 33 (89) 26 (70) 26 (84) 31 (76)\\nMinority 7 (16) 6 (21) 3 (8) 10 (27) 4 (13) 9 (22)\\nMixed race 1 (2) 1 (3) 1 (3) 1 (3) 1 (3) 1 (2)\\nEducation: n (%)\\nHigh school 28 (62) 17 (59) 20 (54) 25 (68) 18 (58) 25 (61)\\nCollege degree 17 (38) 12 (41) 17 (46) 12 (32) 13 (42) 16 (39)\\nDRD2 n (%)\\nA1/A1 & A1/A2 20 (44) 17 (59) 17 (55) 19 (46)\\nA2/A2 25 (56) 12 (41) 14 (45) 22 (54)\\nSLC6A3 n (%)\\n9/9 & 9/10 19 (43) 12 (43) 17 (47) 14 (39)\\n10/10 25 (57) 16 (57) 19 (53) 22 (61)\\nNote.\\n</SENT 29_01>\\n<SENT 29_02>\\nDietary restraint scores were derived from the Three-Factor Eating Questionnaire (Stunkard & Messick, 1985).DRD2 /H11005dopamine D2 receptor;\\nBMI /H11005body mass index.aSignificant differences between obese and nonobese,p /H11021.001.bSignificant differences betweenDRD2 genotypes, p /H11021.001.This document is copyrighted by the American Psychological Association or one of its allied publishers.This article is intended solely for the personal use of the individual user and is not to be disseminated broadly.\\n</SENT 29_02>\\n',\n",
       "  [\"1. What was the original error in Table 2 of Epstein et al.'s (2007) article?\",\n",
       "   '2. How does the correction in Table 2 affect the interpretation of the results in the study?',\n",
       "   '3. Are there any significant differences in age, BMI, dietary restraint, binge eating scale, baseline hunger, or average food liking between the A1/A1 & A1/A2 and A2/A2 genotypes?',\n",
       "   '4. How do the corrected participant characteristics in Table 2 impact the conclusions drawn in the study?',\n",
       "   '5. What is the relationship between the dopamine D2 receptor genotype and energy intake in obese and nonobese humans, as suggested by the corrected data in Table 2?'])]"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "question_context_pair"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "\n",
    "with open(f\"eval_data_generated_{date_time_str}.json\", \"w\") as f:\n",
    "    json.dump(question_context_pair, f)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "kalbecdt-genomic-paper-extractor-proto-tKxcan3_-py3.11",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
